<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.27">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Edwin D.H.M. Reuvers">
<meta name="author" content="Dinant A. Kistemaker">
<meta name="dcterms.date" content="2026-02-26">

<title>Accuracy of experimentally estimated muscle properties: Evaluation and improvement using a newly developed toolbox – Muscle property estimation</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.min.js" integrity="sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2" crossorigin="anonymous"></script><script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-ed96de9b727972fe78a7b5d16c58bf87.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-6790a8849efb91e08b49b6924511a186.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<link href="../site_libs/quarto-contrib/fontawesome6-0.1.0/all.css" rel="stylesheet">
<link href="../site_libs/quarto-contrib/fontawesome6-0.1.0/latex-fontsize.css" rel="stylesheet">
<script src="../site_libs/quarto-contrib/iconify-2.0.0/iconify-icon.min.js"></script>
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script>
  MathJax = {
    tex: {
      inlineMath: [['$', '$'], ['\\(', '\\)']]
    }
  };
</script>
<script id="MathJax-script" async="" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js">
</script>
<script src="https://cdn.jsdelivr.net/npm/requirejs@2.3.6/require.min.js" integrity="sha384-c9c+LnTbwQ3aujuU7ULEPVvgLs+Fn6fJUvIGTsuu1ZcCf11fiEubah0ttpca4ntM sha384-6V1/AdqZRWk1KAlWbKBlGhN7VG4iE/yAZcO6NZPMF8od0vukrvr0tg4qY6NSrItx" crossorigin="anonymous"></script>

<script type="application/javascript">define('jquery', [],function() {return window.jQuery;})</script>

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN" && texText && texText.data) {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Muscle property estimation</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../data/readme.html"> 
<span class="menu-text"><iconify-icon inline="" icon="fa6-regular:floppy-disk" aria-label="Icon floppy-disk from fa6-regular Iconify.design set." title="Icon floppy-disk from fa6-regular Iconify.design set."></iconify-icon> Data</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../analysis/readme.html"> 
<span class="menu-text"><iconify-icon inline="" icon="fa6-solid:magnifying-glass-chart" aria-label="Icon magnifying-glass-chart from fa6-solid Iconify.design set." title="Icon magnifying-glass-chart from fa6-solid Iconify.design set."></iconify-icon> Analysis</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../figures/all_figures.html"> 
<span class="menu-text"><iconify-icon inline="" icon="carbon:qq-plot" aria-label="Icon qq-plot from carbon Iconify.design set." title="Icon qq-plot from carbon Iconify.design set."></iconify-icon> Figures</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../tables/all_tables.html"> 
<span class="menu-text"><iconify-icon inline="" icon="carbon:data-table" aria-label="Icon data-table from carbon Iconify.design set." title="Icon data-table from carbon Iconify.design set."></iconify-icon> Tables</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../manuscript/preprint.html" aria-current="page"> 
<span class="menu-text"><iconify-icon inline="" icon="carbon:document" aria-label="Icon document from carbon Iconify.design set." title="Icon document from carbon Iconify.design set."></iconify-icon> Manuscript</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
    <a href="https://bsky.app/profile/edwinreuvers.bsky.social" title="" class="quarto-navigation-tool px-1" aria-label="Bluesky - Edwin Reuvers"><i class="bi bi-bluesky"></i></a>
    <a href="https://github.com/edwinreuvers" title="GitHub - Edwin Reuvers" class="quarto-navigation-tool px-1" aria-label="GitHub - Edwin Reuvers"><i class="bi bi-github"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">1</span> Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">2</span> Methods</a>
  <ul class="collapse">
  <li><a href="#sec-m-model" id="toc-sec-m-model" class="nav-link" data-scroll-target="#sec-m-model"><span class="header-section-number">2.1</span> Muscle model</a></li>
  <li><a href="#simulated-and-in-situ-data" id="toc-simulated-and-in-situ-data" class="nav-link" data-scroll-target="#simulated-and-in-situ-data"><span class="header-section-number">2.2</span> Simulated and <em>in situ</em> data</a></li>
  <li><a href="#sec-m-est-procedure" id="toc-sec-m-est-procedure" class="nav-link" data-scroll-target="#sec-m-est-procedure"><span class="header-section-number">2.3</span> Parameter value estimation procedure</a></li>
  <li><a href="#sec-m-sa" id="toc-sec-m-sa" class="nav-link" data-scroll-target="#sec-m-sa"><span class="header-section-number">2.4</span> Sensitivity analysis</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">3</span> Results</a>
  <ul class="collapse">
  <li><a href="#evaluating-parameter-value-estimation-accuracy---simulated-data" id="toc-evaluating-parameter-value-estimation-accuracy---simulated-data" class="nav-link" data-scroll-target="#evaluating-parameter-value-estimation-accuracy---simulated-data"><span class="header-section-number">3.1</span> Evaluating parameter value estimation accuracy - simulated data</a></li>
  <li><a href="#sensitivity-analysis" id="toc-sensitivity-analysis" class="nav-link" data-scroll-target="#sensitivity-analysis"><span class="header-section-number">3.2</span> Sensitivity analysis</a></li>
  <li><a href="#evaluating-model-predictions---in-situ-data" id="toc-evaluating-model-predictions---in-situ-data" class="nav-link" data-scroll-target="#evaluating-model-predictions---in-situ-data"><span class="header-section-number">3.3</span> Evaluating model predictions - <em>in situ</em> data</a></li>
  </ul></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">4</span> Discussion</a></li>
  <li><a href="#acknowledgments" id="toc-acknowledgments" class="nav-link" data-scroll-target="#acknowledgments">Acknowledgments</a></li>
  <li><a href="#funding" id="toc-funding" class="nav-link" data-scroll-target="#funding">Funding</a></li>
  <li><a href="#data-and-resource-availability" id="toc-data-and-resource-availability" class="nav-link" data-scroll-target="#data-and-resource-availability">Data and resource availability</a></li>
  <li><a href="#glossary" id="toc-glossary" class="nav-link" data-scroll-target="#glossary">Glossary</a></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  <li><a href="#supplementary-material" id="toc-supplementary-material" class="nav-link" data-scroll-target="#supplementary-material">Supplementary material</a>
  <ul class="collapse">
  <li><a href="#supplemental-figures" id="toc-supplemental-figures" class="nav-link" data-scroll-target="#supplemental-figures"><span class="header-section-number">4.1</span> Supplemental figures</a></li>
  <li><a href="#supplemental-tables" id="toc-supplemental-tables" class="nav-link" data-scroll-target="#supplemental-tables"><span class="header-section-number">4.2</span> Supplemental tables</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><ul><li><a href="https://github.com/edwinreuvers/acc-mp-estimation/blob/main/manuscript/preprint.ipynb" class="toc-action"><i class="bi bi-github"></i>View source</a></li></ul></div><div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="https://doi.org/10.1101/2025.09.29.678508"><i class="bi bi-pencil-square"></i>Preprint</a></li><li><a href="preprint.pdf"><i class="bi bi-file-pdf"></i>Preprint (PDF)</a></li></ul></div><div class="quarto-alternate-notebooks"><h2>Notebooks</h2><ul><li><a href="../figures/all_figures.html"><i class="bi bi-journal-code"></i>Figures</a></li><li><a href="../tables/all_tables.html"><i class="bi bi-journal-code"></i>Tables</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><p>Accuracy of experimentally estimated muscle properties: Evaluation and improvement using a newly developed toolbox</p></h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Authors</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author"><a href="https://edwinreuvers.com/">Edwin D.H.M. Reuvers</a> <a href="mailto:edwin.reuvers.academia@pm.me" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/https://orcid.org/0000-0001-8932-1959" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Vrije Universiteit Amsterdam
          </p>
      </div>
    <div class="quarto-title-meta-contents">
    <p class="author">Dinant A. Kistemaker <a href="mailto:da.kistemaker@vu.nl" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/https://orcid.org/0000-0003-1535-009X" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            Vrije Universiteit Amsterdam
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
    <div>
    <!-- <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">February 26, 2026</p>
    </div> -->
  </div>
  
    
  </div>
  
<div>
  <div class="abstract">
    <div class="block-title">Abstract</div>
    <p>The mechanical behaviour of a muscle-tendon complex depends on properties such as the force-length relationships, the force-velocity relationship, and the excitation dynamics. Quick-release and step-ramp experiments are commonly used to estimate these properties. The accuracy of these methods is unclear, as the actual values of these properties are unknown in experiments on real muscle. We conducted a modelling study using a Hill-type muscle-tendon complex model with literature-derived parameter values and simulated quick-release, step-ramp, and isometric experiments. From the simulated experiments, we assessed how accurately the model’s parameter values could be retrieved. Using a method traditionally used in literature, the series elastic element stiffness was underestimated by ~35%, due to the incorrect assumption that muscle fibres do not shorten during quick releases. Consequently, this yielded an overestimation of the excitation dynamics activation time constants of ~20%. We developed an improved method that accounted for muscle fibre length shortening during quick releases. Using our improved method, all parameter values closely matched their actual values. A sensitivity analysis showed that the most critical parameters were robust to perturbations in experimental data. Lastly, we compared Hill-type MTC model predictions against <em>in situ</em> data from three rat m. gastrocnemius medialis muscles. Predictions based on parameters from the improved method showed closer agreement than those based on the traditional method — both for quick-release, step-ramp, and isometric experiments, as well as for independent stretch-shortening cycles. In conclusion, the improved method enables more accurate estimates of muscle-tendon complex properties, addressing limitations of the traditionally used method.</p>
  </div>
</div>


</header>


<section id="introduction" class="level2" data-number="1">
<h2 data-number="1" class="anchored" data-anchor-id="introduction"><span class="header-section-number">1</span> Introduction</h2>
<p>In the early 20th century, A.V. Hill laid the foundation of muscle mechanics by recognising that muscles contain not only muscle fibres (the contractile element; CE) but also connective tissue, both parallel to the muscle fibre (the parallel elastic element; PEE) and in series with the muscle fibre (the tendon or serial elastic element; SEE). Together these three elements form the muscle-tendon-complex (MTC). The contraction dynamics describe how CE force emerges from the interplay between the force-length properties of CE, PEE and SEE, and the force-velocity properties of CE. In addition, CE force depends on the active state <span class="citation" data-cites="ebashi_calcium_1968">(the relative amount of <span class="math inline">\(Ca^{2+}\)</span> bound to troponin C, <a href="#ref-ebashi_calcium_1968" role="doc-biblioref">Ebashi and Endo 1968</a>)</span>. Active state, in turn, depends on activation via the excitation dynamics. Consequently, the mechanical behaviour of MTC arises from the intricate interplay between the contraction and the excitation dynamics.</p>
<p>In his famous <span class="citation" data-cites="hill_heat_1938">(<a href="#ref-hill_heat_1938" role="doc-biblioref">1938</a>)</span> paper A.V. Hill performed so-called ‘isotonic quick-release experiments’ using the now-renowned Levin-Wyman lever <span class="citation" data-cites="levin_viscous_1927">(<a href="#ref-levin_viscous_1927" role="doc-biblioref">Levin and Wyman 1927</a>)</span> to estimate the CE force-velocity relationship. In these experiments, MTC was attached to a lever and maximally stimulated. When SEE force plateaued (i.e., MTC was isometrically delivering force), a magnet was released from the lever causing a sudden drop in the force acting on the MTC. As SEE force remained constant right after this drop in SEE force, SEE length remained constant and therefore all MTC length changes were attributed to CE length changes. The CE velocity corresponding to the constant SEE force after the drop was then calculated as the maximum rate of change of MTC length over time. Consequently, each isotonic quick-release experiment contributed a single data point of the CE force-velocity relationship. Nowadays, servomotors are typically used in experiments on isolated MTCs to control MTC length (changes). In contrast to the force-controlled experiments with the Levin-Wyman lever, servomotor control the MTC length over time (see <a href="#suppfig-servo-vs-lever" class="quarto-xref">Figure S1</a>). In order to estimate the CE force-velocity relationship using a servomotor, <span class="citation" data-cites="cecchi_force-velocity_1978">Cecchi, Colomo, and Lombardi (<a href="#ref-cecchi_force-velocity_1978" role="doc-biblioref">1978</a>)</span> introduced ‘step-ramp experiments’. In these experiments, MTC is kept isometrically under maximal stimulation until SEE force plateaus. Then, a rapid shortening of MTC length is imposed (the ‘step’), immediately followed by a constant-velocity shortening of the MTC (the ‘ramp’). This experimental protocol results in a brief plateau in SEE force just after the ramp. During this brief period, SEE length remains constant and therefore CE velocity equals MTC velocity. Thus, step-ramp experiments with servomotors serve as a valid alternative for isotonic quick-release experiments to estimate the CE force-velocity relationship.</p>
<p>In addition to estimating the CE force-velocity relationship using isotonic quick-release experiments, the research group of A.V. Hill also used these experiments to approximate SEE stiffness. The underlying idea was that the drop in MTC force was so fast that, due to contraction dynamics, CE had no time to shorten and all MTC shortening could thus be directly attributed to SEE shortening. Consequently, this experiment allowed to directly relate SEE length changes to changes in SEE force and therefore to estimate SEE stiffness. As mentioned above, in isotonic quick-release experiments, a sudden drop in force acting on the MTC causes a rapid change in MTC length. While in these lever-based experiments MTC force is controlled, in experiments using a servomotor MTC length is controlled. To estimate SEE stiffness with a servomotor setup, the opposite approach is taken: a rapid change in MTC length is imposed by the motor, resulting in a rapid change in SEE force. Somewhat confusingly, these servomotor-based experiments are also referred to as ‘quick-release experiments’, even though their approach differs fundamentally from that of an isotonic quick-release experiment. In the remainder of this paper, the term quick-release experiments will exclusively refer to servomotor-based length-controlled quick-release experiments. In quick-release experiments using a servomotor, the drop in MTC length takes several milliseconds (e.g., approximately 10 ms with the commonly used Aurora Scientific’s 300C series motor). As CE shortens within this short time period, the assumption that MTC length changes are exclusively due to SEE length changes may not be entirely valid. This raises the question to what extent SEE stiffness can be accurately estimated based on quick-release experiments using servomotors.</p>
<p>A.V. Hill already acknowledged in <span class="citation" data-cites="hill_series_1950">(<a href="#ref-hill_series_1950" role="doc-biblioref">1950</a>)</span> that CE shortens during the brief period in which SEE force decreases – even for his isotonic quick-release experiments using the Levin-Wyman lever – and introduced a method to correct for this CE shortening. In this method, the CE shortening that occurred due to the quick-release was estimated based on the CE force-velocity relationship of the MTC investigated. Thus, additional experimental work – such as performing step-ramp experiments and estimating the CE force-velocity relationship – are required to correct for CE shortening during the quick-release. <span class="citation" data-cites="van_zandwijk_evaluation_1997">Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref">1997</a>)</span> and <span class="citation" data-cites="lemaire_comparison_2016">Lemaire et al. (<a href="#ref-lemaire_comparison_2016" role="doc-biblioref">2016</a>)</span> developed similar methods to correct for CE shortening. As expected, <span class="citation" data-cites="van_zandwijk_evaluation_1997">Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref">1997</a>)</span> showed that this method results in higher estimates of SEE stiffness than with the method without correcting for CE shortening. Nevertheless, the accuracy of SEE stiffness estimates remains uncertain, as the actual SEE stiffness is unknown in experiments, making it infeasible to compare the estimated value against the actual one. Consequently, it remains unclear whether the additional complexity and experimental work required for the improved method are truly justified in comparison with the simpler traditional method, especially when the only interest is SEE stiffness.</p>
<p>In experimental studies in which MTC properties are estimated, SEE stiffness is typically estimated first in order to discriminate between SEE stiffness and the other properties, such as those of the CE force-length relationship. In experiments, CE length is unknown and therefore the properties of the CE force-length relationship cannot be directly estimated based on experimental data. Typically, the CE force-length properties are estimated based on the measured MTC force-length relationship <span class="citation" data-cites="blumel_determining_2012 lemaire_comparison_2016">(e.g., <a href="#ref-blumel_determining_2012" role="doc-biblioref">Blümel et al. 2012</a>; <a href="#ref-lemaire_comparison_2016" role="doc-biblioref">Lemaire et al. 2016</a>)</span>. However, different combinations of CE and SEE properties can yield almost identical MTC force-length relationships. Consequently, inaccuracies in SEE stiffness estimates can propagate to errors in the estimation of the CE force-length properties. While predictions under isometric conditions may remain reasonably accurate, these errors can lead to substantially different mechanical behaviour during dynamic contractions such as stretch-shortening cycles (see <a href="#fig-i-effect-ksee" class="quarto-xref">Figure&nbsp;1</a>). As such, incorrect estimation of SEE stiffness might (partially) explain difference between predictions derived by a Hill-type MTC model and experimental results. In sum, the question arises to which extent the estimates of other MTC properties are affected by inaccurate SEE stiffness estimates and whether more accurate SEE stiffness estimates improves predictions derived by a Hill-type MTC model.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-i-effect-ksee" class="cell" data-execution_count="13">
<div class="cell-output cell-output-display">
<div id="fig-i-effect-ksee" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-i-effect-ksee-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-i-effect-ksee-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-i-effect-ksee-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Effect of a 2.5-fold difference in SEE stiffness on simulated mechanical behaviour. This example shows that some contractions may lead to similar mechanical behaviour across distinct parameter sets, others may differ substantially.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-i-effect-ksee">Source: Figures</a></div>
<p>The potential effect of error propagation also extends to the estimation of the excitation dynamics properties. In the absence of histochemical data, excitation dynamics properties are commonly estimated by fitting predicted SEE force of the Hill-type MTC model to experimental data <span class="citation" data-cites="blumel_determining_2012 lemaire_comparison_2016">(e.g., <a href="#ref-blumel_determining_2012" role="doc-biblioref">Blümel et al. 2012</a>; <a href="#ref-lemaire_comparison_2016" role="doc-biblioref">Lemaire et al. 2016</a>)</span>. As SEE force over time obviously depend on the contraction dynamics properties (e.g., the force-length relationships and the force-velocity relationship), the estimation of the excitation dynamics properties is not independent of the estimation of the contraction dynamics properties. Consequently, inaccuracies in SEE stiffness estimates can propagate to inaccuracies in estimated excitation dynamics properties. This interdependence raises an additional question of what experimental data should be used in order to minimise this potential error propagation.</p>
<p>The primary objective of this study was to systematically evaluate the accuracy of estimating contraction and excitation dynamics properties on the basis of data collected in commonly used experiments. Specifically, we aimed to compare the results of a method that does not correct for CE shortening in quick-release experiments (the ‘traditional method’) with a method that does correct for CE shortening (the ‘improved method’). As explained earlier, the limitation of relying exclusively on an experimental approach is that the actual properties are unknown, which renders a reliable assessment of the accuracy of the estimated properties not feasible. To circumvent this problem, we conducted a modelling study using a Hill-type MTC model. First, we obtained three different sets of parameter values of a Hill-type MTC model from existing literature. Using these parameter values, we simulated data of quick-release, step-ramp and isometric protocols mimicking experiments on isolated MTCs using servomotors. We then investigated how accurately we could retrieve the model’s parameter values (with respect to their ‘actual’ value). In addition, we employed a comprehensive sensitivity analysis to assess the sensitivity of parameter values against perturbations in experimental data and to examine the interdependency of the estimated properties. The secondary objective of this study was to evaluate whether predictions from a Hill-type MTC model using parameter values estimated with the improved method better matched experimental data than predictions using parameter values estimated with the traditional method. To this end, we used <em>in situ</em> data from experiments on three rat m. gastrocnemius medialis. We estimated the contraction and excitation dynamics properties using both the traditional and the improved method, and compared the resulting model predictions — based on each parameter set — to experimental force data. Overall, this study provides insights into the accuracy of existing commonly used methods for contraction and excitation dynamics property estimation and their influence on predictions of mechanical behaviour derived by a Hill-type MTC model. The approach that we designed in this study for estimating contraction and excitation dynamics properties based on data of quick-release, step-ramp and isometric experiments using servomotors, is made available as an open-source toolbox (<a href="https://github.com/edwinreuvers/mp-estimator">https://github.com/edwinreuvers/mp-estimator</a>).</p>
</section>
<section id="methods" class="level2" data-number="2">
<h2 data-number="2" class="anchored" data-anchor-id="methods"><span class="header-section-number">2</span> Methods</h2>
<p>The primary objective of this study was to assess the accuracy of estimating contraction and excitation dynamics properties. For this purpose, we employed a Hill-type MTC model to simulate MTC mechanical behaviour, as detailed in <a href="#sec-m-model" class="quarto-xref">Section&nbsp;2.1</a>. Using parameter values obtained from existing literature, we simulated data of quick-release, step-ramp, and isometric experiments, mimicking experiments on isolated MTCs with servomotors, as described in <a href="#sec-m-sim-data" class="quarto-xref">Section&nbsp;2.2.1</a>. We then estimated the contraction and excitation dynamics parameter values using two methods: one with (the ‘traditional method’) and one without (the ‘improved method’) a correction for CE shortening due to the quick-release (see <a href="#sec-m-est-procedure" class="quarto-xref">Section&nbsp;2.3</a>). Additionally, we conducted comprehensive sensitivity analyses to assess the accuracy of the improved method, as outlined in <a href="#sec-m-sa" class="quarto-xref">Section&nbsp;2.4</a>. The secondary objective was to evaluate predictions derived by a Hill-type MTC model using two sets of parameter values: one estimated by the traditional method, the other by the improved method. For this purpose, we used <em>in situ</em> data (see <a href="#sec-m-insitu-data" class="quarto-xref">Section&nbsp;2.2.2</a>) from three rat m. gastrocnemius medialis. Predictions were assessed against quick-release, step-ramp, and isometric experiments, as well as independently measured stretch-shortening cycles. This allowed us to investigate whether the improved method enhanced the predictions derived by a Hill-type MTC model.</p>
<section id="sec-m-model" class="level3" data-number="2.1">
<h3 data-number="2.1" class="anchored" data-anchor-id="sec-m-model"><span class="header-section-number">2.1</span> Muscle model</h3>
<section id="contraction-dynamics" class="level4" data-number="2.1.1">
<h4 data-number="2.1.1" class="anchored" data-anchor-id="contraction-dynamics"><span class="header-section-number">2.1.1</span> Contraction dynamics</h4>
<p>We employed a Hill-type MTC model consisting of a contractile element (CE) and a parallel elastic element (PEE), which were both in series with a serial elastic element (SEE), as depicted in <a href="#fig-m-hill-model" class="quarto-xref">Figure&nbsp;2</a>. CE represented the contractile element of the muscle fibres, while PEE and SEE represented the tissues arranged in parallel and in series with the muscle fibres, respectively. Given the negligible small mass of MTC compared to the forces typically delivered by CE, PEE, and SEE, we simplified the model by neglecting the second-order dynamics of the system:</p>
<p><span id="eq-lcelseelmtc"><span class="math display">\[
\begin{aligned}
  L_{MTC}             &amp;= L_{CE} + L_{SEE} +  \\
  F_{SEE}               &amp;= F_{CE} + F_{PEE}
\end{aligned}
\tag{1}\]</span></span></p>
<p><span class="math inline">\(L_{MTC}\)</span>, <span class="math inline">\(L_{CE}\)</span>, <span class="math inline">\(L_{SEE}\)</span> and <span class="math inline">\(L_{PEE}\)</span> denote MTC, CE, PEE and SEE length respectively, while <span class="math inline">\(F_{SEE}\)</span>, <span class="math inline">\(F_{CE}\)</span>, and <span class="math inline">\(F_{PEE}\)</span> denote the CE, PEE and SEE force. PEE and SEE were assumed be purely elastic and were modelled as quadratic springs <span class="citation" data-cites="zajac_muscle_1989">(<a href="#ref-zajac_muscle_1989" role="doc-biblioref">Zajac 1989</a>,)</span>:</p>
<p><span id="eq-fee"><span class="math display">\[
F_{EE}=
  \begin{cases}
    k_{SEE} \cdot (l_{EE}-L_{EE}^{0})^2, &amp; \text{if}\ l_{EE} \ge L_{EE}^{0} \\
    0, &amp; \text{otherwise}
  \end{cases}
\tag{2}\]</span></span></p>
<p><span class="math inline">\(L_{EE}\)</span> denotes the length of either PEE or SEE and <span class="math inline">\(L_{EE}^{0}\)</span> the slack length of either PEE or SEE. <span class="math inline">\(k_{EE}\)</span> denotes a parameter that scales the stiffness of either PEE or SEE.</p>
<p>Isometric CE force depends on CE length <span class="citation" data-cites="blix_lange_1892 blix_lange_1893 hill_length_1925">(<a href="#ref-blix_lange_1892" role="doc-biblioref">Blix 1892</a>, <a href="#ref-blix_lange_1893" role="doc-biblioref">1893</a>; <a href="#ref-hill_length_1925" role="doc-biblioref">A. V. Hill 1925</a>)</span>. We simplified the classic sarcomere force-length relationship <span class="citation" data-cites="gordon_variation_1966 walker_i_1974">(<a href="#ref-gordon_variation_1966" role="doc-biblioref">Gordon, Huxley, and Julian 1966</a>; <a href="#ref-walker_i_1974" role="doc-biblioref">Walker and Schrodt 1974</a>,)</span> to a second order polynomial, which describes the force-length relationship reasonably well <span class="citation" data-cites="bobbert_force-length_1990 woittiez_three-dimensional_1984">(<a href="#ref-bobbert_force-length_1990" role="doc-biblioref">Bobbert, Ettema, and Huijing 1990</a>; <a href="#ref-woittiez_three-dimensional_1984" role="doc-biblioref">Woittiez et al. 1984</a>)</span>:</p>
<p><span id="eq-forcelength"><span class="math display">\[
F_{CE}^{isom,rel} =
  \begin{cases}
    \frac{-1}{w^{2}} \cdot (L_{CE}^{rel}-1)^2+1, &amp; \text{if} \ F_{CE}^{isom,rel} &gt; 0.1 \\
    s_{exp} \cdot e^{k_{exp} \cdot L_{CE}^{rel}}, &amp; \text{otherwise}
  \end{cases}
\tag{3}\]</span></span> <span class="math inline">\(F_{CE}^{isom,rel}\)</span> denotes the CE isometric force normalised by <span class="math inline">\(F_{CE}^{max}\)</span>, <span class="math inline">\(w\)</span> determines the width of the CE force-length relationship (for <span class="math inline">\(F_{CE}^{isom,rel} &gt; 0.1\)</span>) and <span class="math inline">\(L_{CE}^{rel}\)</span> denotes CE length normalised by CE optimum length (<span class="math inline">\(L_{CE}^{opt}\)</span>, the CE length at <span class="math inline">\(F_{CE}^{isom,rel}=1\)</span>). Exponential tails were added to the isometric CE force-length relationship such that it had a continuous first derivative with respect to <span class="math inline">\(L_{CE}^{rel}\)</span> (see <a href="#fig-m-hill-model" class="quarto-xref">Figure&nbsp;2</a>). The parameter values of <span class="math inline">\(s_{exp}\)</span> and <span class="math inline">\(k_{exp}\)</span> were determined separately for the ascending and descending limb of the CE force-length relationship.</p>
<p>The concentric CE force-velocity was modelled according to <span class="citation" data-cites="hill_heat_1938">Archibald Vivian Hill (<a href="#ref-hill_heat_1938" role="doc-biblioref">1938</a>)</span>. The original description was adjusted as suggested by <span class="citation" data-cites="van_soest_contribution_1993">Soest and Bobbert (<a href="#ref-van_soest_contribution_1993" role="doc-biblioref">1993</a>)</span> to account for situations where the active state <span class="citation" data-cites="ebashi_calcium_1968">(<span class="math inline">\(q\)</span>, the relative amount of <span class="math inline">\(Ca^{2+}\)</span> bound to troponin C, <a href="#ref-ebashi_calcium_1968" role="doc-biblioref">Ebashi and Endo 1968</a>)</span> is not maximal:</p>
<p><span id="eq-fvcon"><span class="math display">\[
F_{CE} = \frac{q \cdot (F_{CE}^{isom,rel} \cdot F_{CE}^{max} \cdot b + a \cdot v_{CE})}{b-v_{CE}}
\tag{4}\]</span></span></p>
<p><span class="math inline">\(F_{CE}\)</span> denotes the CE force and <span class="math inline">\(v_{CE}\)</span> denotes the CE velocity. <span class="math inline">\(a\)</span> is a parameter that defines the curvature of the concentric CE force-velocity relationship and defines together with <span class="math inline">\(b\)</span> and <span class="math inline">\(F_{CE}^{max}\)</span> the maximum shortening velocity (<span class="math inline">\(v_{CE}^{max}\)</span>, <span class="math inline">\(v_{CE}^{max} = -b/a \cdot F_{CE}^{max}\)</span>). The maximum shortening velocity is scaled by <span class="math inline">\(F_{CE}^{isom,rel}\)</span> due to the formulation of the CE force-velocity relationship (see <a href="#eq-fvcon" class="quarto-xref">Equation&nbsp;4</a>). However, the maximum shortening velocity has been reported to be more or less constant above optimum CE length <span class="citation" data-cites="gordon_variation_1966 stern_computer_1974">(<a href="#ref-gordon_variation_1966" role="doc-biblioref">Gordon, Huxley, and Julian 1966</a>; <a href="#ref-stern_computer_1974" role="doc-biblioref">Stern 1974</a>)</span>. Based on this observation, the parameter <span class="math inline">\(a\)</span> was scaled by <span class="math inline">\(F_{CE}^{isom,rel}\)</span> above optimum CE length (<span class="math inline">\(L_{CE}^{rel} &gt; 1\)</span>) to make the maximum shortening velocity independent of CE length above optimum CE length. The parameter <span class="math inline">\(b\)</span> was scaled with <span class="math inline">\(b_{scale}\)</span> for low levels of active state in order to make the maximal contraction velocity dependent on the active state <span class="citation" data-cites="petrofsky_influence_1981">(<a href="#ref-petrofsky_influence_1981" role="doc-biblioref">Petrofsky and Phillips 1981</a>)</span>. The original description of this scaling by <span class="citation" data-cites="van_soest_contribution_1993">Soest and Bobbert (<a href="#ref-van_soest_contribution_1993" role="doc-biblioref">1993</a>)</span> was somewhat reformulated, such that <span class="math inline">\(b_{scale}\)</span> was continuously differentiable with respect to <span class="math inline">\(q\)</span>:</p>
<p><span id="eq-bscale"><span class="math display">\[
\begin{aligned}
b_{scale} &amp;= \frac{1}{1 + e^{-b_{shape} \cdot (q-q_{sc})}} \\[0.5em]
\text{with} \quad q_{sc} &amp;= \frac{log(\frac{1}{b_{scale}^{min}-1}+q_0 \cdot b_{shape})}{b_{shape}}
\end{aligned}
\tag{5}\]</span></span></p>
<p><span class="math inline">\(q_0\)</span> denotes the minimum value of <span class="math inline">\(q\)</span>, <span class="math inline">\(b_{scale}^{min}\)</span> denotes the minimal scale factor of <span class="math inline">\(b\)</span> (i.e., the minimal value of <span class="math inline">\(b_{scale}\)</span>) and <span class="math inline">\(b_{shape}\)</span> defines the steepness of the relation between <span class="math inline">\(b_{scale}\)</span> and <span class="math inline">\(q\)</span>. The value of <span class="math inline">\(b_{shape}\)</span> was obtained by minimising the least square error between the formulation of <span class="citation" data-cites="van_soest_contribution_1993">Soest and Bobbert (<a href="#ref-van_soest_contribution_1993" role="doc-biblioref">1993</a>)</span> and the reformulated version of the relation between <span class="math inline">\(b_{scale}\)</span> and <span class="math inline">\(q\)</span>.</p>
<p>The eccentric part of the CE force-velocity relationship was modelled as a slanted hyperbola function <span class="citation" data-cites="van_soest_consequences_2005">(<a href="#ref-van_soest_consequences_2005" role="doc-biblioref">Van Soest, Gföhler, and Casius 2005</a>)</span> of the following form: <span id="eq-forcevelocityecc"><span class="math display">\[
F_{CE}^{rel} = \frac{c_4 - (c_3 + v_{CE}^{rel} \cdot (c_1 + c_2 \cdot v_{CE}^{rel})}{c_3 \cdot v_{CE}^{rel}}
\tag{6}\]</span></span></p>
<p><span class="math inline">\(c_1\)</span>, <span class="math inline">\(c_2\)</span>, <span class="math inline">\(c_3\)</span> and <span class="math inline">\(c_4\)</span> denote parameters that define the shape of the slanted hyperbola and depend on <span class="math inline">\(F_{CE}^{isom,rel}\)</span> and <span class="math inline">\(q\)</span>. These parameters were chosen such that (1) the concentric and eccentric curves were continuous in the point <span class="math inline">\(v_{CE}^{rel}=0\)</span>; (2) the ratio between the eccentric and concentric derivatives of <span class="math inline">\(\frac{dF_{CE}^{rel}}{dv_{CE}^{rel}}\)</span> in the point <span class="math inline">\(v_{CE}^{rel}=0\)</span> was defined by <span class="math inline">\(r_{slope}\)</span>; (3) the oblique asymptote of the eccentric curve was defined by <span class="math inline">\(F_{asymp}\)</span> and (4) the slope of the slanted asymptote was defined by <span class="math inline">\(r_{as}\)</span>.</p>
</section>
<section id="excitation-dynamics" class="level4" data-number="2.1.2">
<h4 data-number="2.1.2" class="anchored" data-anchor-id="excitation-dynamics"><span class="header-section-number">2.1.2</span> Excitation dynamics</h4>
<p>The excitation dynamics were modelled in two steps. The first step, to which we refer as the calcium dynamics, related the rate of change in normalised free <span class="math inline">\(Ca^{2+}\)</span> concentration between the myofilaments (<span class="math inline">\(\gamma\)</span>) to normalised <span class="math inline">\(CE\)</span> stimulation (<span class="math inline">\(STIM\)</span>) and the normalised free <span class="math inline">\(Ca^{2+}\)</span> concentration between the myofilaments itself <span class="citation" data-cites="hatze_myocybernetic_1981">(<a href="#ref-hatze_myocybernetic_1981" role="doc-biblioref">Hatze 1981</a>, pp 31-42)</span>:</p>
<p><span id="eq-gammad"><span class="math display">\[
\dot{\gamma}=
  \begin{cases}
    \frac{STIM \cdot (1 - \gamma_0) - \gamma + \gamma_0}{\tau_{act}}, &amp; \text{if} \ STIM \ge \gamma \\
    \frac{STIM \cdot (1 - \gamma_0) - \gamma + \gamma_0}{\tau_{deact}}, &amp; \text{otherwise}
  \end{cases}
\tag{7}\]</span></span></p>
<p><span class="math inline">\(\gamma_0\)</span> denotes the minimum value of <span class="math inline">\(\gamma\)</span> and had a arbitrary small value such that <a href="#eq-activestate" class="quarto-xref">Equation&nbsp;8</a> was always solvable. <span class="math inline">\(\tau_{act}\)</span> and <span class="math inline">\(\tau_{deact}\)</span> are both time constants of the first-order differential equations describing the calcium dynamics. The second step involved the relation between active state (<span class="math inline">\(q\)</span>) and <span class="math inline">\(\gamma\)</span>, to which we refer as the <span class="math inline">\(q-[Ca^{2+}]\)</span> relation. It is well-known that muscle fibres become increasingly sensitive to <span class="math inline">\([Ca^{2+}]\)</span> as their length increases <span class="citation" data-cites="kistemaker_length-dependent_2005 rack_effects_1969 stephenson_effects_1982">(<a href="#ref-kistemaker_length-dependent_2005" role="doc-biblioref">Kistemaker, Van Soest, and Bobbert 2005</a>; <a href="#ref-rack_effects_1969" role="doc-biblioref">Rack and Westbury 1969</a>; <a href="#ref-stephenson_effects_1982" role="doc-biblioref">Stephenson and Williams 1982</a>)</span>. Consequently, <span class="math inline">\(q\)</span> was also made dependent on <span class="math inline">\(L_{CE}^{rel}\)</span>. The <span class="math inline">\(q-[Ca^{2+}]\)</span> relation was modelled according to <span class="citation" data-cites="hatze_myocybernetic_1981">Hatze (<a href="#ref-hatze_myocybernetic_1981" role="doc-biblioref">1981</a>, pp 31-42)</span>, but was mathematically reformulated to ensure that the parameter values were physiologically meaningful and to facilitate its application for Optimal Control <span class="citation" data-cites="kistemaker_limiting_2023">(e.g., <a href="#ref-kistemaker_limiting_2023" role="doc-biblioref">Kistemaker et al. 2023</a>)</span>:</p>
<p><span id="eq-activestate"><span class="math display">\[
  \begin{aligned}
    q &amp;= q_0 + \frac{1-q_0}{1+(kCa \cdot \gamma)^{A_{act}} \cdot e^{a_{act} \cdot B_{act}}}
    \\[0.5em]
      \text{with} \quad A_{act} &amp;= \log_{10}(e^{-a_{act}}) \\
      \text{and} \quad B_{act} &amp;= b_{act,1} + b_{act,2} \cdot L_{CE}^{rel} + b_{act,3} \cdot {L_{CE}^{rel}}^2
  \end{aligned}
\tag{8}\]</span></span></p>
<p><span class="math inline">\(kCa\)</span> relates <span class="math inline">\(\gamma\)</span> to the actual <span class="math inline">\(Ca^{2+}\)</span> concentration between the myofilaments and <span class="math inline">\(B_{act}\)</span> denotes the <span class="math inline">\(pCa^{2+}\)</span> level at which <span class="math inline">\(q=0.5\)</span>. <span class="math inline">\(B_{act}\)</span> depended on <span class="math inline">\(L_{CE}^{rel}\)</span> (<a href="#eq-activestate" class="quarto-xref">Equation&nbsp;8</a>), while parameter <span class="math inline">\(a_{act}\)</span> determined the steepness of the relation between <span class="math inline">\(\gamma\)</span> and <span class="math inline">\(q\)</span>.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-m-hill-model" class="cell" data-execution_count="3">
<div class="cell-output cell-output-display">
<div id="fig-m-hill-model" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-m-hill-model-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-m-hill-model-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-m-hill-model-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Aspects of the Hill-type MTC model used, illustrated at the top. <span class="math inline">\(L_{CE}\)</span>, <span class="math inline">\(L_{PEE}\)</span> and <span class="math inline">\(L_{SEE}\)</span> denote the CE, parallel elastic element (SEE) and serial elastic element (SEE) length. CE represents the contractile part of the muscle fibres, while PEE and SEE represent all elastic tissue in parallel or in series, respectively, with CE. In the Hill-type MTC model, CE force depends on active state (A), CE length (B) and CE velocity (C). The effect of CE stimulation on active state (<span class="math inline">\(q\)</span>) is illustrated in D. A) The relationship between normalised free <span class="math inline">\(Ca^{2+}\)</span> concentration between the myofilaments (<span class="math inline">\(\gamma\)</span>) and <span class="math inline">\(q\)</span>. <span class="math inline">\(q\)</span> also depends on relative CE length (<span class="math inline">\(L_{CE}^{rel}\)</span>). B) The product of <span class="math inline">\(q\)</span> and the normalised active CE force-length relationship for different values of <span class="math inline">\(\gamma\)</span>. C) The CE force-velocity relationship for different values of <span class="math inline">\(q\)</span>. D) <span class="math inline">\(q\)</span> over time before, during and after CE stimulation for <span class="math inline">\(L_{CE}^{rel}=1\)</span>. CE stimulation is maximal during the period indicated by the black bar and ‘off’ elsewhere.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-m-hill-model">Source: Figures</a></div>
</section>
</section>
<section id="simulated-and-in-situ-data" class="level3" data-number="2.2">
<h3 data-number="2.2" class="anchored" data-anchor-id="simulated-and-in-situ-data"><span class="header-section-number">2.2</span> Simulated and <em>in situ</em> data</h3>
<section id="sec-m-sim-data" class="level4" data-number="2.2.1">
<h4 data-number="2.2.1" class="anchored" data-anchor-id="sec-m-sim-data"><span class="header-section-number">2.2.1</span> Simulated data</h4>
<p>To simulate quick-release, step-ramp, and isometric experiments, we used three parameter sets of a Hill-type MTC model from existing literature for three rat m. gastrocnemius medialis (GM1, GM2, and GM3; see <a href="#supptbl-overview" class="quarto-xref">Table S1</a>).</p>
<p><strong>Quick-release experiments</strong> Each quick-release experiment consisted of an isometric phase until SEE force plateaued, followed by a rapid (step) change (10 ms) in MTC length and then followed by another isometric phase (see <a href="#fig-m-protocols" class="quarto-xref">Figure&nbsp;3</a>A). CE stimulation was maximal (i.e., <span class="math inline">\(STIM = 1\)</span>) during the first isometric phase and continued at this maximal level until it was switched off shortly after the step change in MTC length occurred. The step change in MTC length was 0.2 mm and was chosen such that the resulting change in SEE force was about 5-10% of <span class="math inline">\(F_{CE}^{max}\)</span>. To prevent irreversible damage to muscle fibres, the maximal MTC length used in experiments is typically not far above the MTC length that yields maximal isometric SEE force (<span class="math inline">\(L_{MTC}^{opt}\)</span>). Accordingly, we simulated quick-release experiments at various initial MTC lengths, ranging from a very short MTC length (i.e., a length yielding very low isometric SEE force) and at every 1 mm increment up to a MTC length that was maximal 3 mm above <span class="math inline">\(L_{MTC}^{opt}\)</span>.</p>
<p><strong>Step-ramp experiments</strong> Each step-ramp experiment consisted of an isometric phase slightly above (<span class="math inline">\(\pm\)</span> 0.5 mm) <span class="math inline">\(L_{MTC}^{opt}\)</span>, followed by a rapid (step) change (10 ms) in MTC length and then a constant MTC velocity ramp (see <a href="#fig-m-protocols" class="quarto-xref">Figure&nbsp;3</a>B). CE stimulation was maximal (i.e., <span class="math inline">\(STIM = 1\)</span>) during the first isometric phase and continued at this maximal level until it was switched off 0.1 s after the step change in MTC length occurred. The change in MTC length of this step and the constant MTC velocity of the ramp were chosen such that this resulted in a more or less a constant SEE force (and therefore constant CE force) at the beginning of the ramp. We simulated 9 step-ramp experiments with different combinations of step sizes and constant MTC velocity ramps, to cover a substantial part of the concentric CE force-velocity relationship.</p>
<p><strong>Isometric experiments</strong> Isometric experiments were simulated for a combination of different MTC lengths (about 0, -2, -4 and -6 mm below <span class="math inline">\(L_{MTC}^{opt}\)</span>) and stimulation durations (35, 65 and 95 ms). CE stimulation was maximal (i.e., <span class="math inline">\(STIM = 1\)</span>) during the indicated stimulation duration and fully off elsewhere (see <a href="#fig-m-protocols" class="quarto-xref">Figure&nbsp;3</a>C). The combination of four different MTC lengths and three different stimulation durations resulted in twelve different isometric experiments.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-m-protocols" class="cell" data-execution_count="4">
<div class="cell-output cell-output-display">
<div id="fig-m-protocols" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-m-protocols-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-m-protocols-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-m-protocols-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Example of simulated data of quick-release (A), step-ramp (B) and isometric (C) experiments.** Top: MTC length over time. Bottom: SEE force over time. CE stimulation is maximal during the period indicated by the black bar, and ‘off’ elsewhere. For each experiment, we obtained specific datapoints of SEE force and the corresponding MTC length, which were used to estimate contraction and excitation dynamics parameter values.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-m-protocols">Source: Figures</a></div>
</section>
<section id="sec-m-insitu-data" class="level4" data-number="2.2.2">
<h4 data-number="2.2.2" class="anchored" data-anchor-id="sec-m-insitu-data"><span class="header-section-number">2.2.2</span> <em>In situ</em> data</h4>
<p>We also used data from an <em>in situ</em> experiment on isolated rat m. gastrocnemius medialis, conducted on three male Wistar rats. The experiment included quick-release, step-ramp and isometric experiments, which were similar to those described in <a href="#sec-m-sim-data" class="quarto-xref">Section&nbsp;2.2.1</a>. For each rat, we performed between 11 and 13 quick-release experiments, 10 and 12 step-ramp experiments and 12 isometric experiments. Below, we provide a brief description of the experimental procedure, which is fully detailed in <span class="citation" data-cites="reuvers_maximising_2025">(<a href="#ref-reuvers_maximising_2025" role="doc-biblioref">Reuvers et al. 2025</a>)</span>.</p>
<p>In the experiment, approved by the Committee on the Ethics of Animal Experimentation at the Vrije Universiteit (Permit Number: FBW- AVD11200202114471), rats were first anesthetised with urethane. The hindlimb was then shaved, and the overlying skin and m. biceps femoris were removed. The medial and lateral part of m. gastrocnemius were carefully separated from their surrounding tissue and exposed as much as possible. The rats were placed in the experimental setup with the hindlimb, femur and foot fully fixed. The distal end of the calcaneal tendon was attached to a servomotor (Aurora 309C, Aurora Scientific, Aurora, Canada) using Kevlar thread and aligned to ensure that m. gastrocnemius medialis pulled in its natural direction. All nerves not innervating m. gastrocnemius medialis were severed. M. gastrocnemius medialis was stimulated via a cuff-electrode placed on the sciatic nerve, with the proximal nerves crushed to prevent spinal reflexes. This experimental setup allowed for precise control of m. gastrocnemius medialis length changes and stimulation as well as accurate measurement of m. gastrocnemius medialis force.</p>
</section>
</section>
<section id="sec-m-est-procedure" class="level3" data-number="2.3">
<h3 data-number="2.3" class="anchored" data-anchor-id="sec-m-est-procedure"><span class="header-section-number">2.3</span> Parameter value estimation procedure</h3>
<p>The general procedure to estimate the parameter values involved minimising the sum of squared differences between the data and the values based on the estimated parameter values. To distinguish between these, we used subscripts: for example, the value of the SEE force before the quick-release of the data is indicated by <span class="math inline">\(F_{SEE}^{QR-,data}\)</span> while the value based on the estimated parameter values is indicated by <span class="math inline">\(F_{SEE}^{QR-,est}\)</span>.</p>
<p>The data of the quick-release experiments were used to estimate the parameter values of the CE, PEE and SEE force-length relationships. The data of the step-ramp experiments were used to estimate the parameter values of the CE force-velocity relationship. The data of the isometric experiments were used to estimate the parameter values of the excitation dynamics. Below we provide an overview on the methods used to estimate the parameter values. We also offer an open-source toolbox that automates the estimation of contraction and excitation dynamics parameter values.</p>
<section id="sec-m-estfl" class="level4" data-number="2.3.1">
<h4 data-number="2.3.1" class="anchored" data-anchor-id="sec-m-estfl"><span class="header-section-number">2.3.1</span> CE, SEE and PEE force-length parameter value estimation</h4>
<p>As explained in the introduction, different combinations of contraction dynamics parameters can yield almost identical mechanical behaviour under isometric conditions (see also <a href="#fig-i-effect-ksee" class="quarto-xref">Figure&nbsp;1</a>). For this reason, it is important to estimate SEE stiffness first, in order to discriminate between SEE stiffness on the one hand, and <span class="math inline">\(L_{CE}^{opt}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span> on the other. In our approach, the parameter that scales SEE stiffness (<span class="math inline">\(k_{SEE}\)</span>) is therefore estimated first. This is followed by the estimation of the PEE parameters, as <span class="math inline">\(k_{SEE}\)</span> is also required for their estimation. Finally, the parameters <span class="math inline">\(F_{CE}^{max}\)</span>, <span class="math inline">\(L_{CE}^{opt}\)</span>, and <span class="math inline">\(L_{SEE}^0\)</span> are estimated.</p>
<p><strong>Estimation of SEE stiffness.</strong> The first step was to estimate SEE stiffness. The model SEE force depends on the parameter values of <span class="math inline">\(k_{SEE}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span>, and obviously on SEE length. The problem, however, is that SEE length is unknown in experiments. This makes it challenging to estimate the parameter values concerning the SEE force-length relationship. Fortunately, quick-release experiments provide a way out. During quick-release experiments with a servomotor, the motor quickly shortens MTC length such that there is a rapid decline in SEE force (the ‘quick-release’). Due to the shortness of this timeframe, CE shortening is minimal such that <em>almost all</em> MTC shortening is taken up by SEE. Often, in experimental studies, it is assumed that <em>all</em> MTC shortening can be attributed to SEE shortening. In reality, this assumption leads to an overestimation of SEE shortening, as CE also shortens during this short timeframe. We introduced a method to correct for this overestimation of SEE shortening (see <a href="#sec-m-im" class="quarto-xref">Section&nbsp;2.3.4</a>) and estimated the parameter values considering both methods: without and with correcting for CE shortening during the quick-release.</p>
<p>Obtaining the SEE length change due to the quick-release (<span class="math inline">\(\Delta L_{SEE}^{QR}\)</span>) is a crucial step. This is because SEE length after the quick-release (<span class="math inline">\(L_{SEE}^{QR+}\)</span>) can then be expressed as SEE length before the quick-release (<span class="math inline">\(L_{SEE}^{QR-}\)</span>) plus the change in SEE length due to the quick-release: <span class="math inline">\(L_{SEE}^{QR+} = L_{SEE}^{QR-} + \Delta L_{SEE}^{QR}\)</span>. This allowed us to rewrite <a href="#eq-fee" class="quarto-xref">Equation&nbsp;2</a> to estimate SEE force immediately before (<span class="math inline">\(F_{SEE}^{QR-,est}\)</span>) and after (<span class="math inline">\(F_{SEE}^{QR+,est}\)</span>) the quick-release. This yielded:</p>
<p><span id="eq-fseefit1"><span class="math display">\[
  \begin{split}
    F_{SEE}^{QR-,est}     &amp;= k_{SEE} \cdot (L_{SEE}^{QR-} - L_{SEE}^0)^2 \\
    F_{SEE}^{QR+,est}     &amp;= k_{SEE} \cdot (L_{SEE}^{QR-} + \Delta L_{SEE}^{QR} - L_{SEE}^0)^2
  \end{split}  
\tag{9}\]</span></span></p>
<p>Now, there are two unknown parameters (i.e., <span class="math inline">\(k_{SEE}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span>), while also SEE length before the quick-release is also unknown for each quick-release experiment. Consequently, there are more unknowns than equations. To address this issue, we replaced <span class="math inline">\(L_{SEE}^{QR-} - L_{SEE}^0\)</span> with a temporary parameter <span class="math inline">\(c_{SEE}\)</span>. This yielded:</p>
<p><span id="eq-fseefit2"><span class="math display">\[
  \begin{split}
    F_{SEE}^{QR-,est}     &amp;= k_{SEE} \cdot (c_{SEE})^2 \\
    F_{SEE}^{QR+,est}     &amp;= k_{SEE} \cdot (c_{SEE} + \Delta L_{SEE}^{QR})^2
  \end{split}  
\tag{10}\]</span></span></p>
<p>As <span class="math inline">\(k_{SEE}\)</span> scales SEE stiffness, its value is constant across all quick-release experiments. In contrast, <span class="math inline">\(c_{SEE}\)</span> depends on the parameter <span class="math inline">\(L_{SEE}^0\)</span> and SEE length before the quick-release, which differs among each quick-release experiment. Consequently, <span class="math inline">\(c_{SEE}\)</span> should be determined for each quick-release experiment individually. Visually, <span class="math inline">\(c_{SEE}\)</span> determines the shift along the x-axis on the SEE force-length relationship, while <span class="math inline">\(k_{SEE}\)</span> scales SEE stiffness and consequently SEE force (see <a href="#fig-m-ksee" class="quarto-xref">Figure&nbsp;4</a>A). The value of <span class="math inline">\(k_{SEE}\)</span> (and <span class="math inline">\(c_{SEE}\)</span> for each quick-release experiment) were found by minimising the sum of squared differences between the data and estimated SEE force before (<span class="math inline">\(F_{SEE}^{QR-}\)</span>) and after (<span class="math inline">\(F_{SEE}^{QR+}\)</span>) the quick-release.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-m-ksee" class="cell" data-execution_count="5">
<div class="cell-output cell-output-display">
<div id="fig-m-ksee" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-m-ksee-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-m-ksee-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-m-ksee-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Graphical illustration of SEE stiffness parameter estimation. For each quick-release experiment, SEE force before (<span class="math inline">\(F_{SEE}^{QR-,data}\)</span>) and after (<span class="math inline">\(F_{SEE}^{QR+,data}\)</span>) the quick-release was obtained from the data, as well as the corresponding decrease in SEE length (<span class="math inline">\(\Delta L_{SEE}^{QR}\)</span>). Since SEE length prior to the quick-release is typically unknown, a temporary parameter (<span class="math inline">\(c_{SEE}\)</span>) was introduced for each experiment to represent the difference between SEE length before the quick-release and SEE slack length. A) A single quick-release experiment yields two unknowns: <span class="math inline">\(c_{SEE}\)</span> and a parameter that scales SEE stiffness (<span class="math inline">\(k_{SEE}\)</span>). B) Running multiple quick-release experiments yields n+1 unknowns: n values of <span class="math inline">\(c_{SEE}\)</span> (one for each experiment) and the parameter scaling SEE stiffness. Here, two quick-release experiments are illustrated (one indicated with black dots, the other with grey dots), while the estimated SEE force-length relationship is depicted with the black line.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-m-ksee">Source: Figures</a></div>
<p><strong>Estimation of PEE parameter values.</strong> The second step was to estimate PEE stiffness. The model PEE force depends on the parameter values of <span class="math inline">\(k_{PEE}\)</span> and <span class="math inline">\(L_{PEE}^0\)</span>, and obviously on PEE length. In experiments, SEE force is measured, while PEE force is required to estimate PEE stiffness. As such, experimental data should be used in which CE force is negligible such that PEE force is approximately equal to SEE force. At the beginning of the quick-release experiments, the active state is so low that CE force is negligible. We selected an interval of 10 ms in which the SEE force was minimal (<span class="math inline">\(F_{SEE}^{min}\)</span>, orange dot in <a href="#fig-m-protocols" class="quarto-xref">Figure&nbsp;3</a>A).</p>
<p>In experiments, PEE length is unknown. This problem can be addressed as follows. First, PEE length equals MTC length minus SEE length: <span class="math inline">\(L_{PEE} = L_{MTC} - L_{SEE}\)</span> (<a href="#eq-lcelseelmtc" class="quarto-xref">Equation&nbsp;1</a>). Second, SEE length is the sum of SEE slack length and SEE elongation (<span class="math inline">\(E_{SEE}\)</span>: <span class="math inline">\(L_{SEE} = L_{SEE}^0 + E_{SEE}\)</span>). This allowed us to rewrite <a href="#eq-fee" class="quarto-xref">Equation&nbsp;2</a> to estimate PEE force (<span class="math inline">\(F_{PEE}^{est}\)</span>), yielding:</p>
<p><span id="eq-fpeefit1"><span class="math display">\[
F_{PEE}^{est} = k_{PEE} \cdot (L_{MTC} -  - L_{SEE}^0 - E_{SEE} - L_{PEE}^0)^2
\tag{11}\]</span></span></p>
<p>Now, there are two unknown parameter values (i.e., <span class="math inline">\(k_{PEE}\)</span> and <span class="math inline">\(L_{PEE}^0\)</span>), while also SEE elongation is unknown for every quick-release experiment. Consequently, there are more unknowns than equations. To address this, we replaced <span class="math inline">\(L_{PEE}^0 + L_{SEE}^0\)</span> with a temporary parameter <span class="math inline">\(c_{PEE}\)</span>. Subsequently, SEE elongation was computed as <span class="math inline">\(\sqrt{\frac{F_{SEE}^{MIN,data}}{k_{SEE}}}\)</span>, given the estimated SEE stiffness scaling factor in the previous step. This yielded:</p>
<p><span id="eq-fpeefit2"><span class="math display">\[
F_{PEE}^{est} = k_{PEE} \cdot (L_{MTC} - \sqrt{\frac{F_{SEE}^{MIN,data}}{k_{SEE}}} - c_{PEE})^2
\tag{12}\]</span></span></p>
<p>As <span class="math inline">\(k_{PEE}\)</span> only scales PEE stiffness, its value is constant across all quick-release experiments. Similarly, <span class="math inline">\(c_{PEE}\)</span> is the sum of <span class="math inline">\(L_{SEE}^0\)</span> and <span class="math inline">\(L_{PEE}^0\)</span> and should therefore be constant for each quick-release experiment. The values of <span class="math inline">\(k_{PEE}\)</span> and <span class="math inline">\(c_{PEE}\)</span> were then computed by minimising the sum of squared differences between <span class="math inline">\(F_{PEE}^{est}\)</span> and <span class="math inline">\(F_{SEE}^{min,data}\)</span>.</p>
<p><strong>Estimation of <span class="math inline">\(F_{CE}^{max}\)</span>, <span class="math inline">\(L_{CE}^{opt}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span>.</strong> The third step was to estimate <span class="math inline">\(F_{CE}^{max}\)</span>, <span class="math inline">\(L_{CE}^{opt}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span>. Before the quick-release, MTC is isometrically delivering force. We used the SEE force at this instant (<span class="math inline">\(F_{SEE}^{QR-,data}\)</span>, black dot in <a href="#fig-m-protocols" class="quarto-xref">Figure&nbsp;3</a>A) and the corresponding MTC length to obtain the MTC force-length relationship from the data. We then estimated maximal isometric CE force (<span class="math inline">\(F_{CE}^{max}\)</span>), CE optimum length (<span class="math inline">\(L_{CE}^{opt}\)</span>) and SEE slack length (<span class="math inline">\(L_{SEE}^0\)</span>) by minimising the sum of squared differences between the data and estimated MTC force-length relationship. Lastly, <span class="math inline">\(L_{PEE}^0\)</span> was computed by subtracting <span class="math inline">\(L_{SEE}^0\)</span> from <span class="math inline">\(c_{PEE}\)</span>.</p>
</section>
<section id="sec-m-estfv" class="level4" data-number="2.3.2">
<h4 data-number="2.3.2" class="anchored" data-anchor-id="sec-m-estfv"><span class="header-section-number">2.3.2</span> CE force-velocity parameter value estimation</h4>
<p>The values of the CE force–velocity relationship parameters <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span> were estimated from data obtained during the plateau phase of SEE force in step-ramp experiments. This approach was chosen because SEE length is constant when SEE force is constant. Consequently, CE velocity equals MTC velocity under these conditions. Following this argument, we first identified a 10 ms interval in which SEE force changed the least. Second, we derived CE length and CE force as functions of time using <a href="#eq-lcelseelmtc" class="quarto-xref">Equation&nbsp;1</a> and <a href="#eq-fee" class="quarto-xref">Equation&nbsp;2</a>. Third, we calculated the CE velocity as the time-derivative of CE length. Finally, we averaged CE force and CE velocity over the 10 ms interval. This procedure yielded the data of the CE force-velocity relationship.</p>
<p>Now, the model CE force-velocity relationship can be fit to that of the data to obtain value of <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span>. However, employing a method that simply minimises the sum of ordinary least squares differences would not be appropriate because in experiments there is uncertainty in both measured CE force and CE velocity. Instead, we used a total least squares method that minimised the distance between the modelled CE force (<span class="math inline">\(F_{CE}^{est}\)</span>) and CE velocity (<span class="math inline">\(v_{CE}^{est}\)</span>) (now called: model values) and those of the data (see also <a href="#fig-m-fv" class="quarto-xref">Figure&nbsp;5</a>). To do this, we had to find the nearest model values to the CE force of the data (<span class="math inline">\(F_{CE}^{data}\)</span>) and CE velocity of the data (<span class="math inline">\(v_{CE}^{data}\)</span>) (now called: datapoints). We used the following observations: 1) the model value has to satisfy <a href="#eq-fvcon" class="quarto-xref">Equation&nbsp;4</a> and 2) the derivative of <span class="math inline">\(\frac{dF_{CE}}{dv_{CE}}\)</span> of the model value should be perpendicular to the line from the datapoint to the model value (see <a href="#fig-m-fv" class="quarto-xref">Figure&nbsp;5</a>). Hence, the model value could be found by solving <a href="#eq-fvcon" class="quarto-xref">Equation&nbsp;4</a> and <a href="#eq-fvperpline" class="quarto-xref">Equation&nbsp;13</a>:</p>
<p><span id="eq-fvperpline"><span class="math display">\[
  \frac{dv_{CE}^{est}}{dF_{CE}^{est}} \cdot (v_{CE}^{est} - v_{CE}^{data}) = F_{CE}^{data} - F_{CE}^{est}
\tag{13}\]</span></span></p>
<p>The following cost-function was then minimised to find the parameter values of <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span>:</p>
<p><span class="math display">\[
  J = \sum_{i=1}^{n} {\biggl(\frac{F_{CE}^{data} - F_{CE}^{est}}{c_1}\biggr)}^2 + {\biggl(\frac{v_{CE}^{data} - v_{CE}^{est}}{c_2}\biggr)}^2
\]</span></p>
<p><span class="math inline">\(c_1\)</span> and <span class="math inline">\(c_2\)</span> denote scaling factors such that both terms of the cost-function are more or less equally weighted, which was done by setting <span class="math inline">\(c_1\)</span> equal to the maximal range in CE force of the data and setting <span class="math inline">\(c_2\)</span> equal to the maximal range of CE velocity of the data.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-m-fv" class="cell" data-execution_count="6">
<div class="cell-output cell-output-display">
<div id="fig-m-fv" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-m-fv-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-m-fv-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-m-fv-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: Graphical illustration of the CE force-velocity parameter estimation using total least squares. For each step-ramp experiment, CE force and CE velocity was obtained during the interval in which SEE force changed the least. The nearest point on the CE force-velocity relationship was identified based on two criteria: it should satisfy <a href="#eq-fvcon" class="quarto-xref">Equation&nbsp;4</a>, and the line from the datapoint to the CE force-velocity relationship should be perpendicular the CE force-velocity relationship. This was done for all step-ramp experiments (other datapoints are depicted in grey), while the estimated CE force-velocity relationshp is depicted with the black line.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-m-fv">Source: Figures</a></div>
</section>
<section id="excitation-dynamics-parameter-value-estimation" class="level4" data-number="2.3.3">
<h4 data-number="2.3.3" class="anchored" data-anchor-id="excitation-dynamics-parameter-value-estimation"><span class="header-section-number">2.3.3</span> Excitation dynamics parameter value estimation</h4>
<p>We estimated the parameters values of <span class="math inline">\(\tau_{act}\)</span> and <span class="math inline">\(\tau_{deact}\)</span> based on isometric experiments. The choice to estimate only these parameters of the excitation dynamics was made because it is generally challenging to discriminate between parameter values of both the calcium dynamics (<a href="#eq-gammad" class="quarto-xref">Equation&nbsp;7</a>) and the <span class="math inline">\(q-[Ca^{2+}]\)</span> relation (<a href="#eq-activestate" class="quarto-xref">Equation&nbsp;8</a>) based on mechanical measurements outlined above. The parameter values of the <span class="math inline">\(q-[Ca^{2+}]\)</span> relation were set to those used in simulating the isometric experiments. Consequently, these parameters matched their actual values for the simulated data, but obviously were suboptimal for the <em>in situ</em> data. To estimate the parameter values of <span class="math inline">\(\tau_{act}\)</span> and <span class="math inline">\(\tau_{deact}\)</span>, we minimised the sum of squared differences between the SEE force of the data and the estimated SEE force based on the parameter values. For this, an interval of the data was used starting from maximal CE stimulation to 0.1 s after CE stimulation ceased off.</p>
</section>
<section id="sec-m-im" class="level4" data-number="2.3.4">
<h4 data-number="2.3.4" class="anchored" data-anchor-id="sec-m-im"><span class="header-section-number">2.3.4</span> Parameter estimation: traditional method &amp; improved method</h4>
<p>To estimate the SEE stiffness it is generally assumed that the quick-release is so fast that all MTC shortening is taken up by SEE. As A.V. Hill already acknowledged in <span class="citation" data-cites="hill_series_1950">(<a href="#ref-hill_series_1950" role="doc-biblioref">1950</a>)</span>, this is not the case in reality as CE also shortens. Although this CE shortening might be small, SEE is in general stiff and therefore small errors in SEE length may result in large errors in predicted SEE forces and thus in estimated SEE stiffness. We used two different methods to estimate the parameter values: 1) the traditional method and the improved method. In the traditional method, we followed the procedure outlined in section <a href="#sec-m-estfl" class="quarto-xref">Section&nbsp;2.3.1</a> and <a href="#sec-m-estfv" class="quarto-xref">Section&nbsp;2.3.2</a>. In the improved method, we also followed the procedure outlined in <a href="#sec-m-estfl" class="quarto-xref">Section&nbsp;2.3.1</a> and <a href="#sec-m-estfv" class="quarto-xref">Section&nbsp;2.3.2</a> after which we incorporated an additional procedure to correct for CE shortening due to the quick-release. This additional procedure was as follows: 1) We calculated CE length just before the (step) change in MTC length occurred using <a href="#eq-lcelseelmtc" class="quarto-xref">Equation&nbsp;1</a>, <a href="#eq-fee" class="quarto-xref">2</a>, <a href="#eq-forcelength" class="quarto-xref">3</a> and <a href="#eq-activestate" class="quarto-xref">8</a>, assuming that CE velocity was 0 and that <span class="math inline">\(\gamma\)</span> equalled 1. 2) We performed a (short) simulation from the time just before the (step) change in MTC length to just after the (step) change in MTC length. 3) We computed the change in CE length over this time interval as the average CE length slightly before and slightly after the (step) change in MTC length. 4) We subtracted the change in CE length from the change in MTC length to obtain the change in SEE length. In addition to correcting for CE length change during the quick-release, we incorporated a step to obtain better estimates of the actual PEE force at the time instance of minimum SEE force (<span class="math inline">\(F_{SEE}^{min,data}\)</span>; see <a href="#fig-m-protocols" class="quarto-xref">Figure&nbsp;3</a>). We computed CE force at this time instance using <a href="#eq-forcelength" class="quarto-xref">Equation&nbsp;3</a> and <a href="#eq-activestate" class="quarto-xref">8</a>, assuming that <span class="math inline">\(\gamma\)</span> equalled its minimum value. This CE force was then subtracted from <span class="math inline">\(F_{SEE}^{min,data}\)</span>.</p>
<p>The improved method leads to a different (i.e., higher) SEE stiffness, which affect the estimation of the parameter values of the CE, PEE and SEE force-length relationships (see <a href="#sec-m-estfl" class="quarto-xref">Section&nbsp;2.3.1</a>). These estimated parameter values were then used to again estimate the parameter values of the CE force-velocity relationship (see section <a href="#sec-m-estfv" class="quarto-xref">Section&nbsp;2.3.2</a>). As the parameter values of the CE force-velocity relationship slightly changed, the estimate of CE length change due to the quick-release also changed and therefore estimated SEE stiffness changed. We found that this process always converged to a stable solution and therefore we used an iterative process until the change in all parameter values was less than 0.1% (see <a href="#suppfig-flowchart" class="quarto-xref">Figure S2</a>).</p>
</section>
</section>
<section id="sec-m-sa" class="level3" data-number="2.4">
<h3 data-number="2.4" class="anchored" data-anchor-id="sec-m-sa"><span class="header-section-number">2.4</span> Sensitivity analysis</h3>
<section id="monte-carlo-simulations" class="level4" data-number="2.4.1">
<h4 data-number="2.4.1" class="anchored" data-anchor-id="monte-carlo-simulations"><span class="header-section-number">2.4.1</span> Monte Carlo simulations</h4>
<p>In experiments on isolated MTC, the MTC force-length relationship can shift due to irreversible damage of SEE <span class="citation" data-cites="aubert_tension-length_1951">(<a href="#ref-aubert_tension-length_1951" role="doc-biblioref">Aubert, Roquet, and Van der Elst 1951</a>)</span>. We observed this phenomenon in an experiment involving isolated rat m. gastrocnemius medialis, where the MTC force-length relationship shifted by about 1 mm (~8% <span class="math inline">\(L_{CE}^{opt}\)</span>) over a period of approximately 8 hours (Reuvers et al., unpublished observation). To assess the influence of these shifts on the parameter value estimation, we performed Monte Carlo simulations. We assumed that the observed shifts of the MTC force-length relationship were caused solely by a decrease in SEE stiffness. Accordingly, we decreased SEE stiffness to cause random shifts of the MTC force-length relationship between 0 and 1 mm. This way, we simulated data as if they were collected at random intervals throughout an 8-hour period. Importantly, changes in SEE stiffness affect the step size and constant MTC velocity ramp required for a more or less a constant SEE force at the beginning of the ramp in step-ramp experiments. Therefore, we adjusted the step size and constant MTC velocity ramp for each simulated step-ramp trial individually. Parameter values of each MTC were then estimated using the ‘perturbed’ data using the improved method (see <a href="#sec-m-im" class="quarto-xref">Section&nbsp;2.3.4</a>). This process was repeated 50 times for each MTC, allowing us to evaluate both the mean and the spread of the estimated parameter values.</p>
</section>
<section id="interdependency-of-parameter-values" class="level4" data-number="2.4.2">
<h4 data-number="2.4.2" class="anchored" data-anchor-id="interdependency-of-parameter-values"><span class="header-section-number">2.4.2</span> Interdependency of parameter values</h4>
<p>The parameter estimation procedure detailed above follows a fixed order in which certain parameters are estimated before others. For instance, <span class="math inline">\(k_{SEE}\)</span> is estimated first and therefore affects the estimation of <span class="math inline">\(L_{SEE}^0\)</span> and <span class="math inline">\(L_{CE}^{opt}\)</span>. This sequence creates an interdependency among the parameter values, which has been identified as a potential cause for substantial variation in parameter values among individuals, even within the same muscle type <span class="citation" data-cites="lemaire_comparison_2016">(<a href="#ref-lemaire_comparison_2016" role="doc-biblioref">Lemaire et al. 2016</a>)</span>. To investigate this interdependency, we systematically changed parameter values of our model. Specifically, we increased and decreased each parameter value by 5% relative to the values obtained with the improved method. After this, we re-estimated all other parameter values. This approach allowed us to assess the influence of changes in one parameter on the estimation of others.</p>
</section>
</section>
</section>
<section id="results" class="level2" data-number="3">
<h2 data-number="3" class="anchored" data-anchor-id="results"><span class="header-section-number">3</span> Results</h2>
<section id="evaluating-parameter-value-estimation-accuracy---simulated-data" class="level3" data-number="3.1">
<h3 data-number="3.1" class="anchored" data-anchor-id="evaluating-parameter-value-estimation-accuracy---simulated-data"><span class="header-section-number">3.1</span> Evaluating parameter value estimation accuracy - simulated data</h3>
<div id="cell-3" class="cell" data-execution_count="1">
<details class="code-fold">
<summary>Code</summary>
<div class="code-copy-outer-scaffold"><div class="sourceCode cell-code" id="cb1"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Load packages</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os, pickle, sys</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> numpy <span class="im">as</span> np</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Set directories</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>cwd <span class="op">=</span> Path.cwd()</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a>baseDir <span class="op">=</span> cwd.parent</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>dataDir <span class="op">=</span> baseDir <span class="op">/</span> <span class="st">'data'</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>funcDir <span class="op">=</span> baseDir <span class="op">/</span> <span class="st">'analysis'</span> <span class="op">/</span> <span class="st">'functions'</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>sys.path.append(<span class="bu">str</span>(funcDir))</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Custom functions</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> hillmodel, stats</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Extract parameters</span></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>muscles <span class="op">=</span> [<span class="st">'GMs1'</span>,<span class="st">'GMs2'</span>,<span class="st">'GMs3'</span>]</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a>param_keys <span class="op">=</span> [<span class="st">'a'</span>, <span class="st">'b'</span>, <span class="st">'kpee'</span>, <span class="st">'ksee'</span>, <span class="st">'fmax'</span>, <span class="st">'lce_opt'</span>, <span class="st">'lpee0'</span>, <span class="st">'lsee0'</span>, <span class="st">'tact'</span>, <span class="st">'tdeact'</span>]</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>orParms, tmParms, imParms <span class="op">=</span> [], [], []</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> mus <span class="kw">in</span> muscles:   </span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a>    orPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_OR.pkl'</span>), <span class="st">'rb'</span>))</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>    tmPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>]</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>    imPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_IM.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>]</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a>    parms <span class="op">=</span> [orPar[k] <span class="cf">for</span> k <span class="kw">in</span> param_keys]</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a>    orParms.append(parms)</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a>    parms <span class="op">=</span> [tmPar[k] <span class="cf">for</span> k <span class="kw">in</span> param_keys]</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a>    tmParms.append(parms)</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>    parms <span class="op">=</span> [imPar[k] <span class="cf">for</span> k <span class="kw">in</span> param_keys]</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a>    imParms.append(parms)</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a><span class="co"># Store in pandas dataframe</span></span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a>df_or <span class="op">=</span> pd.DataFrame(<span class="bu">list</span>(<span class="bu">zip</span>(<span class="op">*</span>orParms)), columns<span class="op">=</span>[<span class="st">'GM1'</span>, <span class="st">'GM2'</span>, <span class="st">'GM3'</span>], index<span class="op">=</span>param_keys) <span class="co"># df with actual/real parameter values</span></span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>df_tm <span class="op">=</span> pd.DataFrame(<span class="bu">list</span>(<span class="bu">zip</span>(<span class="op">*</span>tmParms)), columns<span class="op">=</span>[<span class="st">'GM1'</span>, <span class="st">'GM2'</span>, <span class="st">'GM3'</span>], index<span class="op">=</span>param_keys) <span class="co"># df with estimated parameter values using the traditional method</span></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a>df_im <span class="op">=</span> pd.DataFrame(<span class="bu">list</span>(<span class="bu">zip</span>(<span class="op">*</span>imParms)), columns<span class="op">=</span>[<span class="st">'GM1'</span>, <span class="st">'GM2'</span>, <span class="st">'GM3'</span>], index<span class="op">=</span>param_keys) <span class="co"># df with estimated parameter values using the improved method</span></span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Compute percentage differences from real/actual and estimated parameter values</span></span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Positive: Estimated is that % higher than Real</span></span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a><span class="co"># Negative: Estimated is that % lower than Real</span></span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a>df_tm_p <span class="op">=</span> stats.pdiff(df_tm, df_or)</span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a>df_im_p <span class="op">=</span> stats.pdiff(df_im, df_or)</span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Variables related to overestimation of length changes due to QR</span></span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a>deltaLsee_tm   <span class="op">=</span> np.full((<span class="bu">len</span>(muscles),<span class="dv">10</span>), np.nan)</span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a>deltaLsee_real <span class="op">=</span> np.full((<span class="bu">len</span>(muscles),<span class="dv">10</span>), np.nan)</span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> iMus,mus <span class="kw">in</span> <span class="bu">enumerate</span>(muscles):   </span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a>    orPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_OR.pkl'</span>), <span class="st">'rb'</span>))</span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a>    tmPar,dataQRtm <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>:<span class="dv">2</span>]</span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a>    fseeQRpre <span class="op">=</span> dataQRtm[<span class="st">'fseeQRpre'</span>]</span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a>    fseeQRpst <span class="op">=</span> dataQRtm[<span class="st">'fseeQRpst'</span>]</span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a>    lseeQRpre <span class="op">=</span> (fseeQRpre<span class="op">/</span>orPar[<span class="st">'ksee'</span>])<span class="op">**</span><span class="fl">0.5</span> <span class="op">+</span> orPar[<span class="st">'lsee0'</span>]</span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a>    lseeQRpst <span class="op">=</span> (fseeQRpst<span class="op">/</span>orPar[<span class="st">'ksee'</span>])<span class="op">**</span><span class="fl">0.5</span> <span class="op">+</span> orPar[<span class="st">'lsee0'</span>]</span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Compute change in SEE length due to QR</span></span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a>    deltaLsee_tm[iMus] <span class="op">=</span> dataQRtm[<span class="st">'lseeQRpre'</span>]<span class="op">-</span>dataQRtm[<span class="st">'lseeQRpst'</span>]  <span class="co"># [m] estimated SEE length change due to QR</span></span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a>    deltaLsee_real[iMus] <span class="op">=</span> lseeQRpre<span class="op">-</span>lseeQRpst  <span class="co"># [m] real/actual SEE length change due to QR</span></span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a><span class="co"># Compute percentage difference from real to estimated SEE length change due to QR</span></span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a><span class="co"># Positive: TM is that % higher than Real</span></span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a><span class="co"># Negative: TM is that % lower than Real</span></span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a>p_deltaLsee <span class="op">=</span> stats.pdiff(deltaLsee_tm, deltaLsee_real)  <span class="co"># [%] percentage difference </span></span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a>p_deltaLseeAvg <span class="op">=</span> np.mean(p_deltaLsee,<span class="dv">1</span>)  <span class="co"># [%] average per muscle</span></span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a>p_deltaLseeStd <span class="op">=</span> np.std(p_deltaLsee,<span class="dv">1</span>)  <span class="co"># [%] std per muscle</span></span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a><span class="co"># Compute difference between real and estimated CE length change due to QR</span></span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a>deltaLce <span class="op">=</span> deltaLsee_real<span class="op">-</span>deltaLsee_tm  <span class="co"># [m]</span></span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a>deltaLceAvg <span class="op">=</span> np.mean(deltaLce,<span class="dv">1</span>)  <span class="co"># [m] average per muscle</span></span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a>deltaLceStd <span class="op">=</span> np.std(deltaLce,<span class="dv">1</span>)  <span class="co"># [m] std per muscle</span></span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.1.1: Traditional vs. Improved method</span></span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a><span class="co"># Extract all contraction dynamics parameters except ksee</span></span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a>p_TM_con <span class="op">=</span> df_tm_p.loc[[<span class="st">'a'</span>,<span class="st">'b'</span>, <span class="st">'kpee'</span>, <span class="st">'fmax'</span>, <span class="st">'lce_opt'</span>, <span class="st">'lpee0'</span>, <span class="st">'lsee0'</span>]]</span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference: from real to estimated SEE length change</span></span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a>p_TM_deltaLsee_avgall <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_deltaLseeAvg<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="co">#  [%] avg. percentage difference</span></span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference over all muscles: from real to estimated SEE stiffness</span></span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a>p_TM_ksee_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_tm_p<span class="sc">.</span>loc[<span class="st">'ksee'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="co">#  [%] avg. percentage difference</span></span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference over all muscles: from real to estimated tact</span></span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a>p_TM_tact_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_tm_p<span class="sc">.</span>loc[<span class="st">'tact'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">%'</span>  <span class="co"># [%] avg. percentage difference</span></span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a><span class="co"># Max. difference over all muscles: from real to estimated tact</span></span>
<span id="cb1-86"><a href="#cb1-86" aria-hidden="true" tabindex="-1"></a>p_TM_con_max <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_TM_con<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.1f}</span><span class="ss">'</span>  <span class="co"># [%] max. percentage difference</span></span>
<span id="cb1-87"><a href="#cb1-87" aria-hidden="true" tabindex="-1"></a><span class="co"># Max. difference over all muscles: from real to estimated tact</span></span>
<span id="cb1-88"><a href="#cb1-88" aria-hidden="true" tabindex="-1"></a>p_IM_max <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_im_p<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co"># [%] max. percentage difference</span></span>
<span id="cb1-89"><a href="#cb1-89" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-90"><a href="#cb1-90" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.1.2: Estimation of SEE stiffness. </span></span>
<span id="cb1-91"><a href="#cb1-91" aria-hidden="true" tabindex="-1"></a><span class="co"># Difference between SEE elongation of estimated parameter value and real/actual value</span></span>
<span id="cb1-92"><a href="#cb1-92" aria-hidden="true" tabindex="-1"></a><span class="co"># Positive: estimated value is higher than real/actual value</span></span>
<span id="cb1-93"><a href="#cb1-93" aria-hidden="true" tabindex="-1"></a>d_esee <span class="op">=</span> (df_tm.loc[<span class="st">'fmax'</span>]<span class="op">/</span>df_tm.loc[<span class="st">'ksee'</span>])<span class="op">**</span><span class="fl">0.5</span> <span class="op">-</span> (df_or.loc[<span class="st">'fmax'</span>]<span class="op">/</span>df_or.loc[<span class="st">'ksee'</span>])<span class="op">**</span><span class="fl">0.5</span> <span class="co"># [m]</span></span>
<span id="cb1-94"><a href="#cb1-94" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-95"><a href="#cb1-95" aria-hidden="true" tabindex="-1"></a><span class="co"># Correlation coefficients</span></span>
<span id="cb1-96"><a href="#cb1-96" aria-hidden="true" tabindex="-1"></a>r_see <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-97"><a href="#cb1-97" aria-hidden="true" tabindex="-1"></a>r_pee <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-98"><a href="#cb1-98" aria-hidden="true" tabindex="-1"></a>r_mtc <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-99"><a href="#cb1-99" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> iMus,mus <span class="kw">in</span> <span class="bu">enumerate</span>(muscles):   </span>
<span id="cb1-100"><a href="#cb1-100" aria-hidden="true" tabindex="-1"></a>    parFile <span class="op">=</span> os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM.pkl'</span>)</span>
<span id="cb1-101"><a href="#cb1-101" aria-hidden="true" tabindex="-1"></a>    muspar, dataQR, dataSR, dataACTout <span class="op">=</span> pickle.load(<span class="bu">open</span>(parFile, <span class="st">'rb'</span>))</span>
<span id="cb1-102"><a href="#cb1-102" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-103"><a href="#cb1-103" aria-hidden="true" tabindex="-1"></a>    <span class="co"># SEE</span></span>
<span id="cb1-104"><a href="#cb1-104" aria-hidden="true" tabindex="-1"></a>    lseeData <span class="op">=</span> np.hstack((dataQR[<span class="st">'lseeQRpre'</span>],dataQR[<span class="st">'lseeQRpst'</span>]))</span>
<span id="cb1-105"><a href="#cb1-105" aria-hidden="true" tabindex="-1"></a>    fseeData <span class="op">=</span> np.hstack((dataQR[<span class="st">'fseeQRpre'</span>],dataQR[<span class="st">'fseeQRpst'</span>]))</span>
<span id="cb1-106"><a href="#cb1-106" aria-hidden="true" tabindex="-1"></a>    fseeMdl <span class="op">=</span> hillmodel.LEE2Force(lseeData,<span class="dv">0</span>,muspar)[<span class="dv">0</span>]</span>
<span id="cb1-107"><a href="#cb1-107" aria-hidden="true" tabindex="-1"></a>    r_see[iMus] <span class="op">=</span> np.corrcoef(fseeData, fseeMdl)[<span class="dv">0</span>,<span class="dv">1</span>]</span>
<span id="cb1-108"><a href="#cb1-108" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-109"><a href="#cb1-109" aria-hidden="true" tabindex="-1"></a>    <span class="co"># PEE</span></span>
<span id="cb1-110"><a href="#cb1-110" aria-hidden="true" tabindex="-1"></a>    fpeeMdl <span class="op">=</span> hillmodel.LEE2Force(<span class="dv">0</span>,dataQR[<span class="st">'lpeeQR'</span>],muspar)[<span class="dv">1</span>]</span>
<span id="cb1-111"><a href="#cb1-111" aria-hidden="true" tabindex="-1"></a>    r_pee[iMus] <span class="op">=</span> np.corrcoef(dataQR[<span class="st">'fpeeQR'</span>], fpeeMdl)[<span class="dv">0</span>,<span class="dv">1</span>]</span>
<span id="cb1-112"><a href="#cb1-112" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-113"><a href="#cb1-113" aria-hidden="true" tabindex="-1"></a>    <span class="co"># MTC</span></span>
<span id="cb1-114"><a href="#cb1-114" aria-hidden="true" tabindex="-1"></a>    fseeMdl <span class="op">=</span> hillmodel.ForceEQ(dataQR[<span class="st">'lmtcQRpre'</span>],<span class="dv">1</span>,muspar)[<span class="dv">0</span>]</span>
<span id="cb1-115"><a href="#cb1-115" aria-hidden="true" tabindex="-1"></a>    r_mtc[iMus] <span class="op">=</span> np.corrcoef(dataQR[<span class="st">'fseeQRpre'</span>], fseeMdl)[<span class="dv">0</span>,<span class="dv">1</span>]</span>
<span id="cb1-116"><a href="#cb1-116" aria-hidden="true" tabindex="-1"></a>r_all <span class="op">=</span> np.hstack((r_see,r_pee,r_mtc))</span>
<span id="cb1-117"><a href="#cb1-117" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-118"><a href="#cb1-118" aria-hidden="true" tabindex="-1"></a><span class="co"># Percentage difference from real to estimated parameter</span></span>
<span id="cb1-119"><a href="#cb1-119" aria-hidden="true" tabindex="-1"></a>p_TM_ksee <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> df_tm_p.loc[<span class="st">'ksee'</span>]]  <span class="co"># [%] per muscle</span></span>
<span id="cb1-120"><a href="#cb1-120" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg&amp;Std per muscle: differences from real to estimated parameters </span></span>
<span id="cb1-121"><a href="#cb1-121" aria-hidden="true" tabindex="-1"></a>p_TM_deltaLsee_avgmus <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> p_deltaLseeAvg]  <span class="co"># [%]</span></span>
<span id="cb1-122"><a href="#cb1-122" aria-hidden="true" tabindex="-1"></a>p_TM_deltaLsee_stdmus <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> p_deltaLseeStd]  <span class="co"># [%]</span></span>
<span id="cb1-123"><a href="#cb1-123" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg&amp;Std per muscle: CE shortening during QR</span></span>
<span id="cb1-124"><a href="#cb1-124" aria-hidden="true" tabindex="-1"></a>p_TM_deltaLce_avgmus <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="op">*</span><span class="fl">1e6</span><span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> deltaLceAvg]  <span class="co"># [um] avg. per muscle</span></span>
<span id="cb1-125"><a href="#cb1-125" aria-hidden="true" tabindex="-1"></a>p_TM_deltaLce_stdmus <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="op">*</span><span class="fl">1e6</span><span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> deltaLceStd]  <span class="co"># [um] std per muscle</span></span>
<span id="cb1-126"><a href="#cb1-126" aria-hidden="true" tabindex="-1"></a><span class="co"># Difference from real to estimate in SEE elongation @ fmax</span></span>
<span id="cb1-127"><a href="#cb1-127" aria-hidden="true" tabindex="-1"></a>d_TM_esee <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="op">*</span><span class="fl">1e3</span><span class="sc">:0.2f}</span><span class="ss">'</span> <span class="cf">for</span>  v <span class="kw">in</span> d_esee] <span class="co">#  [mm] </span></span>
<span id="cb1-128"><a href="#cb1-128" aria-hidden="true" tabindex="-1"></a><span class="co"># Minimum correlation coefficient</span></span>
<span id="cb1-129"><a href="#cb1-129" aria-hidden="true" tabindex="-1"></a>r_TM_min <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>np<span class="sc">.</span><span class="bu">min</span>(np.floor(r_all<span class="op">*</span><span class="dv">100</span>)<span class="op">/</span><span class="dv">100</span>)<span class="sc">:0.2f}</span><span class="ss">'</span>  <span class="co"># []</span></span>
<span id="cb1-130"><a href="#cb1-130" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-131"><a href="#cb1-131" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.1.2: Estimation of PEE parameter values</span></span>
<span id="cb1-132"><a href="#cb1-132" aria-hidden="true" tabindex="-1"></a>nPEEdata <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-133"><a href="#cb1-133" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> iMus,mus <span class="kw">in</span> <span class="bu">enumerate</span>(muscles):   </span>
<span id="cb1-134"><a href="#cb1-134" aria-hidden="true" tabindex="-1"></a>    muspar, dataQR, dataSR, dataACT <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM.pkl'</span>), <span class="st">'rb'</span>))</span>
<span id="cb1-135"><a href="#cb1-135" aria-hidden="true" tabindex="-1"></a>    lpeeQR <span class="op">=</span> dataQR[<span class="st">'lpeeQR'</span>]  <span class="co"># [m] PEE length of data</span></span>
<span id="cb1-136"><a href="#cb1-136" aria-hidden="true" tabindex="-1"></a>    fpeeQR <span class="op">=</span> dataQR[<span class="st">'fpeeQR'</span>]  <span class="co"># [N] PEE force of data</span></span>
<span id="cb1-137"><a href="#cb1-137" aria-hidden="true" tabindex="-1"></a>    fpeeModel <span class="op">=</span> hillmodel.LEE2Force(<span class="dv">0</span>,lpeeQR,muspar)[<span class="dv">1</span>]  <span class="co"># [N] PEE force estimated</span></span>
<span id="cb1-138"><a href="#cb1-138" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-139"><a href="#cb1-139" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Compute how many data points available</span></span>
<span id="cb1-140"><a href="#cb1-140" aria-hidden="true" tabindex="-1"></a>    nPEEdata[iMus] <span class="op">=</span> np.<span class="bu">sum</span>(fpeeModel<span class="op">&gt;</span><span class="dv">0</span>)</span>
<span id="cb1-141"><a href="#cb1-141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-142"><a href="#cb1-142" aria-hidden="true" tabindex="-1"></a><span class="co"># Dislay percentage differences from real to estimated parameters</span></span>
<span id="cb1-143"><a href="#cb1-143" aria-hidden="true" tabindex="-1"></a>p_TM_kpee <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.1f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> df_tm_p.loc[<span class="st">'kpee'</span>]]  <span class="co"># [%] percentage difference</span></span>
<span id="cb1-144"><a href="#cb1-144" aria-hidden="true" tabindex="-1"></a>p_TM_lpee0 <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.1f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> df_tm_p.loc[<span class="st">'lpee0'</span>]]  <span class="co"># [%] percentage difference</span></span>
<span id="cb1-145"><a href="#cb1-145" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-146"><a href="#cb1-146" aria-hidden="true" tabindex="-1"></a><span class="co"># Display amount of datapoints available</span></span>
<span id="cb1-147"><a href="#cb1-147" aria-hidden="true" tabindex="-1"></a>n_PEE_data <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> nPEEdata]  <span class="co"># [ ]</span></span>
<span id="cb1-148"><a href="#cb1-148" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-149"><a href="#cb1-149" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.1.2: Estimation of Fcemax, Lceopt, Lsee0</span></span>
<span id="cb1-150"><a href="#cb1-150" aria-hidden="true" tabindex="-1"></a><span class="co"># Dislay percentage differences from real to estimated parameters</span></span>
<span id="cb1-151"><a href="#cb1-151" aria-hidden="true" tabindex="-1"></a>p_TM_fl_max <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_tm_p<span class="sc">.</span>loc[[<span class="st">'fmax'</span>,<span class="st">'lce_opt'</span>,<span class="st">'lsee0'</span>],:]<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.1f}</span><span class="ss">'</span>  <span class="co"># [%] maximal percentage difference</span></span>
<span id="cb1-152"><a href="#cb1-152" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-153"><a href="#cb1-153" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.1.2: Estimation of CE force-velocity parameter values</span></span>
<span id="cb1-154"><a href="#cb1-154" aria-hidden="true" tabindex="-1"></a><span class="co"># Dislay percentage differences from real to estimated parameters</span></span>
<span id="cb1-155"><a href="#cb1-155" aria-hidden="true" tabindex="-1"></a>p_TM_a_max <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_tm_p<span class="sc">.</span>loc[<span class="st">'a'</span>,:]<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.1f}</span><span class="ss">'</span>  <span class="co"># [%] maximal percentage difference</span></span>
<span id="cb1-156"><a href="#cb1-156" aria-hidden="true" tabindex="-1"></a>p_TM_b_max <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_tm_p<span class="sc">.</span>loc[<span class="st">'b'</span>,:]<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.1f}</span><span class="ss">'</span>  <span class="co"># [%] maximal percentage difference</span></span>
<span id="cb1-157"><a href="#cb1-157" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-158"><a href="#cb1-158" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.1.2: Estimation of excitation dynamics parameter values</span></span>
<span id="cb1-159"><a href="#cb1-159" aria-hidden="true" tabindex="-1"></a>p_tact_QR <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-160"><a href="#cb1-160" aria-hidden="true" tabindex="-1"></a>p_tdeact_QR <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-161"><a href="#cb1-161" aria-hidden="true" tabindex="-1"></a>p_tact_SR <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-162"><a href="#cb1-162" aria-hidden="true" tabindex="-1"></a>p_tdeact_SR <span class="op">=</span> np.full(<span class="bu">len</span>(muscles), np.nan)</span>
<span id="cb1-163"><a href="#cb1-163" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> iMus, mus <span class="kw">in</span> <span class="bu">enumerate</span>(muscles):   </span>
<span id="cb1-164"><a href="#cb1-164" aria-hidden="true" tabindex="-1"></a>    orPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_OR.pkl'</span>), <span class="st">'rb'</span>))</span>
<span id="cb1-165"><a href="#cb1-165" aria-hidden="true" tabindex="-1"></a>    qrPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM_QR.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>]</span>
<span id="cb1-166"><a href="#cb1-166" aria-hidden="true" tabindex="-1"></a>    srPar <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM_SR.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>]</span>
<span id="cb1-167"><a href="#cb1-167" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb1-168"><a href="#cb1-168" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Compute percentage difference from real to estimated parameter</span></span>
<span id="cb1-169"><a href="#cb1-169" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Positive: estimated is that % higher than Real</span></span>
<span id="cb1-170"><a href="#cb1-170" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Negative: estimated is that % lower than Real</span></span>
<span id="cb1-171"><a href="#cb1-171" aria-hidden="true" tabindex="-1"></a>    p_tact_QR[iMus] <span class="op">=</span> stats.pdiff(qrPar[<span class="st">'tact'</span>],orPar[<span class="st">'tact'</span>])</span>
<span id="cb1-172"><a href="#cb1-172" aria-hidden="true" tabindex="-1"></a>    p_tact_SR[iMus] <span class="op">=</span> stats.pdiff(srPar[<span class="st">'tact'</span>],orPar[<span class="st">'tact'</span>])</span>
<span id="cb1-173"><a href="#cb1-173" aria-hidden="true" tabindex="-1"></a>    p_tdeact_QR[iMus] <span class="op">=</span> stats.pdiff(qrPar[<span class="st">'tdeact'</span>],orPar[<span class="st">'tdeact'</span>])</span>
<span id="cb1-174"><a href="#cb1-174" aria-hidden="true" tabindex="-1"></a>    p_tdeact_SR[iMus] <span class="op">=</span> stats.pdiff(srPar[<span class="st">'tdeact'</span>],orPar[<span class="st">'tdeact'</span>])</span>
<span id="cb1-175"><a href="#cb1-175" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-176"><a href="#cb1-176" aria-hidden="true" tabindex="-1"></a><span class="co"># Dislay percentage differences from real to estimated parameters by using ISOM data</span></span>
<span id="cb1-177"><a href="#cb1-177" aria-hidden="true" tabindex="-1"></a>p_TM_tact <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> df_tm_p.loc[<span class="st">'tact'</span>]]  <span class="co"># [%] percentage difference</span></span>
<span id="cb1-178"><a href="#cb1-178" aria-hidden="true" tabindex="-1"></a>p_TM_tdeact <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>v<span class="sc">:0.1f}</span><span class="ss">'</span> <span class="cf">for</span> v <span class="kw">in</span> df_tm_p.loc[<span class="st">'tdeact'</span>]]  <span class="co"># [%] percentage difference</span></span>
<span id="cb1-179"><a href="#cb1-179" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-180"><a href="#cb1-180" aria-hidden="true" tabindex="-1"></a><span class="co"># Display percentage difference from real to estimated parameters by using</span></span>
<span id="cb1-181"><a href="#cb1-181" aria-hidden="true" tabindex="-1"></a><span class="co"># either QR or SR data</span></span>
<span id="cb1-182"><a href="#cb1-182" aria-hidden="true" tabindex="-1"></a>p_tact_QR_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_tact_QR<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co">#  [%] percentage difference when using QR data</span></span>
<span id="cb1-183"><a href="#cb1-183" aria-hidden="true" tabindex="-1"></a>p_tact_SR_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_tact_SR<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co">#  [%] percentage difference when using SR data</span></span>
<span id="cb1-184"><a href="#cb1-184" aria-hidden="true" tabindex="-1"></a>p_tdeact_QR_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_tdeact_QR<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co">#  [%] percentage difference when using QR data</span></span>
<span id="cb1-185"><a href="#cb1-185" aria-hidden="true" tabindex="-1"></a>p_tdeact_SR_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_tdeact_SR<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co">#  [%] percentage difference when using SR data</span></span>
<span id="cb1-186"><a href="#cb1-186" aria-hidden="true" tabindex="-1"></a><span class="co"># For tdeact: differences are similar so show avg of the two</span></span>
<span id="cb1-187"><a href="#cb1-187" aria-hidden="true" tabindex="-1"></a>p_tdeact_QRSR <span class="op">=</span> np.concatenate([p_tdeact_QR, p_tdeact_SR])</span>
<span id="cb1-188"><a href="#cb1-188" aria-hidden="true" tabindex="-1"></a>p_tdeact_QRSR_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_tdeact_QRSR<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co">#  [%] percentage difference when using QR/SR data</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</details>
</div>
<section id="traditional-method-versus-improved-method" class="level4" data-number="3.1.1">
<h4 data-number="3.1.1" class="anchored" data-anchor-id="traditional-method-versus-improved-method"><span class="header-section-number">3.1.1</span> Traditional method versus Improved method</h4>
<p>The traditional method did not correct for CE shortening during quick-release experiments. Consequently, SEE shortening due to the quick-release was overestimated by 24% on average. This overestimation of SEE shortening caused a decrease in <span class="math inline">\(k_{SEE}\)</span> (and thus SEE stiffness) of 34% on average. The estimation of the activation time constant was most affected by the underestimated SEE stiffness, resulting in an underestimation of 19%% on average. All other estimated parameter values were within 8.7% of their actual values (<a href="#tbl-r-pdiff" class="quarto-xref">Table&nbsp;1</a>). These findings show the influence of CE shortening — even within a very brief 10 ms interval — on the parameter value estimation of both the contraction dynamics and excitation dynamics.</p>
<p>In contrast, the improved method corrected for CE shortening during the quick-release experiments. The correction substantially reduced the overestimation of SEE shortening, yielding an accurate estimate of <span class="math inline">\(k_{SEE}\)</span>. As a result, all estimated parameter values deviated by no more than 3% from their actual values (<a href="#tbl-r-pdiff" class="quarto-xref">Table&nbsp;1</a>). These results demonstrate that accounting for CE shortening - even over a brief 10 ms interval - substantially improves the accuracy of the parameter value estimation.</p>
<p>For interested readers, we discuss below the specific factors contributing to the differences between the actual contraction and excitation dynamics parameter values and those estimated based on the traditional method.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-r-protocols" class="cell" data-execution_count="7">
<div class="cell-output cell-output-display">
<div id="fig-r-protocols" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-r-protocols-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-r-protocols-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-r-protocols-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: Representative example of SEE force over time during a quick-release (A), step-ramp (B) and isometric experiment (C). The inset in (A) depicts the SEE force over time around the quick-release. The SEE force over time is depicted for four sets of parameter values: 1) the actual values (i.e., literature-obtained; black solid line), those obtained with the traditional method (TM; orange dashed line), those obtained with the improved method (IM; green dashed-dotted line) and those resulting from the Monte Carlo Simulations (MC; blue dotted line, with shaded 95% confidence interval).
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-r-protocols">Source: Figures</a></div>
<div class="quarto-embed-nb-cell">
<div class="cell" data-execution_count="20">
<div id="tbl-r-pdiff" class="cell quarto-float quarto-figure quarto-figure-center anchored" data-execution_count="20">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-r-pdiff-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Percentage differences between estimated and actual MTC parameter values.
</figcaption>
<div aria-describedby="tbl-r-pdiff-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output cell-output-display">



<meta charset="utf-8">


<div id="ddykzrorci" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>
#ddykzrorci table {
          font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
          -webkit-font-smoothing: antialiased;
          -moz-osx-font-smoothing: grayscale;
        }

#ddykzrorci thead, tbody, tfoot, tr, td, th { border-style: none !important; }
 tr { background-color: transparent !important; }
#ddykzrorci p { margin: 0 !important; padding: 0 !important; }
 #ddykzrorci .gt_table { display: table !important; border-collapse: collapse !important; line-height: normal !important; margin-left: auto !important; margin-right: auto !important; color: #333333 !important; font-size: 16px !important; font-weight: normal !important; font-style: normal !important; background-color: #FFFFFF !important; width: auto !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #A8A8A8 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #A8A8A8 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; }
 #ddykzrorci .gt_caption { padding-top: 4px !important; padding-bottom: 4px !important; }
 #ddykzrorci .gt_title { color: #333333 !important; font-size: 125% !important; font-weight: initial !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; border-bottom-color: #FFFFFF !important; border-bottom-width: 0 !important; }
 #ddykzrorci .gt_subtitle { color: #333333 !important; font-size: 85% !important; font-weight: initial !important; padding-top: 3px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; border-top-color: #FFFFFF !important; border-top-width: 0 !important; }
 #ddykzrorci .gt_heading { background-color: #FFFFFF !important; text-align: center !important; border-bottom-color: #FFFFFF !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #ddykzrorci .gt_bottom_border { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #ddykzrorci .gt_col_headings { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #ddykzrorci .gt_col_heading { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; overflow-x: hidden !important; }
 #ddykzrorci .gt_column_spanner_outer { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; padding-top: 0 !important; padding-bottom: 0 !important; padding-left: 4px !important; padding-right: 4px !important; }
 #ddykzrorci .gt_column_spanner_outer:first-child { padding-left: 0 !important; }
 #ddykzrorci .gt_column_spanner_outer:last-child { padding-right: 0 !important; }
 #ddykzrorci .gt_column_spanner { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; overflow-x: hidden !important; display: inline-block !important; width: 100% !important; }
 #ddykzrorci .gt_spanner_row { border-bottom-style: hidden !important; }
 #ddykzrorci .gt_group_heading { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; text-align: left !important; }
 #ddykzrorci .gt_empty_group_heading { padding: 0.5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: middle !important; }
 #ddykzrorci .gt_from_md> :first-child { margin-top: 0 !important; }
 #ddykzrorci .gt_from_md> :last-child { margin-bottom: 0 !important; }
 #ddykzrorci .gt_row { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; margin: 10px !important; border-top-style: solid !important; border-top-width: 1px !important; border-top-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; overflow-x: hidden !important; }
 #ddykzrorci .gt_stub { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; }
 #ddykzrorci .gt_stub_row_group { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; vertical-align: top !important; }
 #ddykzrorci .gt_row_group_first td { border-top-width: 2px !important; }
 #ddykzrorci .gt_row_group_first th { border-top-width: 2px !important; }
 #ddykzrorci .gt_striped { color: #333333 !important; background-color: #F4F4F4 !important; }
 #ddykzrorci .gt_table_body { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #ddykzrorci .gt_grand_summary_row { color: #333333 !important; background-color: #FFFFFF !important; text-transform: inherit !important; padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; }
 #ddykzrorci .gt_first_grand_summary_row_bottom { border-top-style: double !important; border-top-width: 6px !important; border-top-color: #D3D3D3 !important; }
 #ddykzrorci .gt_last_grand_summary_row_top { border-bottom-style: double !important; border-bottom-width: 6px !important; border-bottom-color: #D3D3D3 !important; }
 #ddykzrorci .gt_sourcenotes { color: #333333 !important; background-color: #FFFFFF !important; border-bottom-style: none !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; }
 #ddykzrorci .gt_sourcenote { font-size: 90% !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; }
 #ddykzrorci .gt_left { text-align: left !important; }
 #ddykzrorci .gt_center { text-align: center !important; }
 #ddykzrorci .gt_right { text-align: right !important; font-variant-numeric: tabular-nums !important; }
 #ddykzrorci .gt_font_normal { font-weight: normal !important; }
 #ddykzrorci .gt_font_bold { font-weight: bold !important; }
 #ddykzrorci .gt_font_italic { font-style: italic !important; }
 #ddykzrorci .gt_super { font-size: 65% !important; }
 #ddykzrorci .gt_footnote_marks { font-size: 75% !important; vertical-align: 0.4em !important; position: initial !important; }
 #ddykzrorci .gt_asterisk { font-size: 100% !important; vertical-align: 0 !important; }
 
</style>

<table class="gt_table do-not-create-environment cell caption-top table table-sm table-striped small" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 6%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
</colgroup>
<thead>
<tr class="gt_col_headings gt_spanner_row header">
<th rowspan="2" id="index" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"></th>
<th colspan="3" id="Traditional-method" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Traditional method</span></th>
<th colspan="3" id="Improved-method" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Improved method</span></th>
<th colspan="3" id="Monte-Carlo" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Monte Carlo</span></th>
</tr>
<tr class="gt_col_headings even">
<th id="TM_GMs1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM1</th>
<th id="TM_GMs2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM2</th>
<th id="TM_GMs3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM3</th>
<th id="IM_GMs1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM1</th>
<th id="IM_GMs2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM2</th>
<th id="IM_GMs3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM3</th>
<th id="MC_GMs1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM1</th>
<th id="MC_GMs2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM2</th>
<th id="MC_GMs3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM3</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left">$a$</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">4</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">3</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">1 ± 2</td>
<td class="gt_row gt_center">6 ± 2</td>
<td class="gt_row gt_center">1 ± 2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$b$</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">2</td>
<td class="gt_row gt_center">1</td>
<td class="gt_row gt_center">1 ± 2</td>
<td class="gt_row gt_center">5 ± 2</td>
<td class="gt_row gt_center">1 ± 2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$F_{CE}^{max}$</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">-0 ± 1</td>
<td class="gt_row gt_center">-2 ± 1</td>
<td class="gt_row gt_center">-0 ± 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$k_{PEE}$</td>
<td class="gt_row gt_center">3</td>
<td class="gt_row gt_center">-9</td>
<td class="gt_row gt_center">9</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">3 ± 17</td>
<td class="gt_row gt_center">3 ± 16</td>
<td class="gt_row gt_center">6 ± 31</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$k_{SEE}$</td>
<td class="gt_row gt_center">-39</td>
<td class="gt_row gt_center">-27</td>
<td class="gt_row gt_center">-37</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-37 ± 6</td>
<td class="gt_row gt_center">-36 ± 6</td>
<td class="gt_row gt_center">-36 ± 6</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$L_{CE}^{opt}$</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">-2</td>
<td class="gt_row gt_center">-3</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0 ± 3</td>
<td class="gt_row gt_center">0 ± 2</td>
<td class="gt_row gt_center">0 ± 3</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$L_{PEE}^0$</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0 ± 3</td>
<td class="gt_row gt_center">1 ± 2</td>
<td class="gt_row gt_center">0 ± 2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$L_{SEE}^0$</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">-0 ± 1</td>
<td class="gt_row gt_center">-0 ± 1</td>
<td class="gt_row gt_center">-0 ± 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$\tau_{act}$</td>
<td class="gt_row gt_center">-21</td>
<td class="gt_row gt_center">-16</td>
<td class="gt_row gt_center">-21</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">-1</td>
<td class="gt_row gt_center">-0</td>
<td class="gt_row gt_center">1 ± 3</td>
<td class="gt_row gt_center">-0 ± 4</td>
<td class="gt_row gt_center">1 ± 4</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$\tau_{deact}$</td>
<td class="gt_row gt_center">-5</td>
<td class="gt_row gt_center">-4</td>
<td class="gt_row gt_center">-5</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">0</td>
<td class="gt_row gt_center">0 ± 1</td>
<td class="gt_row gt_center">0 ± 1</td>
<td class="gt_row gt_center">0 ± 1</td>
</tr>
</tbody>
</table>


</div>
        


</div>
</div>
</figure>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-2" href="../tables/all_tables.html#cell-tbl-r-pdiff">Source: Tables</a></div>
</section>
<section id="understanding-errors-in-parameter-value-estimation" class="level4" data-number="3.1.2">
<h4 data-number="3.1.2" class="anchored" data-anchor-id="understanding-errors-in-parameter-value-estimation"><span class="header-section-number">3.1.2</span> Understanding errors in parameter value estimation</h4>
<p><strong>Estimation of SEE stiffness.</strong> The SEE stiffness parameter (<span class="math inline">\(k_{SEE}\)</span>) was underestimated by 39%, 27% and 37% for GM1, GM2 and GM3, respectively (see <a href="#tbl-r-pdiff" class="quarto-xref">Table&nbsp;1</a>). This overestimation directly resulted from the overestimation of SEE shortening due to the quick-release, which was overestimated by 28±2%, 17±2% and 26±2% for GM1, GM2 and GM3, respectively, averaged across all quick-release experiments. In absolute terms, CE shortened only 43±3, 29±2 and 41±2 µm over the 11 ms interval between the time points at which SEE force and SEE length were sampled (i.e., immediately before and after the quick-release). The amount of CE shortening was smallest in GM2 because GM2 was a slower muscle than GM1 and GM3. Consequently, the overestimation of SEE shortening was also smallest in GM2, which in turn led to the smallest underestimation of <span class="math inline">\(k_{SEE}\)</span>. All in all, this shows that even very small amounts of CE shortening has substantial influence on the estimation of SEE stiffness.</p>
<p>Since SEE was very stiff, the overestimation in SEE elongation at maximal isometric CE force was only 0.50, 0.32 and 0.49 mm for GM1, GM2 and GM3, respectively. This is an important finding because the SEE stiffness is the first parameter in the estimation process and therefore affects all subsequent estimated parameter values. Therefore, since the overestimation of SEE elongation (in mm) at maximal isometric CE force was only small, the impact on the estimation of the other contraction dynamics parameter values was also small.</p>
<p>Lastly, it should be noted that the correlation coefficient between the data SEE force‑length relationship and the one based from the estimated parameters is not an adequate measure of how accurately SEE stiffness is captured. Obviously, a correlation coefficient is not sensitive to the overestimation of SEE shortening due to the quick-release. Consequently, high correlation coefficients (<span class="math inline">\(R^2 &gt;\)</span> 0.99) can still be observed even when SEE stiffness is substantially underestimated. To further investigate this issue, we simulated the quick-release experiments after obtaining all contraction and excitation dynamics parameter values. The simulations clearly showed a slower rise in SEE force after the quick-release in comparison with the experimental data (<a href="#fig-r-protocols" class="quarto-xref">Figure&nbsp;6</a>A), indicating that SEE stiffness was underestimated. These results suggest that the rise in SEE force is a better measure of SEE stiffness estimation accuracy.</p>
</section>
</section>
<section id="sensitivity-analysis" class="level3" data-number="3.2">
<h3 data-number="3.2" class="anchored" data-anchor-id="sensitivity-analysis"><span class="header-section-number">3.2</span> Sensitivity analysis</h3>
<div id="cell-6" class="cell" data-execution_count="2">
<details class="code-fold">
<summary>Code</summary>
<div class="code-copy-outer-scaffold"><div class="sourceCode cell-code" id="cb2"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Load packages</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os, pickle, sys</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Set directories</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>cwd <span class="op">=</span> Path.cwd()</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>baseDir <span class="op">=</span> cwd.parent</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>dataDir <span class="op">=</span> baseDir <span class="op">/</span> <span class="st">'data'</span></span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>funcDir <span class="op">=</span> baseDir <span class="op">/</span> <span class="st">'analysis'</span> <span class="op">/</span> <span class="st">'functions'</span></span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>sys.path.append(<span class="bu">str</span>(funcDir))</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Custom functions</span></span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> stats</span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a><span class="co"># %% 3.2.1: Monte Carlo simulations</span></span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>muscles <span class="op">=</span> [<span class="st">'GMs1'</span>, <span class="st">'GMs2'</span>, <span class="st">'GMs3'</span>]</span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a><span class="co"># Parameters to track</span></span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a>params <span class="op">=</span> [<span class="st">'a'</span>, <span class="st">'b'</span>, <span class="st">'fmax'</span>, <span class="st">'kpee'</span>, <span class="st">'ksee'</span>, <span class="st">'lce_opt'</span>, <span class="st">'lpee0'</span>, <span class="st">'lsee0'</span>, <span class="st">'tact'</span>, <span class="st">'tdeact'</span>]</span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-21"><a href="#cb2-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Store Monte-Carlo results</span></span>
<span id="cb2-22"><a href="#cb2-22" aria-hidden="true" tabindex="-1"></a>results <span class="op">=</span> []</span>
<span id="cb2-23"><a href="#cb2-23" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> mus <span class="kw">in</span> muscles:</span>
<span id="cb2-24"><a href="#cb2-24" aria-hidden="true" tabindex="-1"></a>    par_file <span class="op">=</span> os.path.join(dataDir, mus, <span class="st">'parameters'</span>, <span class="ss">f'</span><span class="sc">{</span>mus<span class="sc">}</span><span class="ss">_OR.pkl'</span>)</span>
<span id="cb2-25"><a href="#cb2-25" aria-hidden="true" tabindex="-1"></a>    or_par <span class="op">=</span> pickle.load(<span class="bu">open</span>(par_file, <span class="st">'rb'</span>))</span>
<span id="cb2-26"><a href="#cb2-26" aria-hidden="true" tabindex="-1"></a>    eseeMaxOR <span class="op">=</span> (or_par[<span class="st">'fmax'</span>]<span class="op">/</span>or_par[<span class="st">'ksee'</span>])<span class="op">**</span><span class="fl">0.5</span></span>
<span id="cb2-27"><a href="#cb2-27" aria-hidden="true" tabindex="-1"></a>        </span>
<span id="cb2-28"><a href="#cb2-28" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb2-29"><a href="#cb2-29" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> iMC <span class="kw">in</span> <span class="bu">range</span>(<span class="dv">1</span>, <span class="dv">51</span>):</span>
<span id="cb2-30"><a href="#cb2-30" aria-hidden="true" tabindex="-1"></a>        mc_file <span class="op">=</span> os.path.join(dataDir, mus, <span class="st">'parameters'</span>,<span class="st">'mc'</span>, <span class="ss">f'</span><span class="sc">{</span>mus<span class="sc">}</span><span class="ss">_MC</span><span class="sc">{</span>iMC<span class="sc">:02d}</span><span class="ss">.pkl'</span>)</span>
<span id="cb2-31"><a href="#cb2-31" aria-hidden="true" tabindex="-1"></a>        mc_par <span class="op">=</span> pickle.load(<span class="bu">open</span>(mc_file, <span class="st">'rb'</span>))[<span class="dv">0</span>]</span>
<span id="cb2-32"><a href="#cb2-32" aria-hidden="true" tabindex="-1"></a>        eseeMaxMC <span class="op">=</span> (mc_par[<span class="st">'fmax'</span>]<span class="op">/</span>mc_par[<span class="st">'ksee'</span>])<span class="op">**</span><span class="fl">0.5</span></span>
<span id="cb2-33"><a href="#cb2-33" aria-hidden="true" tabindex="-1"></a>        </span>
<span id="cb2-34"><a href="#cb2-34" aria-hidden="true" tabindex="-1"></a>        entry <span class="op">=</span> {<span class="st">'muscle'</span>: mus, <span class="st">'MC'</span>: iMC, <span class="st">'d_esee'</span>: (eseeMaxMC<span class="op">-</span>eseeMaxOR)}</span>
<span id="cb2-35"><a href="#cb2-35" aria-hidden="true" tabindex="-1"></a>        <span class="cf">for</span> param <span class="kw">in</span> params:</span>
<span id="cb2-36"><a href="#cb2-36" aria-hidden="true" tabindex="-1"></a>            entry[param] <span class="op">=</span> stats.pdiff(mc_par[param], or_par[param])</span>
<span id="cb2-37"><a href="#cb2-37" aria-hidden="true" tabindex="-1"></a>        </span>
<span id="cb2-38"><a href="#cb2-38" aria-hidden="true" tabindex="-1"></a>        results.append(entry)</span>
<span id="cb2-39"><a href="#cb2-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-40"><a href="#cb2-40" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert to pandas DataFrame</span></span>
<span id="cb2-41"><a href="#cb2-41" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(results)</span>
<span id="cb2-42"><a href="#cb2-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-43"><a href="#cb2-43" aria-hidden="true" tabindex="-1"></a><span class="co"># Compute summary statistics</span></span>
<span id="cb2-44"><a href="#cb2-44" aria-hidden="true" tabindex="-1"></a>df_mean <span class="op">=</span> df.groupby(<span class="st">'muscle'</span>).mean().T</span>
<span id="cb2-45"><a href="#cb2-45" aria-hidden="true" tabindex="-1"></a>df_std <span class="op">=</span> df.groupby(<span class="st">'muscle'</span>).std().T</span>
<span id="cb2-46"><a href="#cb2-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-47"><a href="#cb2-47" aria-hidden="true" tabindex="-1"></a><span class="co"># Extract all contraction dynamics parameters except ksee</span></span>
<span id="cb2-48"><a href="#cb2-48" aria-hidden="true" tabindex="-1"></a>p_MC_con_avg <span class="op">=</span> df_mean.loc[[<span class="st">'a'</span>,<span class="st">'b'</span>, <span class="st">'kpee'</span>, <span class="st">'fmax'</span>, <span class="st">'lce_opt'</span>, <span class="st">'lpee0'</span>, <span class="st">'lsee0'</span>]]</span>
<span id="cb2-49"><a href="#cb2-49" aria-hidden="true" tabindex="-1"></a>p_MC_con_std <span class="op">=</span> df_std.loc[[<span class="st">'a'</span>, <span class="st">'b'</span>, <span class="st">'fmax'</span>, <span class="st">'kpee'</span>, <span class="st">'lce_opt'</span>, <span class="st">'lpee0'</span>, <span class="st">'lsee0'</span>]]</span>
<span id="cb2-50"><a href="#cb2-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-51"><a href="#cb2-51" aria-hidden="true" tabindex="-1"></a><span class="co"># Percentage difference over all muscles: from real to estimated ksee</span></span>
<span id="cb2-52"><a href="#cb2-52" aria-hidden="true" tabindex="-1"></a>p_MC_ksee_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_mean<span class="sc">.</span>loc[<span class="st">'ksee'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span>  <span class="co"># [%] percentage difference</span></span>
<span id="cb2-53"><a href="#cb2-53" aria-hidden="true" tabindex="-1"></a><span class="co"># Absolute difference over all muscles of SEE elongation @ Fcemax</span></span>
<span id="cb2-54"><a href="#cb2-54" aria-hidden="true" tabindex="-1"></a>d_MC_esee_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_mean<span class="sc">.</span>loc[<span class="st">'d_esee'</span>]<span class="sc">.</span>mean()<span class="op">*</span><span class="fl">1e3</span><span class="sc">:0.1f}</span><span class="ss">'</span>  <span class="co"># [mm]</span></span>
<span id="cb2-55"><a href="#cb2-55" aria-hidden="true" tabindex="-1"></a><span class="co"># Max. avg. percentage difference over all muscles: from real to estimated value of contraction dyn. parms</span></span>
<span id="cb2-56"><a href="#cb2-56" aria-hidden="true" tabindex="-1"></a>p_MC_con_maxavg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_MC_con_avg<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.1f}</span><span class="ss">'</span> </span>
<span id="cb2-57"><a href="#cb2-57" aria-hidden="true" tabindex="-1"></a><span class="co"># Max. std. percentage difference over all muscles: from real to estimated value of contraction dyn. parms</span></span>
<span id="cb2-58"><a href="#cb2-58" aria-hidden="true" tabindex="-1"></a>p_MC_con_maxstd <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>p_MC_con_std<span class="sc">.</span><span class="bu">abs</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.0f}</span><span class="ss">'</span> </span>
<span id="cb2-59"><a href="#cb2-59" aria-hidden="true" tabindex="-1"></a><span class="co"># Max. std percentage difference over all muscles: from real to estimated value of kpee</span></span>
<span id="cb2-60"><a href="#cb2-60" aria-hidden="true" tabindex="-1"></a>p_MC_kpee_maxstd <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_std<span class="sc">.</span>loc[<span class="st">'kpee'</span>]<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.0f}</span><span class="ss">'</span></span>
<span id="cb2-61"><a href="#cb2-61" aria-hidden="true" tabindex="-1"></a><span class="co"># Max. avg percentage difference over all muscles: from real to estimated value of tact</span></span>
<span id="cb2-62"><a href="#cb2-62" aria-hidden="true" tabindex="-1"></a>p_MC_tact_maxavg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_mean<span class="sc">.</span>loc[<span class="st">'tact'</span>]<span class="sc">.</span><span class="bu">max</span>()<span class="sc">:0.1f}</span><span class="ss">'</span></span>
<span id="cb2-63"><a href="#cb2-63" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. std percentage difference over all muscles: from real to estimated value of tact</span></span>
<span id="cb2-64"><a href="#cb2-64" aria-hidden="true" tabindex="-1"></a>p_MC_tact_avgstd <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_std<span class="sc">.</span>loc[<span class="st">'tact'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</details>
</div>
<section id="monte-carlo-simulations-1" class="level4" data-number="3.2.1">
<h4 data-number="3.2.1" class="anchored" data-anchor-id="monte-carlo-simulations-1"><span class="header-section-number">3.2.1</span> Monte Carlo simulations</h4>
<p>Monte Carlo simulations were used to examine how shifts in the MTC force–length relationship caused by a decrease in SEE stiffness (e.g., due to irreversible SEE damage in experiments) affect the accuracy of the estimated contraction and excitation dynamics parameter values. The induced shifts of the MTC force-length relationship were between 0–1 mm, and therefore it was no surprise that SEE stiffness decreased by 36% on average — which corresponds to a .5 mm shift.</p>
<p>All other estimated contraction dynamics parameter values were within 6.3% on average of their actual value (<a href="#tbl-r-pdiff" class="quarto-xref">Table&nbsp;1</a>). The variance in the estimated contraction dynamics parameter values was below 31% for all parameters except for the PEE stiffness scaling parameter. The PEE stiffness scaling parameter showed substantially higher variance (with a standard deviation up to 31%), because it was based on four or fewer quick-release trials, making it more sensitive to perturbations in the data. Regarding the excitation dynamics, the influence on the average time constants was minimal (within 0.7%), but affected the variance in the estimated values of the activation time constant (with a standard deviation of 4% on average). These findings indicate that an average decrease in SEE stiffness of 36% has a much smaller effect on the estimated parameter values of the contraction and excitation dynamics, even at the level of an individual muscle.</p>
</section>
<section id="interdependency-of-parameter-values-1" class="level4" data-number="3.2.2">
<h4 data-number="3.2.2" class="anchored" data-anchor-id="interdependency-of-parameter-values-1"><span class="header-section-number">3.2.2</span> Interdependency of parameter values</h4>
<p>We investigated the interdependency of the estimated parameter values by adjusting each parameter value by 5% and re-estimating all other parameter values. <span class="math inline">\(F_{CE}^{max}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span> were the parameters that had most effect on the estimates of the others.</p>
<p>First, <span class="math inline">\(F_{CE}^{max}\)</span> affected the estimation of parameter <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span> of the CE force-velocity relationship. Underestimating <span class="math inline">\(F_{CE}^{max}\)</span> leads to an underestimation of CE force at 0 velocity. Due to the formulation of the CE force-velocity relationship, the curve, by definition, crosses the point at <span class="math inline">\(v_{CE}=0\)</span> and <span class="math inline">\(F_{CE}=F_{CE}^{max}\)</span>. Consequently, the best fit to the data with an underestimated <span class="math inline">\(F_{CE}^{max}\)</span> is a much flatter CE force-velocity relationship, which is realised by an increase in <span class="math inline">\(a\)</span> and a decrease in <span class="math inline">\(b\)</span>. This flatter CE force-velocity relationship affects the estimation of all other contraction dynamics parameter values because the CE force-velocity relationship is used to estimate the CE shortening due to the quick-release (see <a href="#sec-m-im" class="quarto-xref">Section&nbsp;2.3.4</a>). As a result, <span class="math inline">\(k_{SEE}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span> decreased due to underestimating <span class="math inline">\(F_{CE}^{max}\)</span>, whereas <span class="math inline">\(L_{CE}^{opt}\)</span> increased. In summary, <span class="math inline">\(F_{CE}^{max}\)</span> substantially influenced the estimation of all contraction dynamics parameter values mainly by its effect on the CE force-velocity relationship.</p>
<p>Second, <span class="math inline">\(L_{SEE}^0\)</span> affected the estimation of <span class="math inline">\(L_{CE}^{opt}\)</span> and <span class="math inline">\(F_{CE}^{max}\)</span>. As explained earlier, when <span class="math inline">\(L_{SEE}^0\)</span> is overestimated, <span class="math inline">\(L_{CE}^{opt}\)</span> is underestimated, such that <span class="math inline">\(L_{MTC}^{opt}\)</span> remains more or less unaffected. Underestimating <span class="math inline">\(L_{CE}^{opt}\)</span> causes a narrower MTC force-length relationship, leading to an overestimation of <span class="math inline">\(F_{CE}^{max}\)</span> to preserve a good fit between the data and the model. As previously discussed, overestimating <span class="math inline">\(F_{CE}^{max}\)</span>, in turn, resulted in an underestimation of parameters <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span>. Thus, <span class="math inline">\(L_{SEE}^0\)</span> substantially influenced the estimation of all contraction dynamics parameter values mainly by its effect on the MTC force-length relationship.</p>
<p>Taken together, the interdependence of <span class="math inline">\(F_{CE}^{max}\)</span> and <span class="math inline">\(L_{SEE}^0\)</span> with other parameter values underscores the need for precise estimation of these key parameters. In this regard, it is reassuring that these two parameters were found to be robust for perturbations in the experimental data.</p>
<div class="quarto-embed-nb-cell">
<div class="cell" data-execution_count="8">
<div id="tbl-r-interdep" class="cell quarto-float quarto-figure quarto-figure-center anchored" data-execution_count="8">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-r-interdep-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Interdependency of the estimated MTC parameter values. Each entry shows the percentage change in the row parameter resulting from a 5% change in the column parameter. All values are expressed as percentage changes.
</figcaption>
<div aria-describedby="tbl-r-interdep-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output cell-output-display">



<meta charset="utf-8">


<div id="zejledoosh" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>
#zejledoosh table {
          font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
          -webkit-font-smoothing: antialiased;
          -moz-osx-font-smoothing: grayscale;
        }

#zejledoosh thead, tbody, tfoot, tr, td, th { border-style: none !important; }
 tr { background-color: transparent !important; }
#zejledoosh p { margin: 0 !important; padding: 0 !important; }
 #zejledoosh .gt_table { display: table !important; border-collapse: collapse !important; line-height: normal !important; margin-left: auto !important; margin-right: auto !important; color: #333333 !important; font-size: 16px !important; font-weight: normal !important; font-style: normal !important; background-color: #FFFFFF !important; width: auto !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #A8A8A8 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #A8A8A8 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; }
 #zejledoosh .gt_caption { padding-top: 4px !important; padding-bottom: 4px !important; }
 #zejledoosh .gt_title { color: #333333 !important; font-size: 125% !important; font-weight: initial !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; border-bottom-color: #FFFFFF !important; border-bottom-width: 0 !important; }
 #zejledoosh .gt_subtitle { color: #333333 !important; font-size: 85% !important; font-weight: initial !important; padding-top: 3px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; border-top-color: #FFFFFF !important; border-top-width: 0 !important; }
 #zejledoosh .gt_heading { background-color: #FFFFFF !important; text-align: center !important; border-bottom-color: #FFFFFF !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #zejledoosh .gt_bottom_border { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #zejledoosh .gt_col_headings { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #zejledoosh .gt_col_heading { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: bold !important; text-transform: inherit !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; overflow-x: hidden !important; }
 #zejledoosh .gt_column_spanner_outer { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: bold !important; text-transform: inherit !important; padding-top: 0 !important; padding-bottom: 0 !important; padding-left: 4px !important; padding-right: 4px !important; }
 #zejledoosh .gt_column_spanner_outer:first-child { padding-left: 0 !important; }
 #zejledoosh .gt_column_spanner_outer:last-child { padding-right: 0 !important; }
 #zejledoosh .gt_column_spanner { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; overflow-x: hidden !important; display: inline-block !important; width: 100% !important; }
 #zejledoosh .gt_spanner_row { border-bottom-style: hidden !important; }
 #zejledoosh .gt_group_heading { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; text-align: left !important; }
 #zejledoosh .gt_empty_group_heading { padding: 0.5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: middle !important; }
 #zejledoosh .gt_from_md> :first-child { margin-top: 0 !important; }
 #zejledoosh .gt_from_md> :last-child { margin-bottom: 0 !important; }
 #zejledoosh .gt_row { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; margin: 10px !important; border-top-style: solid !important; border-top-width: 1px !important; border-top-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; overflow-x: hidden !important; }
 #zejledoosh .gt_stub { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; }
 #zejledoosh .gt_stub_row_group { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; vertical-align: top !important; }
 #zejledoosh .gt_row_group_first td { border-top-width: 2px !important; }
 #zejledoosh .gt_row_group_first th { border-top-width: 2px !important; }
 #zejledoosh .gt_striped { color: #333333 !important; background-color: #F4F4F4 !important; }
 #zejledoosh .gt_table_body { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #zejledoosh .gt_grand_summary_row { color: #333333 !important; background-color: #FFFFFF !important; text-transform: inherit !important; padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; }
 #zejledoosh .gt_first_grand_summary_row_bottom { border-top-style: double !important; border-top-width: 6px !important; border-top-color: #D3D3D3 !important; }
 #zejledoosh .gt_last_grand_summary_row_top { border-bottom-style: double !important; border-bottom-width: 6px !important; border-bottom-color: #D3D3D3 !important; }
 #zejledoosh .gt_sourcenotes { color: #333333 !important; background-color: #FFFFFF !important; border-bottom-style: none !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; }
 #zejledoosh .gt_sourcenote { font-size: 90% !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; }
 #zejledoosh .gt_left { text-align: left !important; }
 #zejledoosh .gt_center { text-align: center !important; }
 #zejledoosh .gt_right { text-align: right !important; font-variant-numeric: tabular-nums !important; }
 #zejledoosh .gt_font_normal { font-weight: normal !important; }
 #zejledoosh .gt_font_bold { font-weight: bold !important; }
 #zejledoosh .gt_font_italic { font-style: italic !important; }
 #zejledoosh .gt_super { font-size: 65% !important; }
 #zejledoosh .gt_footnote_marks { font-size: 75% !important; vertical-align: 0.4em !important; position: initial !important; }
 #zejledoosh .gt_asterisk { font-size: 100% !important; vertical-align: 0 !important; }
 
</style>

<table class="gt_table do-not-create-environment cell caption-top table table-sm table-striped small" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="gt_col_headings header">
<th id="index" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col"></th>
<th id="a" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col">$a$</th>
<th id="b" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col">$b$</th>
<th id="fmax" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col">$F_{CE}^{max}$</th>
<th id="ksee" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col">$k_{SEE}$</th>
<th id="lce_opt" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col">$L_{CE}^{opt}$</th>
<th id="lsee0" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" style="font-weight: bold" scope="col">$L_{SEE}^0$</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left">$a$</td>
<td class="gt_row gt_center">-</td>
<td class="gt_row gt_center">10.1 ± 1.3</td>
<td class="gt_row gt_center">-14.0 ± 1.0</td>
<td class="gt_row gt_center">-0.6 ± 0.2</td>
<td class="gt_row gt_center">3.4 ± 1.0</td>
<td class="gt_row gt_center">-10.3 ± 2.0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$b$</td>
<td class="gt_row gt_center">2.3 ± 1.2</td>
<td class="gt_row gt_center">-</td>
<td class="gt_row gt_center">-14.3 ± 3.1</td>
<td class="gt_row gt_center">1.1 ± 0.3</td>
<td class="gt_row gt_center">3.9 ± 1.1</td>
<td class="gt_row gt_center">-10.8 ± 1.6</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$F_{CE}^{max}$</td>
<td class="gt_row gt_center">-0.6 ± 0.1</td>
<td class="gt_row gt_center">-0.6 ± 0.1</td>
<td class="gt_row gt_center">-</td>
<td class="gt_row gt_center">-0.6 ± 0.2</td>
<td class="gt_row gt_center">-1.2 ± 0.5</td>
<td class="gt_row gt_center">3.5 ± 0.5</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$k_{PEE}$</td>
<td class="gt_row gt_center">-6.8 ± 2.9</td>
<td class="gt_row gt_center">-6.8 ± 2.9</td>
<td class="gt_row gt_center">-6.9 ± 3.5</td>
<td class="gt_row gt_center">-6.8 ± 3.2</td>
<td class="gt_row gt_center">-6.8 ± 2.9</td>
<td class="gt_row gt_center">-6.8 ± 3.3</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$k_{SEE}$</td>
<td class="gt_row gt_center">52.4 ± 12.6</td>
<td class="gt_row gt_center">52.4 ± 12.8</td>
<td class="gt_row gt_center">54.3 ± 17.0</td>
<td class="gt_row gt_center">-</td>
<td class="gt_row gt_center">52.4 ± 12.8</td>
<td class="gt_row gt_center">52.5 ± 14.3</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$L_{CE}^{opt}$</td>
<td class="gt_row gt_center">1.9 ± 0.6</td>
<td class="gt_row gt_center">1.9 ± 0.6</td>
<td class="gt_row gt_center">-5.4 ± 2.2</td>
<td class="gt_row gt_center">1.9 ± 0.6</td>
<td class="gt_row gt_center">-</td>
<td class="gt_row gt_center">-14.2 ± 3.0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$L_{PEE}^0$</td>
<td class="gt_row gt_center">-0.6 ± 0.7</td>
<td class="gt_row gt_center">-0.6 ± 0.7</td>
<td class="gt_row gt_center">-0.6 ± 0.7</td>
<td class="gt_row gt_center">-0.6 ± 0.7</td>
<td class="gt_row gt_center">-0.6 ± 0.7</td>
<td class="gt_row gt_center">-10.6 ± 1.2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$L_{SEE}^0$</td>
<td class="gt_row gt_center">0.5 ± 0.2</td>
<td class="gt_row gt_center">0.5 ± 0.2</td>
<td class="gt_row gt_center">1.8 ± 0.6</td>
<td class="gt_row gt_center">0.5 ± 0.2</td>
<td class="gt_row gt_center">-1.7 ± 0.6</td>
<td class="gt_row gt_center">-</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$\tau_{act}$</td>
<td class="gt_row gt_center">23.4 ± 4.2</td>
<td class="gt_row gt_center">23.4 ± 4.6</td>
<td class="gt_row gt_center">24.2 ± 5.9</td>
<td class="gt_row gt_center">23.3 ± 5.0</td>
<td class="gt_row gt_center">23.7 ± 4.3</td>
<td class="gt_row gt_center">20.9 ± 6.3</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$\tau_{deact}$</td>
<td class="gt_row gt_center">4.8 ± 0.7</td>
<td class="gt_row gt_center">4.9 ± 0.8</td>
<td class="gt_row gt_center">5.0 ± 3.1</td>
<td class="gt_row gt_center">4.8 ± 0.9</td>
<td class="gt_row gt_center">4.9 ± 1.4</td>
<td class="gt_row gt_center">5.1 ± 3.0</td>
</tr>
</tbody>
</table>


</div>
        


</div>
</div>
</figure>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-2" href="../tables/all_tables.html#cell-tbl-r-interdep">Source: Tables</a></div>
</section>
</section>
<section id="evaluating-model-predictions---in-situ-data" class="level3" data-number="3.3">
<h3 data-number="3.3" class="anchored" data-anchor-id="evaluating-model-predictions---in-situ-data"><span class="header-section-number">3.3</span> Evaluating model predictions - <em>in situ</em> data</h3>
<div id="cell-9" class="cell" data-execution_count="3">
<details class="code-fold">
<summary>Code</summary>
<div class="code-copy-outer-scaffold"><div class="sourceCode cell-code" id="cb3"><pre class="sourceCode python code-with-copy"><code class="sourceCode python"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Load packages</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> os, pickle, sys, glob</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> numpy <span class="im">as</span> np</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> pandas <span class="im">as</span> pd</span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a><span class="im">from</span> pathlib <span class="im">import</span> Path</span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Directories</span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>cwd <span class="op">=</span> Path.cwd()</span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>baseDir <span class="op">=</span> cwd.parent</span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>dataDir <span class="op">=</span> baseDir <span class="op">/</span> <span class="st">'data'</span></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>funcDir <span class="op">=</span> baseDir <span class="op">/</span> <span class="st">'analysis'</span> <span class="op">/</span> <span class="st">'functions'</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>sys.path.append(<span class="bu">str</span>(funcDir))</span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Custom functions</span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a><span class="im">import</span> helpers, stats</span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Extract parameters</span></span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>muscles <span class="op">=</span> [<span class="st">'GMe1'</span>, <span class="st">'GMe2'</span>, <span class="st">'GMe3'</span>]</span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a>param_keys <span class="op">=</span> [<span class="st">'a'</span>, <span class="st">'b'</span>, <span class="st">'kpee'</span>, <span class="st">'ksee'</span>, <span class="st">'fmax'</span>, <span class="st">'lce_opt'</span>, <span class="st">'lpee0'</span>, <span class="st">'lsee0'</span>, <span class="st">'tact'</span>, <span class="st">'tdeact'</span>]</span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>orParms, tmParms, imParms <span class="op">=</span> [], [], []</span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> mus <span class="kw">in</span> muscles:   </span>
<span id="cb3-23"><a href="#cb3-23" aria-hidden="true" tabindex="-1"></a>    tmPar, dataQRtm <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_TM.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>:<span class="dv">2</span>]</span>
<span id="cb3-24"><a href="#cb3-24" aria-hidden="true" tabindex="-1"></a>    imPar, dataQRim <span class="op">=</span> pickle.load(<span class="bu">open</span>(os.path.join(dataDir,mus,<span class="st">'parameters'</span>,mus<span class="op">+</span><span class="st">'_IM.pkl'</span>), <span class="st">'rb'</span>))[<span class="dv">0</span>:<span class="dv">2</span>]</span>
<span id="cb3-25"><a href="#cb3-25" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb3-26"><a href="#cb3-26" aria-hidden="true" tabindex="-1"></a>    parms <span class="op">=</span> [tmPar[k] <span class="cf">for</span> k <span class="kw">in</span> param_keys]</span>
<span id="cb3-27"><a href="#cb3-27" aria-hidden="true" tabindex="-1"></a>    tmParms.append(parms)</span>
<span id="cb3-28"><a href="#cb3-28" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb3-29"><a href="#cb3-29" aria-hidden="true" tabindex="-1"></a>    parms <span class="op">=</span> [imPar[k] <span class="cf">for</span> k <span class="kw">in</span> param_keys]</span>
<span id="cb3-30"><a href="#cb3-30" aria-hidden="true" tabindex="-1"></a>    imParms.append(parms)</span>
<span id="cb3-31"><a href="#cb3-31" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb3-32"><a href="#cb3-32" aria-hidden="true" tabindex="-1"></a><span class="co"># Store in pandas dataframe</span></span>
<span id="cb3-33"><a href="#cb3-33" aria-hidden="true" tabindex="-1"></a>df_tm <span class="op">=</span> pd.DataFrame(<span class="bu">list</span>(<span class="bu">zip</span>(<span class="op">*</span>tmParms)), columns<span class="op">=</span>[<span class="st">'GM1'</span>, <span class="st">'GM2'</span>, <span class="st">'GM3'</span>], index<span class="op">=</span>param_keys) <span class="co"># df with estimated parameter values using the traditional method</span></span>
<span id="cb3-34"><a href="#cb3-34" aria-hidden="true" tabindex="-1"></a>df_im <span class="op">=</span> pd.DataFrame(<span class="bu">list</span>(<span class="bu">zip</span>(<span class="op">*</span>imParms)), columns<span class="op">=</span>[<span class="st">'GM1'</span>, <span class="st">'GM2'</span>, <span class="st">'GM3'</span>], index<span class="op">=</span>param_keys) <span class="co"># df with estimated parameter values using the improved method</span></span>
<span id="cb3-35"><a href="#cb3-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-36"><a href="#cb3-36" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Compute percentage differences from real/actual and estimated parameter values</span></span>
<span id="cb3-37"><a href="#cb3-37" aria-hidden="true" tabindex="-1"></a><span class="co"># Positive: Imporoved is that % higher than Traditional</span></span>
<span id="cb3-38"><a href="#cb3-38" aria-hidden="true" tabindex="-1"></a><span class="co"># Negative: Imporoved is that % lower than Traditional</span></span>
<span id="cb3-39"><a href="#cb3-39" aria-hidden="true" tabindex="-1"></a>df_p <span class="op">=</span> stats.pdiff(df_im, df_tm)</span>
<span id="cb3-40"><a href="#cb3-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-41"><a href="#cb3-41" aria-hidden="true" tabindex="-1"></a><span class="co"># %% Compute RMSDs</span></span>
<span id="cb3-42"><a href="#cb3-42" aria-hidden="true" tabindex="-1"></a>par_models <span class="op">=</span> [<span class="st">'TM'</span>, <span class="st">'IM'</span>]</span>
<span id="cb3-43"><a href="#cb3-43" aria-hidden="true" tabindex="-1"></a>experiments <span class="op">=</span> [<span class="st">'QR'</span>, <span class="st">'SR'</span>, <span class="st">'ISOM'</span>, <span class="st">'SSC'</span>]</span>
<span id="cb3-44"><a href="#cb3-44" aria-hidden="true" tabindex="-1"></a>muscles <span class="op">=</span> [<span class="st">'GMe1'</span>, <span class="st">'GMe2'</span>, <span class="st">'GMe3'</span>]</span>
<span id="cb3-45"><a href="#cb3-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-46"><a href="#cb3-46" aria-hidden="true" tabindex="-1"></a>avg_mean_all <span class="op">=</span> []</span>
<span id="cb3-47"><a href="#cb3-47" aria-hidden="true" tabindex="-1"></a>columns <span class="op">=</span> [<span class="ss">f'</span><span class="sc">{</span>model<span class="sc">}</span><span class="ss">_</span><span class="sc">{</span>exp<span class="sc">}</span><span class="ss">'</span> <span class="cf">for</span> model <span class="kw">in</span> par_models <span class="cf">for</span> exp <span class="kw">in</span> experiments]</span>
<span id="cb3-48"><a href="#cb3-48" aria-hidden="true" tabindex="-1"></a>rows <span class="op">=</span> muscles <span class="op">+</span> [<span class="st">'Avg ± Std'</span>]</span>
<span id="cb3-49"><a href="#cb3-49" aria-hidden="true" tabindex="-1"></a>df <span class="op">=</span> pd.DataFrame(index<span class="op">=</span>rows, columns<span class="op">=</span>columns, dtype<span class="op">=</span><span class="bu">str</span>)</span>
<span id="cb3-50"><a href="#cb3-50" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> exp <span class="kw">in</span> experiments:</span>
<span id="cb3-51"><a href="#cb3-51" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> model <span class="kw">in</span> par_models:</span>
<span id="cb3-52"><a href="#cb3-52" aria-hidden="true" tabindex="-1"></a>        all_rmsd <span class="op">=</span> []</span>
<span id="cb3-53"><a href="#cb3-53" aria-hidden="true" tabindex="-1"></a>        <span class="cf">for</span> mus <span class="kw">in</span> muscles:</span>
<span id="cb3-54"><a href="#cb3-54" aria-hidden="true" tabindex="-1"></a>            <span class="co"># Load parameter</span></span>
<span id="cb3-55"><a href="#cb3-55" aria-hidden="true" tabindex="-1"></a>            parFile <span class="op">=</span> os.path.join(dataDir, mus, <span class="st">'parameters'</span>, <span class="ss">f'</span><span class="sc">{</span>mus<span class="sc">}</span><span class="ss">_</span><span class="sc">{</span>model<span class="sc">}</span><span class="ss">.pkl'</span>)</span>
<span id="cb3-56"><a href="#cb3-56" aria-hidden="true" tabindex="-1"></a>            muspar <span class="op">=</span> pickle.load(<span class="bu">open</span>(parFile, <span class="st">'rb'</span>))[<span class="dv">0</span>]</span>
<span id="cb3-57"><a href="#cb3-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-58"><a href="#cb3-58" aria-hidden="true" tabindex="-1"></a>            dataDirExp <span class="op">=</span> os.path.join(dataDir, mus, <span class="st">'dataExp'</span>, exp)</span>
<span id="cb3-59"><a href="#cb3-59" aria-hidden="true" tabindex="-1"></a>            rrunDirExp <span class="op">=</span> os.path.join(dataDir, mus, <span class="st">'simsExp'</span>, model, exp)</span>
<span id="cb3-60"><a href="#cb3-60" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-61"><a href="#cb3-61" aria-hidden="true" tabindex="-1"></a>            dataFiles <span class="op">=</span> <span class="bu">sorted</span>(glob.glob(os.path.join(dataDirExp, <span class="st">'*'</span>)))</span>
<span id="cb3-62"><a href="#cb3-62" aria-hidden="true" tabindex="-1"></a>            rrunFiles <span class="op">=</span> <span class="bu">sorted</span>(glob.glob(os.path.join(rrunDirExp, <span class="st">'*'</span>)))</span>
<span id="cb3-63"><a href="#cb3-63" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-64"><a href="#cb3-64" aria-hidden="true" tabindex="-1"></a>            rms_list <span class="op">=</span> []</span>
<span id="cb3-65"><a href="#cb3-65" aria-hidden="true" tabindex="-1"></a>            <span class="cf">for</span> dataFile, rrunFile <span class="kw">in</span> <span class="bu">zip</span>(dataFiles, rrunFiles):</span>
<span id="cb3-66"><a href="#cb3-66" aria-hidden="true" tabindex="-1"></a>                dataFilename <span class="op">=</span> os.path.basename(dataFile)[:<span class="op">-</span><span class="dv">4</span>]</span>
<span id="cb3-67"><a href="#cb3-67" aria-hidden="true" tabindex="-1"></a>                rrunFilename <span class="op">=</span> os.path.basename(rrunFile)[:<span class="op">-</span><span class="dv">4</span>]</span>
<span id="cb3-68"><a href="#cb3-68" aria-hidden="true" tabindex="-1"></a>                <span class="cf">if</span> dataFilename[:<span class="op">-</span><span class="dv">3</span>] <span class="op">!=</span> rrunFilename[:<span class="op">-</span><span class="dv">3</span>]:</span>
<span id="cb3-69"><a href="#cb3-69" aria-hidden="true" tabindex="-1"></a>                    <span class="cf">raise</span> <span class="pp">ValueError</span>(<span class="ss">f"File mismatch: </span><span class="sc">{</span>dataFilename<span class="sc">}</span><span class="ss"> vs </span><span class="sc">{</span>rrunFilename<span class="sc">}</span><span class="ss">"</span>)</span>
<span id="cb3-70"><a href="#cb3-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-71"><a href="#cb3-71" aria-hidden="true" tabindex="-1"></a>                dataData <span class="op">=</span> pd.read_csv(dataFile).T.to_numpy()[<span class="dv">0</span>:<span class="dv">4</span>]</span>
<span id="cb3-72"><a href="#cb3-72" aria-hidden="true" tabindex="-1"></a>                rrunData <span class="op">=</span> pd.read_csv(rrunFile).T.to_numpy()</span>
<span id="cb3-73"><a href="#cb3-73" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-74"><a href="#cb3-74" aria-hidden="true" tabindex="-1"></a>                time1, _, stim1, fsee1 <span class="op">=</span> dataData</span>
<span id="cb3-75"><a href="#cb3-75" aria-hidden="true" tabindex="-1"></a>                time2, _, _, fsee2 <span class="op">=</span> rrunData</span>
<span id="cb3-76"><a href="#cb3-76" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-77"><a href="#cb3-77" aria-hidden="true" tabindex="-1"></a>                tStimOn, tStimOff <span class="op">=</span> helpers.get_stim(time1, stim1)[<span class="dv">1</span>:]</span>
<span id="cb3-78"><a href="#cb3-78" aria-hidden="true" tabindex="-1"></a>                iStart <span class="op">=</span> <span class="bu">int</span>(np.argmin(<span class="bu">abs</span>(time1 <span class="op">-</span> tStimOn[<span class="dv">0</span>])))</span>
<span id="cb3-79"><a href="#cb3-79" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-80"><a href="#cb3-80" aria-hidden="true" tabindex="-1"></a>                <span class="cf">if</span> exp <span class="op">==</span> <span class="st">'ISOM'</span>:</span>
<span id="cb3-81"><a href="#cb3-81" aria-hidden="true" tabindex="-1"></a>                    iStop <span class="op">=</span> <span class="bu">int</span>(np.argmin(<span class="bu">abs</span>(time1 <span class="op">-</span> <span class="fl">0.1</span> <span class="op">-</span> tStimOff[<span class="dv">0</span>])))</span>
<span id="cb3-82"><a href="#cb3-82" aria-hidden="true" tabindex="-1"></a>                <span class="cf">elif</span> exp <span class="kw">in</span> [<span class="st">'QR'</span>, <span class="st">'SR'</span>]:</span>
<span id="cb3-83"><a href="#cb3-83" aria-hidden="true" tabindex="-1"></a>                    iStop <span class="op">=</span> <span class="bu">int</span>(np.argmin(<span class="bu">abs</span>(time1 <span class="op">-</span> tStimOff[<span class="dv">0</span>])))</span>
<span id="cb3-84"><a href="#cb3-84" aria-hidden="true" tabindex="-1"></a>                <span class="cf">else</span>:  <span class="co"># SSC</span></span>
<span id="cb3-85"><a href="#cb3-85" aria-hidden="true" tabindex="-1"></a>                    iStop <span class="op">=</span> <span class="bu">int</span>(np.argmin(<span class="bu">abs</span>(time1 <span class="op">-</span> tStimOff[<span class="op">-</span><span class="dv">1</span>])))</span>
<span id="cb3-86"><a href="#cb3-86" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-87"><a href="#cb3-87" aria-hidden="true" tabindex="-1"></a>                rms <span class="op">=</span> stats.rmse(fsee1[iStart:iStop], fsee2[iStart:iStop]) <span class="op">/</span> muspar[<span class="st">'fmax'</span>] <span class="op">*</span> <span class="dv">100</span></span>
<span id="cb3-88"><a href="#cb3-88" aria-hidden="true" tabindex="-1"></a>                rms_list.append(rms)</span>
<span id="cb3-89"><a href="#cb3-89" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-90"><a href="#cb3-90" aria-hidden="true" tabindex="-1"></a>            M <span class="op">=</span> np.mean(rms_list)</span>
<span id="cb3-91"><a href="#cb3-91" aria-hidden="true" tabindex="-1"></a>            S <span class="op">=</span> np.std(rms_list)</span>
<span id="cb3-92"><a href="#cb3-92" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-93"><a href="#cb3-93" aria-hidden="true" tabindex="-1"></a>            df.loc[mus, <span class="ss">f'</span><span class="sc">{</span>model<span class="sc">}</span><span class="ss">_</span><span class="sc">{</span>exp<span class="sc">}</span><span class="ss">'</span>] <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>M<span class="sc">:.1f}</span><span class="ss"> ± </span><span class="sc">{</span>S<span class="sc">:.1f}</span><span class="ss">'</span></span>
<span id="cb3-94"><a href="#cb3-94" aria-hidden="true" tabindex="-1"></a>            all_rmsd <span class="op">+=</span> rms_list  <span class="co"># just mean, for averaging across muscles</span></span>
<span id="cb3-95"><a href="#cb3-95" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-96"><a href="#cb3-96" aria-hidden="true" tabindex="-1"></a>        <span class="co"># Average over muscles for this (exp, model)</span></span>
<span id="cb3-97"><a href="#cb3-97" aria-hidden="true" tabindex="-1"></a>        avg_M <span class="op">=</span> np.mean(all_rmsd)</span>
<span id="cb3-98"><a href="#cb3-98" aria-hidden="true" tabindex="-1"></a>        avg_S <span class="op">=</span> np.std(all_rmsd)</span>
<span id="cb3-99"><a href="#cb3-99" aria-hidden="true" tabindex="-1"></a>        df.loc[<span class="st">'Avg ± Std'</span>, <span class="ss">f'</span><span class="sc">{</span>model<span class="sc">}</span><span class="ss">_</span><span class="sc">{</span>exp<span class="sc">}</span><span class="ss">'</span>] <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>avg_M<span class="sc">:.1f}</span><span class="ss"> ± </span><span class="sc">{</span>avg_S<span class="sc">:.1f}</span><span class="ss">'</span></span>
<span id="cb3-100"><a href="#cb3-100" aria-hidden="true" tabindex="-1"></a>        avg_mean_all.append(avg_M)</span>
<span id="cb3-101"><a href="#cb3-101" aria-hidden="true" tabindex="-1"></a>avg_mean_all <span class="op">=</span> np.array(avg_mean_all)</span>
<span id="cb3-102"><a href="#cb3-102" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-103"><a href="#cb3-103" aria-hidden="true" tabindex="-1"></a><span class="co">#%% 3.3: Evaluating model predictions</span></span>
<span id="cb3-104"><a href="#cb3-104" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference: from traditional to improved method: SEE stiffness</span></span>
<span id="cb3-105"><a href="#cb3-105" aria-hidden="true" tabindex="-1"></a>p_IS_ksee_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_p<span class="sc">.</span>loc[<span class="st">'ksee'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="co">#  [%] avg. percentage difference</span></span>
<span id="cb3-106"><a href="#cb3-106" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference: from traditional to improved method: tact</span></span>
<span id="cb3-107"><a href="#cb3-107" aria-hidden="true" tabindex="-1"></a>p_IS_tact_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_p<span class="sc">.</span>loc[<span class="st">'tact'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="co">#  [%] avg. percentage difference</span></span>
<span id="cb3-108"><a href="#cb3-108" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference: from traditional to improved method: tdeact</span></span>
<span id="cb3-109"><a href="#cb3-109" aria-hidden="true" tabindex="-1"></a>p_IS_tdeact_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_p<span class="sc">.</span>loc[<span class="st">'tdeact'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="co">#  [%] avg. percentage difference</span></span>
<span id="cb3-110"><a href="#cb3-110" aria-hidden="true" tabindex="-1"></a><span class="co"># Avg. difference: from traditional to improved method: lce_opt</span></span>
<span id="cb3-111"><a href="#cb3-111" aria-hidden="true" tabindex="-1"></a>p_IS_lceopt_avg <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>df_p<span class="sc">.</span>loc[<span class="st">'lce_opt'</span>]<span class="sc">.</span>mean()<span class="sc">:0.0f}</span><span class="ss">'</span> <span class="co">#  [%] avg. percentage difference</span></span>
<span id="cb3-112"><a href="#cb3-112" aria-hidden="true" tabindex="-1"></a><span class="co"># Take the mean over all each sort of experiment</span></span>
<span id="cb3-113"><a href="#cb3-113" aria-hidden="true" tabindex="-1"></a>p_IS_rmsd <span class="op">=</span> <span class="ss">f'</span><span class="sc">{</span>stats<span class="sc">.</span>pdiff(avg_mean_all[<span class="dv">1</span>::<span class="dv">2</span>],avg_mean_all[<span class="dv">0</span>::<span class="dv">2</span>])<span class="sc">.</span>mean()<span class="sc">:2.0f}</span><span class="ss">'</span></span></code></pre></div><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></div>
</details>
</div>
<p>We estimated contraction and excitation dynamics parameter values from <em>in situ</em> data of rat m. gastrocnemius medialis (<a href="#supptbl-insitu" class="quarto-xref">Table S2</a>). The improved method yielded a 67% higher SEE stiffness compared to the traditional method. As explained earlier, higher SEE stiffness also affects the estimation of all other parameter values. The most noticeable changes were longer time constants for both the activation (<span class="math inline">\(\tau_{act}\)</span>) and deactivation (<span class="math inline">\(\tau_{deact}\)</span>) dynamics, with increases of 61% and 16% on average, respectively. Another noticeable change was in CE optimum length, which was about 11% higher using the improved method compared to the traditional method.</p>
<p>To assess model predictions, we re-ran quick-release, step-ramp and isometric experiments, as well as stretch-shortening cycles with experimentally measured MTC length and CE stimulation over time as inputs to the model. We compared the model predictions using parameter sets obtained from both methods. For all experiments and rats, the average difference between experimental data and model predictions was -30% smaller on average using parameters from the improved method (<a href="#fig-r-ssc-exp" class="quarto-xref">Figure&nbsp;8</a>; <a href="#supptbl-rmsd" class="quarto-xref">Table S3</a>). Overall, the improved method substantially enhanced the predictions derived by the Hill-type MTC model compared to the traditional method.</p>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-r-insitu-fit" class="cell" data-execution_count="8">
<div class="cell-output cell-output-display">
<div id="fig-r-insitu-fit" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-r-insitu-fit-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-r-insitu-fit-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-r-insitu-fit-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;7: Representative example of experimental <em>in situ</em> data of rat 1 for the SEE force-length relationship (A), PEE force-length relationship (B), MTC force-length relationship (C) and the CE force-velocity relationship (D).** The orange dots depicts the experimental <em>in situ</em> data and the solid black line depicts the model fit.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-r-insitu-fit">Source: Figures</a></div>
<div class="quarto-embed-nb-cell">
<div id="cell-fig-r-ssc-exp" class="cell" data-execution_count="9">
<div class="cell-output cell-output-display">
<div id="fig-r-ssc-exp" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-r-ssc-exp-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-fig-r-ssc-exp-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-r-ssc-exp-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;8: Representative example of experimental <em>in situ</em> data and simulation results of rat 1 for SEE force over time during a quick-release experiment (A), step-ramp experiment (B), isometric experiment (C) and two stretch-shortening cycles (D &amp; E). The simulation results were obtained by re-simulating the experimental protocol with the MTC length and CE stimulation from the experimental data as input and by using either the parameter set obtained with the traditional method (TM; orange dashed line) or the improved method (IM; green dash-dotted line). For each panel, the top plot represents MTC length over time, with the bar indicating its range in mm. CE stimulation is maximal during the periods indicated by the black bars and is ‘off’ elsewhere.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-fig-r-ssc-exp">Source: Figures</a></div>
</section>
</section>
<section id="discussion" class="level2" data-number="4">
<h2 data-number="4" class="anchored" data-anchor-id="discussion"><span class="header-section-number">4</span> Discussion</h2>
<p>The aim of this study was to evaluate the accuracy of estimating contraction and excitation dynamics parameter values on the basis of data collected in commonly used experimental protocols. In real experiments, the actual parameter values are unknown, making it impossible to directly assess estimation accuracy. In this study, we took a different approach: using a Hill-type MTC model with parameter values obtained from literature we generated synthetic ‘data’ by simulating quick-release, step-ramp, and isometric experiments. We then estimated all important contraction and excitation dynamics parameter values by using the synthetic data. Since the actual parameter values were known in this case, we could assess how accurately they were retrieved. Two different estimation methods were compared. The first was the traditional method, commonly used in the literature, which does not account for muscle fibre shortening during quick-release experiments. The second was an improved method that includes a correction for muscle fibre shortening during the quick release. Both methods developed in this study—designed to estimate contraction and excitation dynamics parameters from quick-release, step-ramp, and isometric experiments using servomotors—are made available as an open-source toolbox. In the remainder of this paper, we will discuss 1) the difference between the two methods; 2) the robustness of the improved method and 3) the implications for muscle modelling.</p>
<p>In quick-release experiments using servomotors, muscle fibres shorten, even within the short duration of the release. Obviously, the extent of muscle fibre shortening depends on the duration of the quick-release. Consequently, the longer the quick-release, the more important it becomes to account for muscle fibre shortening when estimating SEE stiffness (see <a href="#suppfig-ksee-qr-duration" class="quarto-xref">Figure S3</a>). For a typical quick-release duration of 10 ms, we found that muscle fibre shortening was nearly 25% of MTC shortening (and thus 33% of SEE shortening), resulting in an underestimation of SEE stiffness by approximately 35%. This finding is important, especially in studies aiming to estimate SEE energy storage from SEE force or length, since energy storage estimates scale linearly with SEE stiffness. Although SEE was substantially underestimated with the traditional method, this had minimal effect on the estimated parameter values of the PEE and CE force-length relationships and the CE force-velocity relationship, but impacted the estimated excitation dynamics parameter values. The improved method, in turn, yielded SEE stiffness values close to the actual ones and substantially enhanced the estimates of the other parameter values.</p>
<p>Our sensitivity analysis revealed substantial interdependencies between certain parameters. Particularly, we observed high sensitivity of estimated parameter values to variations in the maximal isometric CE force (<span class="math inline">\(F_{CE}^{max}\)</span>) and the SEE slack length (<span class="math inline">\(L_{SEE}^0\)</span>) parameter values. Despite this sensitivity, it is important to note that these parameters were accurately estimated even from perturbed experimental data. This was true in general: contraction and excitation dynamics parameter values were quite robust against shifts in the MTC force-length relationship of the simulated data according to the Monte Carlo simulations. These findings indicate the robustness of the improved method.</p>
<p>We applied both the traditional and the improved method to <em>in situ</em> data from quick-release, step-ramp, and isometric experiments on rat m. gastrocnemius medialis. The improved method yielded substantially higher estimates of SEE stiffness compared to the traditional method (67% on average). Using parameter sets from both methods, we simulated the quick-release, step-ramp, and isometric experiments, as well as independent stretch-shortening cycles, all with experimentally obtained MTC length and CE stimulation over time as input to the model. With parameter estimates obtained with the traditional method, the root mean squared difference between predicted and experimentally measured force was about 5.6% of maximal CE force, averaged across all rats and experiments; with parameter estimates obtained with the improved method this was only 3.9%. The small difference between predicted and experimentally observed force over time with parameters from the improved method suggests that muscle force can be accurately predicted with a Hill-type MTC model across a wide variety of contractions. This is a remarkable finding for two reasons. First, the Hill-type MTC model is obviously a simplification of real muscle, abstracting the muscle belly, aponeurosis, and tendon into the CE, SEE, and PEE components, and thus does not account for certain complexities (e.g., non-constant pennation angle, muscle inhomogeneities, etc.). Second, the Hill-type MTC model used in this study consists of 26 parameters, of which only 10 values were estimated. This means that all other parameters were left suboptimal. Despite these simplifications, the model was able to accurately predict muscle behaviour, demonstrating that a relatively simple model with a limited number of estimated parameter values can still capture important aspects of muscle dynamics.</p>
<p>In conclusion, our results demonstrate that the improved parameter estimation method provides accurate and robust estimates of contraction and excitation dynamics properties, offering a reliable approach for muscle property estimation in both biomechanical and muscle physiology research. The approach that we designed in this study is made available as an open-source toolbox (<a href="https://github.com/edwinreuvers/mp-estimator">https://github.com/edwinreuvers/mp-estimator</a>).</p>
<div style="page-break-after: always;"></div>
</section>
<section id="acknowledgments" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="acknowledgments">Acknowledgments</h2>
<p>The authors thank Maarten F. Bobbert for helpful comments on a draft of the paper. The authors also thank Koen K. Lemaire for insightful discussions.</p>
</section>
<section id="funding" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="funding">Funding</h2>
<p>This work was funded by The Dutch Research Council (NWO)&nbsp;[21728 to D.A.K.].</p>
</section>
<section id="data-and-resource-availability" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="data-and-resource-availability">Data and resource availability</h2>
<p>All data, code, and materials used in this study are openly available:</p>
<ul>
<li><p><strong>GitHub repository:</strong> All raw data, processed data, and analysis code are hosted on GitHub at <a href="https://github.com/edwinreuvers/acc-mp-estimation">https://github.com/edwinreuvers/acc-mp-estimation</a>.</p></li>
<li><p><strong>Reproducible analysis website:</strong> Full analysis pipeline — including data, analysis code, and figure/table generation — is available at <a href="https://edwinreuvers.github.io/publications/acc-mp-estimation">https://edwinreuvers.github.io/publications/acc-mp-estimation</a>.</p></li>
</ul>
<div style="page-break-after: always;"></div>
<div style="page-break-after: always;"></div>
</section>
<section id="glossary" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="glossary">Glossary</h2>
<table class="caption-top table">
<colgroup>
<col style="width: 11%">
<col style="width: 88%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Symbol</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(a\)</span></td>
<td style="text-align: left;">Hill constant</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(b\)</span></td>
<td style="text-align: left;">Hill constant</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(b_{min}^{scale}\)</span></td>
<td style="text-align: left;">Minimal scale factor of <span class="math inline">\(b\)</span> (i.e., the minimal value of <span class="math inline">\(b_{scale}\)</span>)</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(b_{shape}\)</span></td>
<td style="text-align: left;">Determines the steepness of the relation between <span class="math inline">\(b_{scale}\)</span> and <span class="math inline">\(q\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(F_{asymp}\)</span></td>
<td style="text-align: left;">Oblique asymptote of the eccentric part of the CE force-velocity relationship</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(F_{CE}^{max}\)</span></td>
<td style="text-align: left;">Maximum isometric CE force</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(k_{PEE}\)</span></td>
<td style="text-align: left;">Scales the PEE stiffness</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(k_{SEE}\)</span></td>
<td style="text-align: left;">Scales the SEE stiffness</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(L_{CE}^{opt}\)</span></td>
<td style="text-align: left;">CE length at which CE can produce maximal isometric force</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(L_{PEE}^0\)</span></td>
<td style="text-align: left;">PEE slack length</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(L_{SEE}^0\)</span></td>
<td style="text-align: left;">SEE slack length</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(r_{as}\)</span></td>
<td style="text-align: left;">Slope of the slanted asymptote of the eccentric part of the CE force-velocity relationship</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(r_{slope}\)</span></td>
<td style="text-align: left;">Ratio between the eccentric and concentric derivatives of <span class="math inline">\(\frac{dF_{CE}^{rel}}{dv_{CE}^{rel}}\)</span> @ <span class="math inline">\(v_{CE}=0\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(w\)</span></td>
<td style="text-align: left;">Determines the width of the CE force-length relationship</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(a_{act}\)</span></td>
<td style="text-align: left;">Determines the steepness of the relation between <span class="math inline">\(\gamma\)</span> and <span class="math inline">\(q\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(b_{act,1}\)</span></td>
<td style="text-align: left;">Determines together with <span class="math inline">\(b_{act,1}\)</span> and <span class="math inline">\(b_{act,3}\)</span> the <span class="math inline">\(pCa^{2+}\)</span> level at which <span class="math inline">\(q=0.5\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(b_{act,2}\)</span></td>
<td style="text-align: left;">Determines together with <span class="math inline">\(b_{act,2}\)</span> and <span class="math inline">\(b_{act,3}\)</span> the <span class="math inline">\(pCa^{2+}\)</span> level at which <span class="math inline">\(q=0.5\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(b_{act,3}\)</span></td>
<td style="text-align: left;">Determines together with <span class="math inline">\(b_{act,1}\)</span> and <span class="math inline">\(b_{act,2}\)</span> the <span class="math inline">\(pCa^{2+}\)</span> level at which <span class="math inline">\(q=0.5\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(k_{Ca}\)</span></td>
<td style="text-align: left;">Relates <span class="math inline">\(\gamma\)</span> to the actual <span class="math inline">\(Ca^{2+}\)</span> concentration</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\gamma_0\)</span></td>
<td style="text-align: left;">Minimal value of <span class="math inline">\(\gamma\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(q_0\)</span></td>
<td style="text-align: left;">Minimal value <span class="math inline">\(q\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\tau_{act}\)</span></td>
<td style="text-align: left;">Activation time constant of the calcium dynamics</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\tau_{deact}\)</span></td>
<td style="text-align: left;">Deactivation time constant of the calcium dynamics</td>
</tr>
</tbody>
</table>
<div style="page-break-after: always;"></div>
</section>
<section id="references" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="references">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-aubert_tension-length_1951" class="csl-entry" role="listitem">
Aubert, X., M. L. Roquet, and J. Van der Elst. 1951. <span>“The <span>Tension</span>-<span>Length</span> <span>Diagram</span> of the <span>Frog</span>’s <span>Sartorius</span> <span>Muscle</span>.”</span> <em>Archives Internationales de Physiologie</em> 59 (2): 239–41. <a href="https://doi.org/10.3109/13813455109145002">https://doi.org/10.3109/13813455109145002</a>.
</div>
<div id="ref-blix_lange_1892" class="csl-entry" role="listitem">
Blix, Magnus. 1892. <span>“Die <span>Länge</span> Und Die <span>Spannung</span> Des <span>Muskels</span>.”</span> <em>Skandinavisches Archiv Für Physiologie</em> 3 (1): 295–318. <a href="https://doi.org/10.1111/j.1748-1716.1892.tb00660.x">https://doi.org/10.1111/j.1748-1716.1892.tb00660.x</a>.
</div>
<div id="ref-blix_lange_1893" class="csl-entry" role="listitem">
———. 1893. <span>“Die <span>Länge</span> Und Die <span>Spannung</span> Des <span>Muskels</span>.”</span> <em>Skandinavisches Archiv Für Physiologie</em> 4 (1): 399–409. <a href="https://doi.org/10.1111/j.1748-1716.1893.tb00749.x">https://doi.org/10.1111/j.1748-1716.1893.tb00749.x</a>.
</div>
<div id="ref-blumel_determining_2012" class="csl-entry" role="listitem">
Blümel, Marcus, Scott L. Hooper, Christoph Guschlbauerc, William E. White, and Ansgar Büschges. 2012. <span>“Determining All Parameters Necessary to Build <span>Hill</span>-Type Muscle Models from Experiments on Single Muscles.”</span> <em>Biological Cybernetics</em> 106 (10): 543–58. <a href="https://doi.org/10.1007/s00422-012-0531-5">https://doi.org/10.1007/s00422-012-0531-5</a>.
</div>
<div id="ref-bobbert_force-length_1990" class="csl-entry" role="listitem">
Bobbert, Maarten F., Gertjan C. Ettema, and Peter A. Huijing. 1990. <span>“The Force-Length Relationship of a Muscle-Tendon Complex: Experimental Results and Model Calculations.”</span> <em>European Journal of Applied Physiology and Occupational Physiology</em> 61 (3): 323–29. <a href="https://doi.org/10.1007/BF00357621">https://doi.org/10.1007/BF00357621</a>.
</div>
<div id="ref-bortolotto_mhc_2000" class="csl-entry" role="listitem">
Bortolotto, Susan K., Maria Cellini, D. George Stephenson, and Gabriela M. M. Stephenson. 2000. <span>“<span>MHC</span> Isoform Composition and <span>Ca2</span>+- or <span>Sr2</span>+-Activation Properties of Rat Skeletal Muscle Fibers.”</span> <em>American Journal of Physiology-Cell Physiology</em> 279 (5): C1564–77. <a href="https://doi.org/10.1152/ajpcell.2000.279.5.C1564">https://doi.org/10.1152/ajpcell.2000.279.5.C1564</a>.
</div>
<div id="ref-burkholder_sarcomere_2001" class="csl-entry" role="listitem">
Burkholder, Thomas J., and Richard L. Lieber. 2001. <span>“Sarcomere <span>Length</span> <span>Operating</span> <span>Range</span> of <span>Vertebrate</span> <span>Muscles</span> <span>During</span> <span>Movement</span>.”</span> <em>Journal of Experimental Biology</em> 204 (9): 1529–36. <a href="https://doi.org/10.1242/jeb.204.9.1529">https://doi.org/10.1242/jeb.204.9.1529</a>.
</div>
<div id="ref-cecchi_force-velocity_1978" class="csl-entry" role="listitem">
Cecchi, G, F Colomo, and V Lombardi. 1978. <span>“Force-Velocity Relation in Normal and Nitrate-Treated Frog Single Muscle Fibres During Rise of Tension in an Isometric Tetanus.”</span> <em>The Journal of Physiology</em> 285 (1): 257–73. <a href="https://doi.org/10.1113/jphysiol.1978.sp012570">https://doi.org/10.1113/jphysiol.1978.sp012570</a>.
</div>
<div id="ref-ebashi_calcium_1968" class="csl-entry" role="listitem">
Ebashi, S., and M. Endo. 1968. <span>“Calcium and Muscle Contraction.”</span> <em>Progress in Biophysics and Molecular Biology</em> 18 (January): 123–83. <a href="https://doi.org/10.1016/0079-6107(68)90023-0">https://doi.org/10.1016/0079-6107(68)90023-0</a>.
</div>
<div id="ref-gordon_variation_1966" class="csl-entry" role="listitem">
Gordon, A. M., A. F. Huxley, and F. J. Julian. 1966. <span>“The Variation in Isometric Tension with Sarcomere Length in Vertebrate Muscle Fibres.”</span> <em>The Journal of Physiology</em> 184 (1): 170–92. <a href="https://doi.org/10.1113/jphysiol.1966.sp007909">https://doi.org/10.1113/jphysiol.1966.sp007909</a>.
</div>
<div id="ref-hatze_myocybernetic_1981" class="csl-entry" role="listitem">
Hatze, Herbert. 1981. <em>Myocybernetic Control Models of Skeletal Muscle</em>. Pretoria: University of South Africa.
</div>
<div id="ref-hill_length_1925" class="csl-entry" role="listitem">
Hill, A. V. 1925. <span>“Length of Muscle, and the Heat and Tension Developed in an Isometric Contraction.”</span> <em>The Journal of Physiology</em> 60 (4): 237–63. <a href="https://doi.org/10.1113/jphysiol.1925.sp002242">https://doi.org/10.1113/jphysiol.1925.sp002242</a>.
</div>
<div id="ref-hill_heat_1938" class="csl-entry" role="listitem">
Hill, Archibald Vivian. 1938. <span>“The Heat of Shortening and the Dynamic Constants of Muscle.”</span> <em>Proceedings of the Royal Society of London. B. Biological Sciences</em> 126 (843): 136–95. <a href="https://doi.org/10.1098/rspb.1938.0050">https://doi.org/10.1098/rspb.1938.0050</a>.
</div>
<div id="ref-hill_series_1950" class="csl-entry" role="listitem">
———. 1950. <span>“The Series Elastic Componet of Muscle.”</span> <em>Proceedings of the Royal Society of London. B. Biological Sciences</em> 137 (887): 273–80. <a href="https://doi.org/10.1098/rspb.1950.0035">https://doi.org/10.1098/rspb.1950.0035</a>.
</div>
<div id="ref-katz_relation_1939" class="csl-entry" role="listitem">
Katz, Bernhard. 1939. <span>“The Relation Between Force and Speed in Muscular Contraction.”</span> <em>The Journal of Physiology</em> 96 (1): 45–64. <a href="https://doi.org/10.1113/jphysiol.1939.sp003756">https://doi.org/10.1113/jphysiol.1939.sp003756</a>.
</div>
<div id="ref-kistemaker_limiting_2023" class="csl-entry" role="listitem">
Kistemaker, Dinant A., Robert M. Terwiel, Edwin D. H. M. Reuvers, and Maarten F. Bobbert. 2023. <span>“Limiting Radial Pedal Forces Greatly Reduces Maximal Power Output and Efficiency in Sprint Cycling; an Optimal Control Study.”</span> <em>Journal of Applied Physiology</em> 134 (4): 980–91. <a href="https://doi.org/10.1152/japplphysiol.00733.2021">https://doi.org/10.1152/japplphysiol.00733.2021</a>.
</div>
<div id="ref-kistemaker_length-dependent_2005" class="csl-entry" role="listitem">
Kistemaker, Dinant A., Arthur (Knoek) J. Van Soest, and Maarten F. Bobbert. 2005. <span>“Length-Dependent [<span>Ca2</span>+] Sensitivity Adds Stiffness to Muscle.”</span> <em>Journal of Biomechanics</em> 38 (9): 1816–21. <a href="https://doi.org/10.1016/j.jbiomech.2004.08.025">https://doi.org/10.1016/j.jbiomech.2004.08.025</a>.
</div>
<div id="ref-lemaire_comparison_2016" class="csl-entry" role="listitem">
Lemaire, Koen K., Guus C. Baan, Richard T. Jaspers, and A. J. ‘Knoek’van Soest. 2016. <span>“Comparison of the Validity of <span>Hill</span> and <span>Huxley</span> Muscle–Tendon Complex Models Using Experimental Data Obtained from Rat m. Soleus&nbsp;in Situ.”</span> <em>Journal of Experimental Biology</em> 219 (14): 2228–28. <a href="https://doi.org/10.1242/jeb.144394">https://doi.org/10.1242/jeb.144394</a>.
</div>
<div id="ref-levin_viscous_1927" class="csl-entry" role="listitem">
Levin, A., and J. Wyman. 1927. <span>“The Viscous Elastic Properties of Muscle.”</span> <em>Proceedings of the Royal Society of London. Series B, Containing Papers of a Biological Character</em> 101 (709): 218–43. <a href="https://doi.org/10.1098/rspb.1927.0014">https://doi.org/10.1098/rspb.1927.0014</a>.
</div>
<div id="ref-petrofsky_influence_1981" class="csl-entry" role="listitem">
Petrofsky, Jerrold Scott, and Chandler Allen Phillips. 1981. <span>“The Influence of Temperature, Initial Length and Electrical Activity on the Force-Velocity Relationship of the Medial Gastrocnemius Muscle of the Cat.”</span> <em>Journal of Biomechanics</em> 14 (5): 297–306. <a href="https://doi.org/10.1016/0021-9290(81)90039-7">https://doi.org/10.1016/0021-9290(81)90039-7</a>.
</div>
<div id="ref-rack_effects_1969" class="csl-entry" role="listitem">
Rack, P. M. H., and D. R. Westbury. 1969. <span>“The Effects of Length and Stimulus Rate on Tension in the Isometric Cat Soleus Muscle.”</span> <em>The Journal of Physiology</em> 204 (2): 443–60. <a href="https://doi.org/10.1113/jphysiol.1969.sp008923">https://doi.org/10.1113/jphysiol.1969.sp008923</a>.
</div>
<div id="ref-reuvers_maximising_2025" class="csl-entry" role="listitem">
Reuvers, Edwin D. H. M, Huub Maas, Wendy Noort, Maarten F. Bobbert, and Dinant A. Kistemaker. 2025. <span>“Maximising Average Mechanical Power Output During Stretch-Shortening Cycles of Rat Medial Gastrocnemius Muscle.”</span> biorXiv. <a href="https://doi.org/10.1101/2025.09.25.678481">https://doi.org/10.1101/2025.09.25.678481</a>.
</div>
<div id="ref-rijkelijkhuizen_forcevelocity_2003" class="csl-entry" role="listitem">
Rijkelijkhuizen, J. M., C. J. de Ruiter, P. A. Huijing, and A. de Haan. 2003. <span>“Force/Velocity Curves of Fast Oxidative and Fast Glycolytic Parts of Rat Medial Gastrocnemius Muscle Vary for Concentric but Not Eccentric Activity.”</span> <em>Pflügers Archiv</em> 446 (4): 497–503. <a href="https://doi.org/10.1007/s00424-003-1052-9">https://doi.org/10.1007/s00424-003-1052-9</a>.
</div>
<div id="ref-van_soest_contribution_1993" class="csl-entry" role="listitem">
Soest, Arthur J. van, and Maarten F. Bobbert. 1993. <span>“The Contribution of Muscle Properties in the Control of Explosive Movements.”</span> <em>Biological Cybernetics</em> 69 (3): 195–204. <a href="https://doi.org/10.1007/BF00198959">https://doi.org/10.1007/BF00198959</a>.
</div>
<div id="ref-stephenson_effects_1982" class="csl-entry" role="listitem">
Stephenson, D. G., and D. A. Williams. 1982. <span>“Effects of Sarcomere Length on the Force—<span class="nocase">pCa</span> Relation in Fast- and Slow-Twitch Skinned Muscle Fibres from the Rat.”</span> <em>The Journal of Physiology</em> 333 (1): 637–53. <a href="https://doi.org/10.1113/jphysiol.1982.sp014473">https://doi.org/10.1113/jphysiol.1982.sp014473</a>.
</div>
<div id="ref-stern_computer_1974" class="csl-entry" role="listitem">
Stern, Jack T. 1974. <span>“Computer Modelling of Gross Muscle Dynamics.”</span> <em>Journal of Biomechanics</em> 7 (5): 411–28. <a href="https://doi.org/10.1016/0021-9290(74)90004-9">https://doi.org/10.1016/0021-9290(74)90004-9</a>.
</div>
<div id="ref-van_soest_consequences_2005" class="csl-entry" role="listitem">
Van Soest, A. J., Margit Gföhler, and L. J. Richard Casius. 2005. <span>“Consequences of <span>Ankle</span> <span>Joint</span> <span>Fixation</span> on <span>FES</span> <span>Cycling</span> <span>Power</span> <span>Output</span>: <span>A</span> <span>Simulation</span> <span>Study</span>.”</span> <em>Medicine &amp; Science in Sports &amp; Exercise</em> 37 (5). <a href="https://journals.lww.com/acsm-msse/fulltext/2005/05000/consequences_of_ankle_joint_fixation_on_fes.14.aspx">https://journals.lww.com/acsm-msse/fulltext/2005/05000/consequences_of_ankle_joint_fixation_on_fes.14.aspx</a>.
</div>
<div id="ref-walker_i_1974" class="csl-entry" role="listitem">
Walker, Sheppard M., and G. Randolph Schrodt. 1974. <span>“I Segment Lengths and Thin Filament Periods in Skeletal Muscle Fibers of the Rhesus Monkey and the Human.”</span> <em>The Anatomical Record</em> 178 (1): 63–81. <a href="https://doi.org/10.1002/ar.1091780107">https://doi.org/10.1002/ar.1091780107</a>.
</div>
<div id="ref-woittiez_three-dimensional_1984" class="csl-entry" role="listitem">
Woittiez, R. D., P. A. Huijing, H. B. K. Boom, and R. H. Rozendal. 1984. <span>“A Three-Dimensional Muscle Model: <span>A</span> Quantified Relation Between Form and Function of Skeletal Muscles.”</span> <em>Journal of Morphology</em> 182 (1): 95–113. <a href="https://doi.org/10.1002/jmor.1051820107">https://doi.org/10.1002/jmor.1051820107</a>.
</div>
<div id="ref-zajac_muscle_1989" class="csl-entry" role="listitem">
Zajac, F. E. 1989. <span>“Muscle and Tendon: Properties, Models, Scaling, and Application to Biomechanics and Motor Control.”</span> <em>Critical Reviews in Biomedical Engineering</em> 17 (4): 359–411.
</div>
<div id="ref-van_zandwijk_evaluation_1997" class="csl-entry" role="listitem">
Zandwijk, Jan Peter van, Guus C. Baan, Maarten F. Bobbert, and Peter A. Huijing. 1997. <span>“Evaluation of a Self-Consistent Method for Calculating Muscle Parameters from a Set of Isokinetic Releases.”</span> <em>Biological Cybernetics</em> 77 (4): 277–81. <a href="https://doi.org/10.1007/s004220050388">https://doi.org/10.1007/s004220050388</a>.
</div>
</div>
</section>
<section id="supplementary-material" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="supplementary-material">Supplementary material</h2>
<section id="supplemental-figures" class="level3" data-number="4.1">
<h3 data-number="4.1" class="anchored" data-anchor-id="supplemental-figures"><span class="header-section-number">4.1</span> Supplemental figures</h3>
<div class="quarto-embed-nb-cell">
<div id="cell-suppfig-servo-vs-lever" class="cell" data-execution_count="10">
<div class="cell-output cell-output-display">
<div id="suppfig-servo-vs-lever" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-servo-vs-lever-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-suppfig-servo-vs-lever-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-servo-vs-lever-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S1: Comparison between a length-controlled step-ramp experiment and a force-controlled quick-release experiment. In length-controlled step-ramp experiment (left), MTC length over time is imposed using a servomotor, while SEE force is measured. MTC velocity is computed at the time instance at which SEE force is near constant. In the force-controlled quick-release experiment (right), SEE force over time is imposed via a lever, while MTC length is measured. MTC velocity is computed at the time instance where MTC velocity is maximal after the change in SEE force. Both methods yield one datapoint (depicted with the orange dot) of the force-velocity relationship.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-suppfig-servo-vs-lever">Source: Figures</a></div>
<div class="quarto-embed-nb-cell">
<div id="cell-suppfig-flowchart" class="cell" data-execution_count="11">
<div class="cell-output cell-output-display">
<div id="suppfig-flowchart" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-flowchart-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-suppfig-flowchart-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-flowchart-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S2: Flowchart of the improved method. Using the improved method, parameter values were estimated until the change in all parameter values was less than 0.1%.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-suppfig-flowchart">Source: Figures</a></div>
<div class="quarto-embed-nb-cell">
<div id="cell-suppfig-ksee-qr-duration" class="cell" data-execution_count="12">
<div class="cell-output cell-output-display">
<div id="suppfig-ksee-qr-duration" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-suppfig figure">
<div aria-describedby="suppfig-ksee-qr-duration-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="preprint_files/figure-html/..-figures-all_figures-suppfig-ksee-qr-duration-output-1.svg" class="img-fluid figure-img">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-suppfig" id="suppfig-ksee-qr-duration-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure S3: Relationship between quick-release duration and SEE stiffness understimation. When the duration of the quick-release increases, the underestimation of SEE stiffness increases.
</figcaption>
</figure>
</div>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-1" href="../figures/all_figures.html#cell-suppfig-ksee-qr-duration">Source: Figures</a></div>
<div style="page-break-after: always;"></div>
</section>
<section id="supplemental-tables" class="level3" data-number="4.2">
<h3 data-number="4.2" class="anchored" data-anchor-id="supplemental-tables"><span class="header-section-number">4.2</span> Supplemental tables</h3>
<div class="quarto-embed-nb-cell">
<div class="cell" data-tbl-cap="MTC parameter values obtained from literature to simulate data.">
<div id="supptbl-overview" class="cell-output cell-output-display quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-supptbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-supptbl quarto-uncaptioned" id="supptbl-overview-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table S1
</figcaption>
<div aria-describedby="supptbl-overview-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">



<meta charset="utf-8">


<div id="ohtsqjewet" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>
#ohtsqjewet table {
          font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
          -webkit-font-smoothing: antialiased;
          -moz-osx-font-smoothing: grayscale;
        }

#ohtsqjewet thead, tbody, tfoot, tr, td, th { border-style: none !important; }
 tr { background-color: transparent !important; }
#ohtsqjewet p { margin: 0 !important; padding: 0 !important; }
 #ohtsqjewet .gt_table { display: table !important; border-collapse: collapse !important; line-height: normal !important; margin-left: auto !important; margin-right: auto !important; color: #333333 !important; font-size: 16px !important; font-weight: normal !important; font-style: normal !important; background-color: #FFFFFF !important; width: auto !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #A8A8A8 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #A8A8A8 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_caption { padding-top: 4px !important; padding-bottom: 4px !important; }
 #ohtsqjewet .gt_title { color: #333333 !important; font-size: 125% !important; font-weight: initial !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; border-bottom-color: #FFFFFF !important; border-bottom-width: 0 !important; }
 #ohtsqjewet .gt_subtitle { color: #333333 !important; font-size: 85% !important; font-weight: initial !important; padding-top: 3px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; border-top-color: #FFFFFF !important; border-top-width: 0 !important; }
 #ohtsqjewet .gt_heading { background-color: #FFFFFF !important; text-align: center !important; border-bottom-color: #FFFFFF !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_bottom_border { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_col_headings { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_col_heading { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; overflow-x: hidden !important; }
 #ohtsqjewet .gt_column_spanner_outer { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; padding-top: 0 !important; padding-bottom: 0 !important; padding-left: 4px !important; padding-right: 4px !important; }
 #ohtsqjewet .gt_column_spanner_outer:first-child { padding-left: 0 !important; }
 #ohtsqjewet .gt_column_spanner_outer:last-child { padding-right: 0 !important; }
 #ohtsqjewet .gt_column_spanner { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; overflow-x: hidden !important; display: inline-block !important; width: 100% !important; }
 #ohtsqjewet .gt_spanner_row { border-bottom-style: hidden !important; }
 #ohtsqjewet .gt_group_heading { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; text-align: left !important; }
 #ohtsqjewet .gt_empty_group_heading { padding: 0.5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: middle !important; }
 #ohtsqjewet .gt_from_md> :first-child { margin-top: 0 !important; }
 #ohtsqjewet .gt_from_md> :last-child { margin-bottom: 0 !important; }
 #ohtsqjewet .gt_row { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; margin: 10px !important; border-top-style: solid !important; border-top-width: 1px !important; border-top-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; overflow-x: hidden !important; }
 #ohtsqjewet .gt_stub { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; }
 #ohtsqjewet .gt_stub_row_group { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; vertical-align: top !important; }
 #ohtsqjewet .gt_row_group_first td { border-top-width: 2px !important; }
 #ohtsqjewet .gt_row_group_first th { border-top-width: 2px !important; }
 #ohtsqjewet .gt_striped { color: #333333 !important; background-color: #F4F4F4 !important; }
 #ohtsqjewet .gt_table_body { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_grand_summary_row { color: #333333 !important; background-color: #FFFFFF !important; text-transform: inherit !important; padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; }
 #ohtsqjewet .gt_first_grand_summary_row_bottom { border-top-style: double !important; border-top-width: 6px !important; border-top-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_last_grand_summary_row_top { border-bottom-style: double !important; border-bottom-width: 6px !important; border-bottom-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_sourcenotes { color: #333333 !important; background-color: #FFFFFF !important; border-bottom-style: none !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; }
 #ohtsqjewet .gt_sourcenote { font-size: 90% !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; }
 #ohtsqjewet .gt_left { text-align: left !important; }
 #ohtsqjewet .gt_center { text-align: center !important; }
 #ohtsqjewet .gt_right { text-align: right !important; font-variant-numeric: tabular-nums !important; }
 #ohtsqjewet .gt_font_normal { font-weight: normal !important; }
 #ohtsqjewet .gt_font_bold { font-weight: bold !important; }
 #ohtsqjewet .gt_font_italic { font-style: italic !important; }
 #ohtsqjewet .gt_super { font-size: 65% !important; }
 #ohtsqjewet .gt_footnote_marks { font-size: 75% !important; vertical-align: 0.4em !important; position: initial !important; }
 #ohtsqjewet .gt_asterisk { font-size: 100% !important; vertical-align: 0 !important; }
 
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-bootstrap="false">
<caption>MTC parameter values obtained from literature to simulate data.</caption>
<thead>
<tr class="gt_col_headings header">
<th class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"></th>
<th id="Unit" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Unit</th>
<th id="GM1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM1</th>
<th id="GM2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM2</th>
<th id="GM3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">GM3</th>
<th id="Reference" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Reference</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="gt_group_heading_row odd">
<th colspan="6" class="gt_group_heading" data-quarto-table-cell-role="th" style="font-style: italic">Contraction dynamics</th>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$a$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">N</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">2.68</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.80</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">2.58</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$b$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">mm</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">41.6</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">24.8</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">41.8</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$b_{scale}^{min}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.100</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.100</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.100</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_soest_contribution_1993">van Soest and Bobbert (<a href="#ref-van_soest_contribution_1993" role="doc-biblioref" aria-expanded="false">1993</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$b_{shape}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">22.0</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">22.0</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">22.0</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_soest_contribution_1993">van Soest and Bobbert (<a href="#ref-van_soest_contribution_1993" role="doc-biblioref" aria-expanded="false">1993</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$F_{asymp}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.50</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.50</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.50</td>
<td class="gt_row gt_left"><span class="citation" data-cites="rijkelijkhuizen_forcevelocity_2003">Rijkelhuizen et al. (<a href="#ref-rijkelijkhuizen_forcevelocity_2003" role="doc-biblioref" aria-expanded="false">2003</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$F_{CE}^{max}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">N</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">13.4</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">13.8</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">12.3</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$k_{PEE}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">N/mm<sup>2</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">213</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">165</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">511</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$k_{SEE}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">N/mm<sup>2</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">4,220</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">3,640</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">3,470</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$L_{CE}^{opt}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">mm</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">13.2</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">12.3</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">11.2</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$L_{PEE}^0$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">mm</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">13.9</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">13.2</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">13.4</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$L_{SEE}^0$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">mm</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">28.3</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">30.3</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">26.5</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$r_{as}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.00371</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.00545</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.00316</td>
<td class="gt_row gt_left">Arbitrary small value</td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$r_{slope}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">mm</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">2.00</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">2.00</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">2.00</td>
<td class="gt_row gt_left"><span class="citation" data-cites="katz_relation_1939">Katz (1939) (<a href="#ref-katz_relation_1939" role="doc-biblioref" aria-expanded="false">1939</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$w$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.500</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.500</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.500</td>
<td class="gt_row gt_left"><span class="citation" data-cites="burkholder_sarcomere_2001">Burkholder and Lieber (<a href="#ref-burkholder_sarcomere_2001" role="doc-biblioref" aria-expanded="false">2001</a>)</span></td>
</tr>
<tr class="gt_group_heading_row even">
<th colspan="6" class="gt_group_heading" data-quarto-table-cell-role="th" style="font-style: italic">Excitation dynamics</th>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$a_{act}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">?</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">−7.37</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">−7.37</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">−7.37</td>
<td class="gt_row gt_left"><span class="citation" data-cites="bortolotto_mhc_2000">Bortolotto et al. (<a href="#ref-bortolotto_mhc_2000" role="doc-biblioref" aria-expanded="false">2000</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$b_{act,1}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">?</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">5.17</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">5.17</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">5.17</td>
<td class="gt_row gt_left"><span class="citation" data-cites="bortolotto_mhc_2000">Bortolotto et al. (<a href="#ref-bortolotto_mhc_2000" role="doc-biblioref" aria-expanded="false">2000</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$b_{act,2}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">?</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.596</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.596</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.596</td>
<td class="gt_row gt_left"><span class="citation" data-cites="stephenson_effects_1982">Stephenson and Williams (<a href="#ref-stephenson_effects_1982" role="doc-biblioref" aria-expanded="false">1982</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$b_{act,3}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">?</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.00</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.00</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">0.00</td>
<td class="gt_row gt_left"><span class="citation" data-cites="stephenson_effects_1982">Stephenson and Williams (<a href="#ref-stephenson_effects_1982" role="doc-biblioref" aria-expanded="false">1982</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$kCa$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">mol/L</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">8.00 × 10<sup>−6</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">8.00 × 10<sup>−6</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">8.00 × 10<sup>−6</sup></td>
<td class="gt_row gt_left"><span class="citation" data-cites="kistemaker_length-dependent_2005">Kistemaker et al. (<a href="#ref-kistemaker_length-dependent_2005" role="doc-biblioref" aria-expanded="false">2005</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$\gamma_0$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.00 × 10<sup>−5</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.00 × 10<sup>−5</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">1.00 × 10<sup>−5</sup></td>
<td class="gt_row gt_left">Arbitrary small value</td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$q_0$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">-</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">5.00 × 10<sup>−3</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">5.00 × 10<sup>−3</sup></td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">5.00 × 10<sup>−3</sup></td>
<td class="gt_row gt_left"><span class="citation" data-cites="hatze_myocybernetic_1981">Hatze (<a href="#ref-hatze_myocybernetic_1981" role="doc-biblioref" aria-expanded="false">1981</a>)</span></td>
</tr>
<tr class="even">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$\tau_{act}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">ms</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">27.0</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">27.0</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">27.0</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
<tr class="odd">
<th class="gt_row gt_left gt_stub" data-quarto-table-cell-role="th" style="border-right: 0px solid #D3D3D3">$\tau_{deact}$</th>
<td class="gt_row gt_left" style="border-right: 2px solid #D3D3D3">ms</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">27.0</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">27.0</td>
<td class="gt_row gt_center" style="border-right: 2px solid #D3D3D3">27.0</td>
<td class="gt_row gt_left"><span class="citation" data-cites="van_zandwijk_evaluation_1997">van Zandwijk et al. (<a href="#ref-van_zandwijk_evaluation_1997" role="doc-biblioref" aria-expanded="false">1997</a>)</span></td>
</tr>
</tbody>
</table>


</div>
        


</div>
</figure>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-2" href="../tables/all_tables.html#cell-supptbl-overview">Source: Tables</a></div>
<div class="quarto-embed-nb-cell">
<div class="cell" data-tbl-cap="Estimated parameter values of the experimentally measured *in situ* data using both the traditional as well as the improved method." data-execution_count="10">
<div id="supptbl-insitu" class="cell-output cell-output-display quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-supptbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-supptbl quarto-uncaptioned" id="supptbl-insitu-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table S2
</figcaption>
<div aria-describedby="supptbl-insitu-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">



<meta charset="utf-8">


<div id="ycyfhpbxpb" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>
#ycyfhpbxpb table {
          font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
          -webkit-font-smoothing: antialiased;
          -moz-osx-font-smoothing: grayscale;
        }

#ycyfhpbxpb thead, tbody, tfoot, tr, td, th { border-style: none !important; }
 tr { background-color: transparent !important; }
#ycyfhpbxpb p { margin: 0 !important; padding: 0 !important; }
 #ycyfhpbxpb .gt_table { display: table !important; border-collapse: collapse !important; line-height: normal !important; margin-left: auto !important; margin-right: auto !important; color: #333333 !important; font-size: 16px !important; font-weight: normal !important; font-style: normal !important; background-color: #FFFFFF !important; width: auto !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #A8A8A8 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #A8A8A8 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_caption { padding-top: 4px !important; padding-bottom: 4px !important; }
 #ycyfhpbxpb .gt_title { color: #333333 !important; font-size: 125% !important; font-weight: initial !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; border-bottom-color: #FFFFFF !important; border-bottom-width: 0 !important; }
 #ycyfhpbxpb .gt_subtitle { color: #333333 !important; font-size: 85% !important; font-weight: initial !important; padding-top: 3px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; border-top-color: #FFFFFF !important; border-top-width: 0 !important; }
 #ycyfhpbxpb .gt_heading { background-color: #FFFFFF !important; text-align: center !important; border-bottom-color: #FFFFFF !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_bottom_border { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_col_headings { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_col_heading { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; overflow-x: hidden !important; }
 #ycyfhpbxpb .gt_column_spanner_outer { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; padding-top: 0 !important; padding-bottom: 0 !important; padding-left: 4px !important; padding-right: 4px !important; }
 #ycyfhpbxpb .gt_column_spanner_outer:first-child { padding-left: 0 !important; }
 #ycyfhpbxpb .gt_column_spanner_outer:last-child { padding-right: 0 !important; }
 #ycyfhpbxpb .gt_column_spanner { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; overflow-x: hidden !important; display: inline-block !important; width: 100% !important; }
 #ycyfhpbxpb .gt_spanner_row { border-bottom-style: hidden !important; }
 #ycyfhpbxpb .gt_group_heading { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; text-align: left !important; }
 #ycyfhpbxpb .gt_empty_group_heading { padding: 0.5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: middle !important; }
 #ycyfhpbxpb .gt_from_md> :first-child { margin-top: 0 !important; }
 #ycyfhpbxpb .gt_from_md> :last-child { margin-bottom: 0 !important; }
 #ycyfhpbxpb .gt_row { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; margin: 10px !important; border-top-style: solid !important; border-top-width: 1px !important; border-top-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; overflow-x: hidden !important; }
 #ycyfhpbxpb .gt_stub { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; }
 #ycyfhpbxpb .gt_stub_row_group { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; vertical-align: top !important; }
 #ycyfhpbxpb .gt_row_group_first td { border-top-width: 2px !important; }
 #ycyfhpbxpb .gt_row_group_first th { border-top-width: 2px !important; }
 #ycyfhpbxpb .gt_striped { color: #333333 !important; background-color: #F4F4F4 !important; }
 #ycyfhpbxpb .gt_table_body { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_grand_summary_row { color: #333333 !important; background-color: #FFFFFF !important; text-transform: inherit !important; padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; }
 #ycyfhpbxpb .gt_first_grand_summary_row_bottom { border-top-style: double !important; border-top-width: 6px !important; border-top-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_last_grand_summary_row_top { border-bottom-style: double !important; border-bottom-width: 6px !important; border-bottom-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_sourcenotes { color: #333333 !important; background-color: #FFFFFF !important; border-bottom-style: none !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; }
 #ycyfhpbxpb .gt_sourcenote { font-size: 90% !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; }
 #ycyfhpbxpb .gt_left { text-align: left !important; }
 #ycyfhpbxpb .gt_center { text-align: center !important; }
 #ycyfhpbxpb .gt_right { text-align: right !important; font-variant-numeric: tabular-nums !important; }
 #ycyfhpbxpb .gt_font_normal { font-weight: normal !important; }
 #ycyfhpbxpb .gt_font_bold { font-weight: bold !important; }
 #ycyfhpbxpb .gt_font_italic { font-style: italic !important; }
 #ycyfhpbxpb .gt_super { font-size: 65% !important; }
 #ycyfhpbxpb .gt_footnote_marks { font-size: 75% !important; vertical-align: 0.4em !important; position: initial !important; }
 #ycyfhpbxpb .gt_asterisk { font-size: 100% !important; vertical-align: 0 !important; }
 
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-bootstrap="false">
<caption>Estimated parameter values of the experimentally measured <em>in situ</em> data using both the traditional as well as the improved method.</caption>
<colgroup>
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
<col style="width: 12%">
</colgroup>
<thead>
<tr class="gt_col_headings gt_spanner_row header">
<th rowspan="2" id="index" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"></th>
<th rowspan="2" id="Unit" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Unit</th>
<th colspan="3" id="Traditional-method" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Traditional method</span></th>
<th colspan="3" id="Improved-method" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Improved method</span></th>
</tr>
<tr class="gt_col_headings even">
<th id="TM_GMe1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Rat 1</th>
<th id="TM_GMe2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Rat 2</th>
<th id="TM_GMe3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Rat 3</th>
<th id="IM_GMe1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Rat 1</th>
<th id="IM_GMe2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Rat 2</th>
<th id="IM_GMe3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">Rat 3</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left">$a$</td>
<td class="gt_row gt_center">N</td>
<td class="gt_row gt_center">8.90</td>
<td class="gt_row gt_center">11.3</td>
<td class="gt_row gt_center">10.5</td>
<td class="gt_row gt_center">8.82</td>
<td class="gt_row gt_center">10.0</td>
<td class="gt_row gt_center">11.0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$b$</td>
<td class="gt_row gt_center">mm</td>
<td class="gt_row gt_center">76.7</td>
<td class="gt_row gt_center">108</td>
<td class="gt_row gt_center">84.6</td>
<td class="gt_row gt_center">81.9</td>
<td class="gt_row gt_center">105</td>
<td class="gt_row gt_center">90.6</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$F_{CE}^{max}$</td>
<td class="gt_row gt_center">N</td>
<td class="gt_row gt_center">15.6</td>
<td class="gt_row gt_center">14.1</td>
<td class="gt_row gt_center">17.1</td>
<td class="gt_row gt_center">15.5</td>
<td class="gt_row gt_center">14.1</td>
<td class="gt_row gt_center">17.0</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$k_{PEE}$</td>
<td class="gt_row gt_center">N/mm<sup>2</sup></td>
<td class="gt_row gt_center">9.83</td>
<td class="gt_row gt_center">7.25</td>
<td class="gt_row gt_center">21.9</td>
<td class="gt_row gt_center">28.7</td>
<td class="gt_row gt_center">24.1</td>
<td class="gt_row gt_center">34.8</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" style="border-top: 2px solid red; border-bottom: 2px solid red">$k_{SEE}$</td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">N/mm<sup>2</sup></td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">608</td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">688</td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">654</td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">952</td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">1260</td>
<td class="gt_row gt_center" style="border-top: 2px solid red; border-bottom: 2px solid red">1050</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$L_{CE}^{opt}$</td>
<td class="gt_row gt_center">mm</td>
<td class="gt_row gt_center">12.4</td>
<td class="gt_row gt_center">12.9</td>
<td class="gt_row gt_center">12.3</td>
<td class="gt_row gt_center">13.7</td>
<td class="gt_row gt_center">14.7</td>
<td class="gt_row gt_center">13.3</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$L_{PEE}^0$</td>
<td class="gt_row gt_center">mm</td>
<td class="gt_row gt_center">12.4</td>
<td class="gt_row gt_center">12.4</td>
<td class="gt_row gt_center">13.3</td>
<td class="gt_row gt_center">15.1</td>
<td class="gt_row gt_center">15.6</td>
<td class="gt_row gt_center">14.5</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$L_{SEE}^0$</td>
<td class="gt_row gt_center">mm</td>
<td class="gt_row gt_center">29.0</td>
<td class="gt_row gt_center">29.7</td>
<td class="gt_row gt_center">25.1</td>
<td class="gt_row gt_center">28.8</td>
<td class="gt_row gt_center">29.3</td>
<td class="gt_row gt_center">25.2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">$\tau_{act}$</td>
<td class="gt_row gt_center">ms</td>
<td class="gt_row gt_center">43.0</td>
<td class="gt_row gt_center">32.4</td>
<td class="gt_row gt_center">23.6</td>
<td class="gt_row gt_center">55.2</td>
<td class="gt_row gt_center">57.7</td>
<td class="gt_row gt_center">41.6</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">$\tau_{deact}$</td>
<td class="gt_row gt_center">ms</td>
<td class="gt_row gt_center">23.8</td>
<td class="gt_row gt_center">20.9</td>
<td class="gt_row gt_center">19.7</td>
<td class="gt_row gt_center">27.1</td>
<td class="gt_row gt_center">25.3</td>
<td class="gt_row gt_center">22.4</td>
</tr>
</tbody>
</table>


</div>
        


</div>
</figure>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-2" href="../tables/all_tables.html#cell-supptbl-insitu">Source: Tables</a></div>
<div class="quarto-embed-nb-cell">
<div class="cell" data-tbl-cap="Root mean squared differences between experimentally measured SEE force histories and those predicted by Hill-type MTC model after estimating all contraction and excitation dynamics parameter values." data-execution_count="22">
<div id="supptbl-rmsd" class="cell-output cell-output-display quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-supptbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-supptbl quarto-uncaptioned" id="supptbl-rmsd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table S3
</figcaption>
<div aria-describedby="supptbl-rmsd-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">



<meta charset="utf-8">


<div id="whpyaipdzt" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>
#whpyaipdzt table {
          font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Helvetica Neue', 'Fira Sans', 'Droid Sans', Arial, sans-serif;
          -webkit-font-smoothing: antialiased;
          -moz-osx-font-smoothing: grayscale;
        }

#whpyaipdzt thead, tbody, tfoot, tr, td, th { border-style: none !important; }
 tr { background-color: transparent !important; }
#whpyaipdzt p { margin: 0 !important; padding: 0 !important; }
 #whpyaipdzt .gt_table { display: table !important; border-collapse: collapse !important; line-height: normal !important; margin-left: auto !important; margin-right: auto !important; color: #333333 !important; font-size: 16px !important; font-weight: normal !important; font-style: normal !important; background-color: #FFFFFF !important; width: auto !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #A8A8A8 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #A8A8A8 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_caption { padding-top: 4px !important; padding-bottom: 4px !important; }
 #whpyaipdzt .gt_title { color: #333333 !important; font-size: 125% !important; font-weight: initial !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; border-bottom-color: #FFFFFF !important; border-bottom-width: 0 !important; }
 #whpyaipdzt .gt_subtitle { color: #333333 !important; font-size: 85% !important; font-weight: initial !important; padding-top: 3px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; border-top-color: #FFFFFF !important; border-top-width: 0 !important; }
 #whpyaipdzt .gt_heading { background-color: #FFFFFF !important; text-align: center !important; border-bottom-color: #FFFFFF !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_bottom_border { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_col_headings { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_col_heading { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; padding-left: 5px !important; padding-right: 5px !important; overflow-x: hidden !important; }
 #whpyaipdzt .gt_column_spanner_outer { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: normal !important; text-transform: inherit !important; padding-top: 0 !important; padding-bottom: 0 !important; padding-left: 4px !important; padding-right: 4px !important; }
 #whpyaipdzt .gt_column_spanner_outer:first-child { padding-left: 0 !important; }
 #whpyaipdzt .gt_column_spanner_outer:last-child { padding-right: 0 !important; }
 #whpyaipdzt .gt_column_spanner { border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: bottom !important; padding-top: 5px !important; padding-bottom: 5px !important; overflow-x: hidden !important; display: inline-block !important; width: 100% !important; }
 #whpyaipdzt .gt_spanner_row { border-bottom-style: hidden !important; }
 #whpyaipdzt .gt_group_heading { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; text-align: left !important; }
 #whpyaipdzt .gt_empty_group_heading { padding: 0.5px !important; color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; vertical-align: middle !important; }
 #whpyaipdzt .gt_from_md> :first-child { margin-top: 0 !important; }
 #whpyaipdzt .gt_from_md> :last-child { margin-bottom: 0 !important; }
 #whpyaipdzt .gt_row { padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; margin: 10px !important; border-top-style: solid !important; border-top-width: 1px !important; border-top-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 1px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 1px !important; border-right-color: #D3D3D3 !important; vertical-align: middle !important; overflow-x: hidden !important; }
 #whpyaipdzt .gt_stub { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; }
 #whpyaipdzt .gt_stub_row_group { color: #333333 !important; background-color: #FFFFFF !important; font-size: 100% !important; font-weight: initial !important; text-transform: inherit !important; border-right-style: solid !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; padding-left: 5px !important; padding-right: 5px !important; vertical-align: top !important; }
 #whpyaipdzt .gt_row_group_first td { border-top-width: 2px !important; }
 #whpyaipdzt .gt_row_group_first th { border-top-width: 2px !important; }
 #whpyaipdzt .gt_striped { color: #333333 !important; background-color: #F4F4F4 !important; }
 #whpyaipdzt .gt_table_body { border-top-style: solid !important; border-top-width: 2px !important; border-top-color: #D3D3D3 !important; border-bottom-style: solid !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_grand_summary_row { color: #333333 !important; background-color: #FFFFFF !important; text-transform: inherit !important; padding-top: 8px !important; padding-bottom: 8px !important; padding-left: 5px !important; padding-right: 5px !important; }
 #whpyaipdzt .gt_first_grand_summary_row_bottom { border-top-style: double !important; border-top-width: 6px !important; border-top-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_last_grand_summary_row_top { border-bottom-style: double !important; border-bottom-width: 6px !important; border-bottom-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_sourcenotes { color: #333333 !important; background-color: #FFFFFF !important; border-bottom-style: none !important; border-bottom-width: 2px !important; border-bottom-color: #D3D3D3 !important; border-left-style: none !important; border-left-width: 2px !important; border-left-color: #D3D3D3 !important; border-right-style: none !important; border-right-width: 2px !important; border-right-color: #D3D3D3 !important; }
 #whpyaipdzt .gt_sourcenote { font-size: 90% !important; padding-top: 4px !important; padding-bottom: 4px !important; padding-left: 5px !important; padding-right: 5px !important; text-align: left !important; }
 #whpyaipdzt .gt_left { text-align: left !important; }
 #whpyaipdzt .gt_center { text-align: center !important; }
 #whpyaipdzt .gt_right { text-align: right !important; font-variant-numeric: tabular-nums !important; }
 #whpyaipdzt .gt_font_normal { font-weight: normal !important; }
 #whpyaipdzt .gt_font_bold { font-weight: bold !important; }
 #whpyaipdzt .gt_font_italic { font-style: italic !important; }
 #whpyaipdzt .gt_super { font-size: 65% !important; }
 #whpyaipdzt .gt_footnote_marks { font-size: 75% !important; vertical-align: 0.4em !important; position: initial !important; }
 #whpyaipdzt .gt_asterisk { font-size: 100% !important; vertical-align: 0 !important; }
 
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-bootstrap="false">
<caption>Root mean squared differences between experimentally measured SEE force histories and those predicted by Hill-type MTC model after estimating all contraction and excitation dynamics parameter values.</caption>
<thead>
<tr class="gt_col_headings gt_spanner_row header">
<th rowspan="2" id="index" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"></th>
<th colspan="4" id="Traditional-method" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Traditional method</span></th>
<th colspan="4" id="Improved-method" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><span class="gt_column_spanner">Improved method</span></th>
</tr>
<tr class="gt_col_headings even">
<th id="TM_QR" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">QR</th>
<th id="TM_SR" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">SR</th>
<th id="TM_ISOM" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">ISOM</th>
<th id="TM_SSC" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">SSC</th>
<th id="IM_QR" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">QR</th>
<th id="IM_SR" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">SR</th>
<th id="IM_ISOM" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">ISOM</th>
<th id="IM_SSC" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col">SSC</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left">Rat 1</td>
<td class="gt_row gt_center">5.6 ± 3.4</td>
<td class="gt_row gt_center">5.6 ± 0.8</td>
<td class="gt_row gt_center">6.8 ± 3.2</td>
<td class="gt_row gt_center">6.2 ± 3.0</td>
<td class="gt_row gt_center">3.3 ± 1.5</td>
<td class="gt_row gt_center">2.8 ± 0.9</td>
<td class="gt_row gt_center">5.4 ± 3.0</td>
<td class="gt_row gt_center">5.4 ± 3.1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">Rat 2</td>
<td class="gt_row gt_center">4.2 ± 1.5</td>
<td class="gt_row gt_center">5.0 ± 0.6</td>
<td class="gt_row gt_center">4.3 ± 1.6</td>
<td class="gt_row gt_center">4.6 ± 2.2</td>
<td class="gt_row gt_center">2.7 ± 1.6</td>
<td class="gt_row gt_center">2.1 ± 0.5</td>
<td class="gt_row gt_center">4.2 ± 2.6</td>
<td class="gt_row gt_center">4.2 ± 2.0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left">Rat 3</td>
<td class="gt_row gt_center">6.4 ± 2.3</td>
<td class="gt_row gt_center">6.8 ± 1.0</td>
<td class="gt_row gt_center">6.1 ± 2.1</td>
<td class="gt_row gt_center">5.5 ± 2.7</td>
<td class="gt_row gt_center">4.2 ± 1.8</td>
<td class="gt_row gt_center">2.6 ± 0.5</td>
<td class="gt_row gt_center">4.9 ± 2.3</td>
<td class="gt_row gt_center">5.0 ± 2.7</td>
</tr>
<tr class="even">
<td class="gt_row gt_left">Avg ± Std</td>
<td class="gt_row gt_center">5.4 ± 2.6</td>
<td class="gt_row gt_center">5.8 ± 1.1</td>
<td class="gt_row gt_center">5.7 ± 2.7</td>
<td class="gt_row gt_center">5.4 ± 2.7</td>
<td class="gt_row gt_center">3.4 ± 1.8</td>
<td class="gt_row gt_center">2.5 ± 0.7</td>
<td class="gt_row gt_center">4.8 ± 2.7</td>
<td class="gt_row gt_center">4.9 ± 2.7</td>
</tr>
</tbody>
</table>


</div>
        


</div>
</figure>
</div>
</div>
<a class="quarto-notebook-link" id="nblink-2" href="../tables/all_tables.html#cell-supptbl-rmsd">Source: Tables</a></div>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
      const outerScaffold = trigger.parentElement.cloneNode(true);
      const codeEl = outerScaffold.querySelector('code');
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
<p>All content licensed under <a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">CC BY-NC-SA 4.0</a> <i class="fa-brands fa-creative-commons" aria-label="creative-commons"></i> <i class="fa-brands fa-creative-commons-by" aria-label="creative-commons-by"></i> <i class="fa-brands fa-creative-commons-nc" aria-label="creative-commons-nc"></i> <i class="fa-brands fa-creative-commons-sa" aria-label="creative-commons-sa"></i></p>
</div>   
    <div class="nav-footer-center">
      &nbsp;
    <div class="toc-actions d-sm-block d-md-none"><ul><li><a href="https://github.com/edwinreuvers/acc-mp-estimation/blob/main/manuscript/preprint.ipynb" class="toc-action"><i class="bi bi-github"></i>View source</a></li></ul></div></div>
    <div class="nav-footer-right">
<p>Made with <i class="fa-brands fa-r-project" aria-label="r-project"></i> and <a href="https://quarto.org/">Quarto</a></p>
</div>
  </div>
</footer>




</body></html>